// @ark-ui/solid@5.14.0/dist/chunk/UK24ZAUL.js
import { useFieldContext } from './NC7W2KIE.js';
import { createSplitProps } from './ZMHI4GDJ.js';
import { ark } from './EPLBB4QN.js';
import { useEnvironmentContext } from './YO2MCGXO.js';
import { useLocaleContext } from './OKZ64GSY.js';
import { createContext } from './TROPIN4C.js';
import { runIfFn } from './DT73WLR4.js';
import { __export } from './ESLJRKWD.js';
import { createComponent, mergeProps as mergeProps$1 } from 'solid-js/web';
import { mergeProps, useMachine, normalizeProps } from '@zag-js/solid';
import { createMemo, createUniqueId } from 'solid-js';
import * as ratingGroup from '@zag-js/rating-group';
export { anatomy } from '@zag-js/rating-group';

// src/components/rating-group/use-rating-group-context.ts
var [RatingGroupProvider, useRatingGroupContext] = createContext({
  hookName: "useRatingGroupContext",
  providerName: "<RatingGroupProvider />"
});

// src/components/rating-group/rating-group-context.tsx
var RatingGroupContext = (props) => props.children(useRatingGroupContext());
var RatingGroupControl = (props) => {
  const ratingGroup2 = useRatingGroupContext();
  const mergedProps = mergeProps(() => ratingGroup2().getControlProps(), props);
  return createComponent(ark.div, mergedProps);
};
var RatingGroupHiddenInput = (props) => {
  const ratingGroup2 = useRatingGroupContext();
  const mergedProps = mergeProps(() => ratingGroup2().getHiddenInputProps(), props);
  const field = useFieldContext();
  return createComponent(ark.input, mergeProps$1({
    get ["aria-describedby"]() {
      return field?.().ariaDescribedby;
    }
  }, mergedProps));
};

// src/components/rating-group/use-rating-group-item-context.ts
var [RatingGroupItemProvider, useRatingGroupItemContext] = createContext({
  hookName: "useRatingGroupItemContext",
  providerName: "<RatingGroupItemProvider />"
});

// src/components/rating-group/rating-group-item.tsx
var RatingGroupItem = (props) => {
  const [itemProps, localProps] = createSplitProps()(props, ["index"]);
  const api = useRatingGroupContext();
  const mergedProps = mergeProps(() => api().getItemProps(itemProps), localProps);
  const itemState = createMemo(() => api().getItemState(itemProps));
  return createComponent(RatingGroupItemProvider, {
    value: itemState,
    get children() {
      return createComponent(ark.span, mergedProps);
    }
  });
};

// src/components/rating-group/rating-group-item-context.tsx
var RatingGroupItemContext = (props) => props.children(useRatingGroupItemContext());
var RatingGroupLabel = (props) => {
  const api = useRatingGroupContext();
  const mergedProps = mergeProps(() => api().getLabelProps(), props);
  return createComponent(ark.label, mergedProps);
};
var useRatingGroup = (props) => {
  const id = createUniqueId();
  const locale = useLocaleContext();
  const environment = useEnvironmentContext();
  const field = useFieldContext();
  const machineProps = createMemo(() => ({
    id,
    ids: {
      label: field?.().ids.label,
      hiddenInput: field?.().ids.control
    },
    disabled: field?.().disabled,
    readOnly: field?.().readOnly,
    required: field?.().required,
    dir: locale().dir,
    getRootNode: environment().getRootNode,
    ...runIfFn(props)
  }));
  const service = useMachine(ratingGroup.machine, machineProps);
  return createMemo(() => ratingGroup.connect(service, normalizeProps));
};

// src/components/rating-group/rating-group-root.tsx
var RatingGroupRoot = (props) => {
  const [useRatingProps, localProps] = createSplitProps()(props, ["allowHalf", "autoFocus", "count", "defaultValue", "disabled", "form", "id", "ids", "name", "onHoverChange", "onValueChange", "readOnly", "required", "translations", "value"]);
  const api = useRatingGroup(useRatingProps);
  const mergedProps = mergeProps(() => api().getRootProps(), localProps);
  return createComponent(RatingGroupProvider, {
    value: api,
    get children() {
      return createComponent(ark.div, mergedProps);
    }
  });
};
var RatingGroupRootProvider = (props) => {
  const [{
    value: ratingGroup2
  }, localProps] = createSplitProps()(props, ["value"]);
  const mergedProps = mergeProps(() => ratingGroup2().getRootProps(), localProps);
  return createComponent(RatingGroupProvider, {
    value: ratingGroup2,
    get children() {
      return createComponent(ark.div, mergedProps);
    }
  });
};

// src/components/rating-group/rating-group.ts
var rating_group_exports = {};
__export(rating_group_exports, {
  Context: () => RatingGroupContext,
  Control: () => RatingGroupControl,
  HiddenInput: () => RatingGroupHiddenInput,
  Item: () => RatingGroupItem,
  ItemContext: () => RatingGroupItemContext,
  Label: () => RatingGroupLabel,
  Root: () => RatingGroupRoot,
  RootProvider: () => RatingGroupRootProvider
});

export { RatingGroupContext, RatingGroupControl, RatingGroupHiddenInput, RatingGroupItem, RatingGroupItemContext, RatingGroupLabel, RatingGroupRoot, RatingGroupRootProvider, rating_group_exports, useRatingGroup, useRatingGroupContext, useRatingGroupItemContext };
