// @ark-ui/solid@5.14.0/dist/chunk/XUBES5JE.js
import { useLocaleContext } from './OKZ64GSY.js';
import { __export } from './ESLJRKWD.js';
import { memo } from 'solid-js/web';
import { formatRelativeTime, formatNumber, formatBytes } from '@zag-js/i18n-utils';
import { createMemo } from 'solid-js';

var FormatByte = (props) => {
  const ctx = useLocaleContext();
  const text = createMemo(() => {
    const {
      value,
      ...intlOptions
    } = props;
    return formatBytes(value, ctx().locale, intlOptions);
  });
  return memo(text);
};
var FormatNumber = (props) => {
  const ctx = useLocaleContext();
  const text = createMemo(() => {
    const {
      value,
      ...intlOptions
    } = props;
    return formatNumber(value, ctx().locale, intlOptions);
  });
  return memo(text);
};
var FormatRelativeTime = (props) => {
  const ctx = useLocaleContext();
  const text = createMemo(() => {
    const {
      value,
      ...intlOptions
    } = props;
    return formatRelativeTime(value, ctx().locale, intlOptions);
  });
  return memo(text);
};

// src/components/format/format.ts
var format_exports = {};
__export(format_exports, {
  Byte: () => FormatByte,
  Number: () => FormatNumber,
  RelativeTime: () => FormatRelativeTime
});

export { FormatByte, FormatNumber, FormatRelativeTime, format_exports };
