// .ignored/lucide-solid/dist/source/aliases/aliases.jsx
/**
* @license lucide-solid v0.514.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/aliases.ts
import { default as default2 } from "../icons/alarm-check";
import { default as default3 } from "../icons/alarm-minus";
import { default as default4 } from "../icons/alarm-plus";
import { default as default5 } from "../icons/arrow-down-az";
import { default as default6 } from "../icons/sort-desc";
import { default as default7 } from "../icons/arrow-down-za";
import { default as default8 } from "../icons/arrow-up-az";
import { default as default9 } from "../icons/sort-asc";
import { default as default10 } from "../icons/arrow-up-za";
import { default as default11 } from "../icons/axis-3-d";
import { default as default12 } from "../icons/verified";
import {
  default as default13
} from "../icons/between-horizonal-end";
import {
  default as default14
} from "../icons/between-horizonal-start";
import { default as default15 } from "../icons/book-template";
import { default as default16 } from "../icons/curly-braces";
import { default as default17 } from "../icons/subtitles";
import {
  default as default18
} from "../icons/area-chart";
import {
  default as default19
} from "../icons/bar-chart-horizontal-big";
import {
  default as default20
} from "../icons/bar-chart-horizontal";
import {
  default as default21
} from "../icons/candlestick-chart";
import {
  default as default22
} from "../icons/bar-chart-big";
import {
  default as default23
} from "../icons/bar-chart-4";
import {
  default as default24
} from "../icons/bar-chart-3";
import {
  default as default25
} from "../icons/line-chart";
import {
  default as default26
} from "../icons/bar-chart";
import {
  default as default27
} from "../icons/bar-chart-2";
import {
  default as default28
} from "../icons/gantt-chart";
import {
  default as default29
} from "../icons/pie-chart";
import {
  default as default30
} from "../icons/scatter-chart";
import { default as default31 } from "../icons/alert-circle";
import { default as default32 } from "../icons/arrow-down-circle";
import { default as default33 } from "../icons/arrow-left-circle";
import { default as default34 } from "../icons/arrow-down-left-from-circle";
import { default as default35 } from "../icons/arrow-down-right-from-circle";
import { default as default36 } from "../icons/arrow-up-left-from-circle";
import { default as default37 } from "../icons/arrow-up-right-from-circle";
import { default as default38 } from "../icons/arrow-right-circle";
import { default as default39 } from "../icons/arrow-up-circle";
import { default as default40 } from "../icons/check-circle";
import { default as default41 } from "../icons/check-circle-2";
import { default as default42 } from "../icons/chevron-down-circle";
import { default as default43 } from "../icons/chevron-left-circle";
import { default as default44 } from "../icons/chevron-right-circle";
import { default as default45 } from "../icons/chevron-up-circle";
import { default as default46 } from "../icons/divide-circle";
import { default as default47 } from "../icons/gauge-circle";
import { default as default48 } from "../icons/help-circle";
import { default as default49 } from "../icons/minus-circle";
import { default as default50 } from "../icons/parking-circle-off";
import { default as default51 } from "../icons/parking-circle";
import { default as default52 } from "../icons/percent-circle";
import { default as default53 } from "../icons/pause-circle";
import { default as default54 } from "../icons/play-circle";
import { default as default55 } from "../icons/plus-circle";
import { default as default56 } from "../icons/power-circle";
import { default as default57 } from "../icons/circle-slashed";
import { default as default58 } from "../icons/stop-circle";
import {
  default as default59
} from "../icons/user-circle-2";
import { default as default60 } from "../icons/user-circle";
import { default as default61 } from "../icons/x-circle";
import { default as default62 } from "../icons/clipboard-signature";
import { default as default63 } from "../icons/clipboard-edit";
import { default as default64 } from "../icons/download-cloud";
import { default as default65 } from "../icons/upload-cloud";
import {
  default as default66
} from "../icons/code-2";
import { default as default67 } from "../icons/columns";
import { default as default68 } from "../icons/columns-settings";
import { default as default69 } from "../icons/table-config";
import { default as default70 } from "../icons/panels-left-right";
import {
  default as default71
} from "../icons/contact-2";
import { default as default72 } from "../icons/percent-diamond";
import {
  default as default73
} from "../icons/globe-2";
import { default as default74 } from "../icons/more-vertical";
import { default as default75 } from "../icons/more-horizontal";
import { default as default76 } from "../icons/file-axis-3-d";
import {
  default as default77
} from "../icons/file-bar-chart";
import {
  default as default78
} from "../icons/file-line-chart";
import {
  default as default79
} from "../icons/file-bar-chart-2";
import {
  default as default80
} from "../icons/file-pie-chart";
import {
  default as default81
} from "../icons/file-cog-2";
import { default as default82 } from "../icons/file-signature";
import { default as default83 } from "../icons/file-edit";
import {
  default as default84
} from "../icons/folder-cog-2";
import { default as default85 } from "../icons/folder-edit";
import {
  default as default86
} from "../icons/filter-x";
import {
  default as default87
} from "../icons/filter";
import { default as default88 } from "../icons/git-commit";
import { default as default89 } from "../icons/grid-2-x-2-check";
import { default as default90 } from "../icons/grid-2-x-2-plus";
import { default as default91 } from "../icons/grid-2-x-2-x";
import { default as default92 } from "../icons/grid-2-x-2";
import { default as default93 } from "../icons/grid";
import { default as default94 } from "../icons/grid-3-x-3";
import { default as default95 } from "../icons/helping-hand";
import {
  default as default96
} from "../icons/ice-cream-2";
import { default as default97 } from "../icons/home";
import { default as default98 } from "../icons/ice-cream";
import { default as default99 } from "../icons/outdent";
import { default as default100 } from "../icons/indent";
import {
  default as default101
} from "../icons/laptop-2";
import { default as default102 } from "../icons/layers-3";
import { default as default103 } from "../icons/loader-2";
import { default as default104 } from "../icons/unlock-keyhole";
import { default as default105 } from "../icons/unlock";
import {
  default as default106
} from "../icons/mic-2";
import { default as default107 } from "../icons/move-3-d";
import { default as default108 } from "../icons/alert-octagon";
import { default as default109 } from "../icons/pause-octagon";
import { default as default110 } from "../icons/x-octagon";
import { default as default111 } from "../icons/paintbrush-2";
import { default as default112 } from "../icons/panel-bottom-inactive";
import { default as default113 } from "../icons/sidebar-close";
import { default as default114 } from "../icons/panel-left-inactive";
import { default as default115 } from "../icons/sidebar-open";
import { default as default116 } from "../icons/sidebar";
import { default as default117 } from "../icons/panel-right-inactive";
import { default as default118 } from "../icons/panel-top-inactive";
import { default as default119 } from "../icons/layout";
import {
  default as default120
} from "../icons/edit-3";
import {
  default as default121
} from "../icons/edit-2";
import { default as default122 } from "../icons/plug-zap-2";
import { default as default123 } from "../icons/form-input";
import { default as default124 } from "../icons/rotate-3-d";
import { default as default125 } from "../icons/rows";
import { default as default126 } from "../icons/panels-top-bottom";
import { default as default127 } from "../icons/scale-3-d";
import {
  default as default128
} from "../icons/send-horizonal";
import { default as default129 } from "../icons/shield-close";
import { default as default130 } from "../icons/sliders";
import { default as default131 } from "../icons/stars";
import { default as default132 } from "../icons/activity-square";
import { default as default133 } from "../icons/arrow-down-left-square";
import { default as default134 } from "../icons/arrow-down-right-square";
import { default as default135 } from "../icons/arrow-left-square";
import { default as default136 } from "../icons/arrow-down-square";
import { default as default137 } from "../icons/arrow-down-left-from-square";
import { default as default138 } from "../icons/arrow-down-right-from-square";
import { default as default139 } from "../icons/arrow-up-left-from-square";
import { default as default140 } from "../icons/arrow-up-right-from-square";
import { default as default141 } from "../icons/arrow-right-square";
import { default as default142 } from "../icons/arrow-up-left-square";
import { default as default143 } from "../icons/arrow-up-right-square";
import { default as default144 } from "../icons/arrow-up-square";
import { default as default145 } from "../icons/asterisk-square";
import { default as default146 } from "../icons/scissors-square-dashed-bottom";
import { default as default147 } from "../icons/gantt-chart-square";
import { default as default148 } from "../icons/square-gantt-chart";
import { default as default149 } from "../icons/check-square-2";
import { default as default150 } from "../icons/check-square";
import { default as default151 } from "../icons/chevron-down-square";
import { default as default152 } from "../icons/chevron-left-square";
import { default as default153 } from "../icons/chevron-right-square";
import { default as default154 } from "../icons/chevron-up-square";
import { default as default155 } from "../icons/code-square";
import { default as default156 } from "../icons/kanban-square-dashed";
import { default as default157 } from "../icons/mouse-pointer-square-dashed";
import { default as default158 } from "../icons/box-select";
import { default as default159 } from "../icons/divide-square";
import { default as default160 } from "../icons/dot-square";
import { default as default161 } from "../icons/equal-square";
import { default as default162 } from "../icons/function-square";
import { default as default163 } from "../icons/kanban-square";
import { default as default164 } from "../icons/library-square";
import { default as default165 } from "../icons/m-square";
import { default as default166 } from "../icons/menu-square";
import { default as default167 } from "../icons/minus-square";
import { default as default168 } from "../icons/inspect";
import { default as default169 } from "../icons/parking-square-off";
import { default as default170 } from "../icons/parking-square";
import { default as default171 } from "../icons/pen-box";
import { default as default172 } from "../icons/pen-square";
import { default as default173 } from "../icons/edit";
import { default as default174 } from "../icons/percent-square";
import { default as default175 } from "../icons/pi-square";
import { default as default176 } from "../icons/pilcrow-square";
import { default as default177 } from "../icons/play-square";
import { default as default178 } from "../icons/plus-square";
import { default as default179 } from "../icons/power-square";
import { default as default180 } from "../icons/scissors-square";
import { default as default181 } from "../icons/sigma-square";
import { default as default182 } from "../icons/slash-square";
import { default as default183 } from "../icons/split-square-horizontal";
import { default as default184 } from "../icons/split-square-vertical";
import {
  default as default185
} from "../icons/user-square-2";
import { default as default186 } from "../icons/terminal-square";
import { default as default187 } from "../icons/user-square";
import { default as default188 } from "../icons/x-square";
import { default as default189 } from "../icons/test-tube-2";
import { default as default190 } from "../icons/text-selection";
import { default as default191 } from "../icons/train";
import { default as default192 } from "../icons/palmtree";
import { default as default193 } from "../icons/alert-triangle";
import {
  default as default194
} from "../icons/tv-2";
import {
  default as default195
} from "../icons/school-2";
import {
  default as default196
} from "../icons/user-check-2";
import {
  default as default197
} from "../icons/user-cog-2";
import {
  default as default198
} from "../icons/user-minus-2";
import {
  default as default199
} from "../icons/user-plus-2";
import {
  default as default200
} from "../icons/user-x-2";
import {
  default as default201
} from "../icons/user-2";
import {
  default as default202
} from "../icons/users-2";
import { default as default203 } from "../icons/fork-knife-crossed";
import { default as default204 } from "../icons/fork-knife";
import {
  default as default205
} from "../icons/wallet-2";
import {
  default as default206
} from "../icons/wand-2";
export {
  default132 as ActivitySquare,
  default2 as AlarmCheck,
  default3 as AlarmMinus,
  default4 as AlarmPlus,
  default31 as AlertCircle,
  default108 as AlertOctagon,
  default193 as AlertTriangle,
  default18 as AreaChart,
  default5 as ArrowDownAz,
  default32 as ArrowDownCircle,
  default34 as ArrowDownLeftFromCircle,
  default137 as ArrowDownLeftFromSquare,
  default133 as ArrowDownLeftSquare,
  default35 as ArrowDownRightFromCircle,
  default138 as ArrowDownRightFromSquare,
  default134 as ArrowDownRightSquare,
  default136 as ArrowDownSquare,
  default7 as ArrowDownZa,
  default33 as ArrowLeftCircle,
  default135 as ArrowLeftSquare,
  default38 as ArrowRightCircle,
  default141 as ArrowRightSquare,
  default8 as ArrowUpAz,
  default39 as ArrowUpCircle,
  default36 as ArrowUpLeftFromCircle,
  default139 as ArrowUpLeftFromSquare,
  default142 as ArrowUpLeftSquare,
  default37 as ArrowUpRightFromCircle,
  default140 as ArrowUpRightFromSquare,
  default143 as ArrowUpRightSquare,
  default144 as ArrowUpSquare,
  default10 as ArrowUpZa,
  default145 as AsteriskSquare,
  default11 as Axis3D,
  default26 as BarChart,
  default27 as BarChart2,
  default24 as BarChart3,
  default23 as BarChart4,
  default22 as BarChartBig,
  default20 as BarChartHorizontal,
  default19 as BarChartHorizontalBig,
  default13 as BetweenHorizonalEnd,
  default14 as BetweenHorizonalStart,
  default15 as BookTemplate,
  default158 as BoxSelect,
  default21 as CandlestickChart,
  default40 as CheckCircle,
  default41 as CheckCircle2,
  default150 as CheckSquare,
  default149 as CheckSquare2,
  default42 as ChevronDownCircle,
  default151 as ChevronDownSquare,
  default43 as ChevronLeftCircle,
  default152 as ChevronLeftSquare,
  default44 as ChevronRightCircle,
  default153 as ChevronRightSquare,
  default45 as ChevronUpCircle,
  default154 as ChevronUpSquare,
  default57 as CircleSlashed,
  default63 as ClipboardEdit,
  default62 as ClipboardSignature,
  default66 as Code2,
  default155 as CodeSquare,
  default67 as Columns,
  default68 as ColumnsSettings,
  default71 as Contact2,
  default16 as CurlyBraces,
  default46 as DivideCircle,
  default159 as DivideSquare,
  default160 as DotSquare,
  default64 as DownloadCloud,
  default173 as Edit,
  default121 as Edit2,
  default120 as Edit3,
  default161 as EqualSquare,
  default76 as FileAxis3D,
  default77 as FileBarChart,
  default79 as FileBarChart2,
  default81 as FileCog2,
  default83 as FileEdit,
  default78 as FileLineChart,
  default80 as FilePieChart,
  default82 as FileSignature,
  default87 as Filter,
  default86 as FilterX,
  default84 as FolderCog2,
  default85 as FolderEdit,
  default204 as ForkKnife,
  default203 as ForkKnifeCrossed,
  default123 as FormInput,
  default162 as FunctionSquare,
  default28 as GanttChart,
  default147 as GanttChartSquare,
  default47 as GaugeCircle,
  default88 as GitCommit,
  default73 as Globe2,
  default93 as Grid,
  default92 as Grid2X2,
  default89 as Grid2X2Check,
  default90 as Grid2X2Plus,
  default91 as Grid2X2X,
  default94 as Grid3X3,
  default48 as HelpCircle,
  default95 as HelpingHand,
  default97 as Home,
  default98 as IceCream,
  default96 as IceCream2,
  default100 as Indent,
  default168 as Inspect,
  default163 as KanbanSquare,
  default156 as KanbanSquareDashed,
  default101 as Laptop2,
  default102 as Layers3,
  default119 as Layout,
  default164 as LibrarySquare,
  default25 as LineChart,
  default103 as Loader2,
  default165 as MSquare,
  default166 as MenuSquare,
  default106 as Mic2,
  default49 as MinusCircle,
  default167 as MinusSquare,
  default75 as MoreHorizontal,
  default74 as MoreVertical,
  default157 as MousePointerSquareDashed,
  default107 as Move3D,
  default99 as Outdent,
  default111 as Paintbrush2,
  default192 as Palmtree,
  default112 as PanelBottomInactive,
  default114 as PanelLeftInactive,
  default117 as PanelRightInactive,
  default118 as PanelTopInactive,
  default70 as PanelsLeftRight,
  default126 as PanelsTopBottom,
  default51 as ParkingCircle,
  default50 as ParkingCircleOff,
  default170 as ParkingSquare,
  default169 as ParkingSquareOff,
  default53 as PauseCircle,
  default109 as PauseOctagon,
  default171 as PenBox,
  default172 as PenSquare,
  default52 as PercentCircle,
  default72 as PercentDiamond,
  default174 as PercentSquare,
  default175 as PiSquare,
  default29 as PieChart,
  default176 as PilcrowSquare,
  default54 as PlayCircle,
  default177 as PlaySquare,
  default122 as PlugZap2,
  default55 as PlusCircle,
  default178 as PlusSquare,
  default56 as PowerCircle,
  default179 as PowerSquare,
  default124 as Rotate3D,
  default125 as Rows,
  default127 as Scale3D,
  default30 as ScatterChart,
  default195 as School2,
  default180 as ScissorsSquare,
  default146 as ScissorsSquareDashedBottom,
  default128 as SendHorizonal,
  default129 as ShieldClose,
  default116 as Sidebar,
  default113 as SidebarClose,
  default115 as SidebarOpen,
  default181 as SigmaSquare,
  default182 as SlashSquare,
  default130 as Sliders,
  default9 as SortAsc,
  default6 as SortDesc,
  default183 as SplitSquareHorizontal,
  default184 as SplitSquareVertical,
  default148 as SquareGanttChart,
  default131 as Stars,
  default58 as StopCircle,
  default17 as Subtitles,
  default69 as TableConfig,
  default186 as TerminalSquare,
  default189 as TestTube2,
  default190 as TextSelection,
  default191 as Train,
  default194 as Tv2,
  default105 as Unlock,
  default104 as UnlockKeyhole,
  default65 as UploadCloud,
  default201 as User2,
  default196 as UserCheck2,
  default60 as UserCircle,
  default59 as UserCircle2,
  default197 as UserCog2,
  default198 as UserMinus2,
  default199 as UserPlus2,
  default187 as UserSquare,
  default185 as UserSquare2,
  default200 as UserX2,
  default202 as Users2,
  default12 as Verified,
  default205 as Wallet2,
  default206 as Wand2,
  default61 as XCircle,
  default110 as XOctagon,
  default188 as XSquare
};
