// .ignored/lucide-solid/dist/source/aliases/prefixed.jsx
/**
* @license lucide-solid v0.514.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/prefixed.ts
import { default as default2 } from "../icons/a-arrow-down";
import { default as default3 } from "../icons/a-arrow-up";
import { default as default4 } from "../icons/a-large-small";
import { default as default5 } from "../icons/accessibility";
import { default as default6 } from "../icons/activity";
import { default as default7 } from "../icons/air-vent";
import { default as default8 } from "../icons/airplay";
import { default as default9 } from "../icons/alarm-clock-off";
import { default as default10 } from "../icons/alarm-clock";
import { default as default11 } from "../icons/alarm-smoke";
import { default as default12 } from "../icons/album";
import { default as default13 } from "../icons/align-center-horizontal";
import { default as default14 } from "../icons/align-center-vertical";
import { default as default15 } from "../icons/align-center";
import { default as default16 } from "../icons/align-end-horizontal";
import { default as default17 } from "../icons/align-end-vertical";
import { default as default18 } from "../icons/align-horizontal-distribute-center";
import { default as default19 } from "../icons/align-horizontal-distribute-end";
import { default as default20 } from "../icons/align-horizontal-distribute-start";
import { default as default21 } from "../icons/align-horizontal-justify-center";
import { default as default22 } from "../icons/align-horizontal-justify-end";
import { default as default23 } from "../icons/align-horizontal-justify-start";
import { default as default24 } from "../icons/align-horizontal-space-around";
import { default as default25 } from "../icons/align-horizontal-space-between";
import { default as default26 } from "../icons/align-justify";
import { default as default27 } from "../icons/align-left";
import { default as default28 } from "../icons/align-right";
import { default as default29 } from "../icons/align-start-horizontal";
import { default as default30 } from "../icons/align-start-vertical";
import { default as default31 } from "../icons/align-vertical-distribute-center";
import { default as default32 } from "../icons/align-vertical-distribute-end";
import { default as default33 } from "../icons/align-vertical-distribute-start";
import { default as default34 } from "../icons/align-vertical-justify-center";
import { default as default35 } from "../icons/align-vertical-justify-start";
import { default as default36 } from "../icons/align-vertical-space-around";
import { default as default37 } from "../icons/align-vertical-justify-end";
import { default as default38 } from "../icons/align-vertical-space-between";
import { default as default39 } from "../icons/ambulance";
import { default as default40 } from "../icons/ampersand";
import { default as default41 } from "../icons/ampersands";
import { default as default42 } from "../icons/amphora";
import { default as default43 } from "../icons/anchor";
import { default as default44 } from "../icons/angry";
import { default as default45 } from "../icons/antenna";
import { default as default46 } from "../icons/annoyed";
import { default as default47 } from "../icons/anvil";
import { default as default48 } from "../icons/aperture";
import { default as default49 } from "../icons/app-window-mac";
import { default as default50 } from "../icons/apple";
import { default as default51 } from "../icons/archive-restore";
import { default as default52 } from "../icons/app-window";
import { default as default53 } from "../icons/archive-x";
import { default as default54 } from "../icons/archive";
import { default as default55 } from "../icons/arrow-big-down-dash";
import { default as default56 } from "../icons/armchair";
import { default as default57 } from "../icons/arrow-big-left-dash";
import { default as default58 } from "../icons/arrow-big-down";
import { default as default59 } from "../icons/arrow-big-right-dash";
import { default as default60 } from "../icons/arrow-big-left";
import { default as default61 } from "../icons/arrow-big-right";
import { default as default62 } from "../icons/arrow-big-up-dash";
import { default as default63 } from "../icons/arrow-big-up";
import { default as default64 } from "../icons/arrow-down-from-line";
import { default as default65 } from "../icons/arrow-down-left";
import { default as default66 } from "../icons/arrow-down-narrow-wide";
import { default as default67 } from "../icons/arrow-down-to-dot";
import { default as default68 } from "../icons/arrow-down-right";
import { default as default69 } from "../icons/arrow-down-up";
import { default as default70 } from "../icons/arrow-down-to-line";
import { default as default71 } from "../icons/arrow-down";
import { default as default72 } from "../icons/arrow-left-from-line";
import { default as default73 } from "../icons/arrow-left-right";
import { default as default74 } from "../icons/arrow-left-to-line";
import { default as default75 } from "../icons/arrow-left";
import { default as default76 } from "../icons/arrow-right-from-line";
import { default as default77 } from "../icons/arrow-right-left";
import { default as default78 } from "../icons/arrow-right-to-line";
import { default as default79 } from "../icons/arrow-right";
import { default as default80 } from "../icons/arrow-up-down";
import { default as default81 } from "../icons/arrow-up-from-dot";
import { default as default82 } from "../icons/arrow-up-from-line";
import { default as default83 } from "../icons/arrow-up-left";
import { default as default84 } from "../icons/arrow-up-right";
import { default as default85 } from "../icons/arrow-up-to-line";
import { default as default86 } from "../icons/arrow-up-wide-narrow";
import { default as default87 } from "../icons/arrow-up";
import { default as default88 } from "../icons/arrows-up-from-line";
import { default as default89 } from "../icons/asterisk";
import { default as default90 } from "../icons/at-sign";
import { default as default91 } from "../icons/atom";
import { default as default92 } from "../icons/audio-waveform";
import { default as default93 } from "../icons/audio-lines";
import { default as default94 } from "../icons/award";
import { default as default95 } from "../icons/axe";
import { default as default96 } from "../icons/baby";
import { default as default97 } from "../icons/backpack";
import { default as default98 } from "../icons/badge-alert";
import { default as default99 } from "../icons/badge-cent";
import { default as default100 } from "../icons/badge-dollar-sign";
import { default as default101 } from "../icons/badge-euro";
import { default as default102 } from "../icons/badge-help";
import { default as default103 } from "../icons/badge-indian-rupee";
import { default as default104 } from "../icons/badge-info";
import { default as default105 } from "../icons/badge-japanese-yen";
import { default as default106 } from "../icons/badge-minus";
import { default as default107 } from "../icons/badge-plus";
import { default as default108 } from "../icons/badge-percent";
import { default as default109 } from "../icons/badge-pound-sterling";
import { default as default110 } from "../icons/badge-russian-ruble";
import { default as default111 } from "../icons/badge-swiss-franc";
import { default as default112 } from "../icons/badge-x";
import { default as default113 } from "../icons/badge";
import { default as default114 } from "../icons/baggage-claim";
import { default as default115 } from "../icons/ban";
import { default as default116 } from "../icons/banana";
import { default as default117 } from "../icons/bandage";
import { default as default118 } from "../icons/banknote-arrow-down";
import { default as default119 } from "../icons/banknote-arrow-up";
import { default as default120 } from "../icons/banknote-x";
import { default as default121 } from "../icons/banknote";
import { default as default122 } from "../icons/barcode";
import { default as default123 } from "../icons/baseline";
import { default as default124 } from "../icons/bath";
import { default as default125 } from "../icons/battery-charging";
import { default as default126 } from "../icons/battery-full";
import { default as default127 } from "../icons/battery-low";
import { default as default128 } from "../icons/battery-medium";
import { default as default129 } from "../icons/battery-warning";
import { default as default130 } from "../icons/battery";
import { default as default131 } from "../icons/battery-plus";
import { default as default132 } from "../icons/beaker";
import { default as default133 } from "../icons/bean-off";
import { default as default134 } from "../icons/bean";
import { default as default135 } from "../icons/bed-double";
import { default as default136 } from "../icons/bed-single";
import { default as default137 } from "../icons/bed";
import { default as default138 } from "../icons/beef";
import { default as default139 } from "../icons/beer-off";
import { default as default140 } from "../icons/beer";
import { default as default141 } from "../icons/bell-dot";
import { default as default142 } from "../icons/bell-electric";
import { default as default143 } from "../icons/bell-minus";
import { default as default144 } from "../icons/bell-off";
import { default as default145 } from "../icons/bell-plus";
import { default as default146 } from "../icons/bell-ring";
import { default as default147 } from "../icons/bell";
import { default as default148 } from "../icons/between-vertical-end";
import { default as default149 } from "../icons/between-vertical-start";
import { default as default150 } from "../icons/biceps-flexed";
import { default as default151 } from "../icons/bike";
import { default as default152 } from "../icons/binary";
import { default as default153 } from "../icons/binoculars";
import { default as default154 } from "../icons/biohazard";
import { default as default155 } from "../icons/bird";
import { default as default156 } from "../icons/blend";
import { default as default157 } from "../icons/bitcoin";
import { default as default158 } from "../icons/blinds";
import { default as default159 } from "../icons/blocks";
import { default as default160 } from "../icons/bluetooth-connected";
import { default as default161 } from "../icons/bluetooth-off";
import { default as default162 } from "../icons/bluetooth-searching";
import { default as default163 } from "../icons/bluetooth";
import { default as default164 } from "../icons/bold";
import { default as default165 } from "../icons/bomb";
import { default as default166 } from "../icons/bolt";
import { default as default167 } from "../icons/bone";
import { default as default168 } from "../icons/book-a";
import { default as default169 } from "../icons/book-alert";
import { default as default170 } from "../icons/book-audio";
import { default as default171 } from "../icons/book-check";
import { default as default172 } from "../icons/book-copy";
import { default as default173 } from "../icons/book-down";
import { default as default174 } from "../icons/book-headphones";
import { default as default175 } from "../icons/book-heart";
import { default as default176 } from "../icons/book-image";
import { default as default177 } from "../icons/book-key";
import { default as default178 } from "../icons/book-lock";
import { default as default179 } from "../icons/book-marked";
import { default as default180 } from "../icons/book-minus";
import { default as default181 } from "../icons/book-open-check";
import { default as default182 } from "../icons/book-open-text";
import { default as default183 } from "../icons/book-open";
import { default as default184 } from "../icons/book-plus";
import { default as default185 } from "../icons/book-text";
import { default as default186 } from "../icons/book-type";
import { default as default187 } from "../icons/book-up-2";
import { default as default188 } from "../icons/book-up";
import { default as default189 } from "../icons/book-user";
import { default as default190 } from "../icons/book-x";
import { default as default191 } from "../icons/book";
import { default as default192 } from "../icons/bookmark-check";
import { default as default193 } from "../icons/bookmark-minus";
import { default as default194 } from "../icons/bookmark-plus";
import { default as default195 } from "../icons/bookmark-x";
import { default as default196 } from "../icons/bookmark";
import { default as default197 } from "../icons/boom-box";
import { default as default198 } from "../icons/bot-message-square";
import { default as default199 } from "../icons/bot-off";
import { default as default200 } from "../icons/bot";
import { default as default201 } from "../icons/box";
import { default as default202 } from "../icons/bow-arrow";
import { default as default203 } from "../icons/boxes";
import { default as default204 } from "../icons/brain-circuit";
import { default as default205 } from "../icons/brackets";
import { default as default206 } from "../icons/brain-cog";
import { default as default207 } from "../icons/brain";
import { default as default208 } from "../icons/brick-wall-fire";
import { default as default209 } from "../icons/brick-wall";
import { default as default210 } from "../icons/briefcase-business";
import { default as default211 } from "../icons/briefcase-conveyor-belt";
import { default as default212 } from "../icons/briefcase-medical";
import { default as default213 } from "../icons/briefcase";
import { default as default214 } from "../icons/bring-to-front";
import { default as default215 } from "../icons/brush-cleaning";
import { default as default216 } from "../icons/brush";
import { default as default217 } from "../icons/bubbles";
import { default as default218 } from "../icons/bug-off";
import { default as default219 } from "../icons/bug";
import { default as default220 } from "../icons/bug-play";
import { default as default221 } from "../icons/building-2";
import { default as default222 } from "../icons/building";
import { default as default223 } from "../icons/bus-front";
import { default as default224 } from "../icons/bus";
import { default as default225 } from "../icons/cable-car";
import { default as default226 } from "../icons/cable";
import { default as default227 } from "../icons/cake-slice";
import { default as default228 } from "../icons/cake";
import { default as default229 } from "../icons/calculator";
import { default as default230 } from "../icons/calendar-arrow-down";
import { default as default231 } from "../icons/calendar-1";
import { default as default232 } from "../icons/calendar-check-2";
import { default as default233 } from "../icons/calendar-arrow-up";
import { default as default234 } from "../icons/calendar-check";
import { default as default235 } from "../icons/calendar-clock";
import { default as default236 } from "../icons/calendar-cog";
import { default as default237 } from "../icons/calendar-days";
import { default as default238 } from "../icons/calendar-fold";
import { default as default239 } from "../icons/calendar-heart";
import { default as default240 } from "../icons/calendar-minus-2";
import { default as default241 } from "../icons/calendar-minus";
import { default as default242 } from "../icons/calendar-off";
import { default as default243 } from "../icons/calendar-plus-2";
import { default as default244 } from "../icons/calendar-plus";
import { default as default245 } from "../icons/calendar-range";
import { default as default246 } from "../icons/calendar-search";
import { default as default247 } from "../icons/calendar-sync";
import { default as default248 } from "../icons/calendar-x-2";
import { default as default249 } from "../icons/calendar-x";
import { default as default250 } from "../icons/calendar";
import { default as default251 } from "../icons/camera-off";
import { default as default252 } from "../icons/camera";
import { default as default253 } from "../icons/candy-cane";
import { default as default254 } from "../icons/candy-off";
import { default as default255 } from "../icons/candy";
import { default as default256 } from "../icons/cannabis";
import { default as default257 } from "../icons/captions-off";
import { default as default258 } from "../icons/car-front";
import { default as default259 } from "../icons/car-taxi-front";
import { default as default260 } from "../icons/car";
import { default as default261 } from "../icons/caravan";
import { default as default262 } from "../icons/card-sim";
import { default as default263 } from "../icons/carrot";
import { default as default264 } from "../icons/case-lower";
import { default as default265 } from "../icons/case-sensitive";
import { default as default266 } from "../icons/case-upper";
import { default as default267 } from "../icons/cassette-tape";
import { default as default268 } from "../icons/cast";
import { default as default269 } from "../icons/cat";
import { default as default270 } from "../icons/castle";
import { default as default271 } from "../icons/cctv";
import { default as default272 } from "../icons/chart-bar-decreasing";
import { default as default273 } from "../icons/chart-bar-increasing";
import { default as default274 } from "../icons/chart-bar-stacked";
import { default as default275 } from "../icons/chart-column-decreasing";
import { default as default276 } from "../icons/chart-column-stacked";
import { default as default277 } from "../icons/chart-gantt";
import { default as default278 } from "../icons/chart-network";
import { default as default279 } from "../icons/chart-no-axes-column-decreasing";
import { default as default280 } from "../icons/chart-no-axes-combined";
import { default as default281 } from "../icons/chart-spline";
import { default as default282 } from "../icons/check-check";
import { default as default283 } from "../icons/check-line";
import { default as default284 } from "../icons/check";
import { default as default285 } from "../icons/chef-hat";
import { default as default286 } from "../icons/chevron-down";
import { default as default287 } from "../icons/cherry";
import { default as default288 } from "../icons/chevron-first";
import { default as default289 } from "../icons/chevron-last";
import { default as default290 } from "../icons/chevron-right";
import { default as default291 } from "../icons/chevron-left";
import { default as default292 } from "../icons/chevron-up";
import { default as default293 } from "../icons/chevrons-down-up";
import { default as default294 } from "../icons/chevrons-left-right-ellipsis";
import { default as default295 } from "../icons/chevrons-down";
import { default as default296 } from "../icons/chevrons-left-right";
import { default as default297 } from "../icons/chevrons-left";
import { default as default298 } from "../icons/chevrons-right-left";
import { default as default299 } from "../icons/chevrons-right";
import { default as default300 } from "../icons/chevrons-up-down";
import { default as default301 } from "../icons/chrome";
import { default as default302 } from "../icons/chevrons-up";
import { default as default303 } from "../icons/church";
import { default as default304 } from "../icons/cigarette-off";
import { default as default305 } from "../icons/cigarette";
import { default as default306 } from "../icons/circle-dashed";
import { default as default307 } from "../icons/circle-dot-dashed";
import { default as default308 } from "../icons/circle-dollar-sign";
import { default as default309 } from "../icons/circle-dot";
import { default as default310 } from "../icons/circle-ellipsis";
import { default as default311 } from "../icons/circle-equal";
import { default as default312 } from "../icons/circle-fading-arrow-up";
import { default as default313 } from "../icons/circle-fading-plus";
import { default as default314 } from "../icons/circle-off";
import { default as default315 } from "../icons/circle-pound-sterling";
import { default as default316 } from "../icons/circle-slash";
import { default as default317 } from "../icons/circle-small";
import { default as default318 } from "../icons/circle";
import { default as default319 } from "../icons/circuit-board";
import { default as default320 } from "../icons/citrus";
import { default as default321 } from "../icons/clapperboard";
import { default as default322 } from "../icons/clipboard-check";
import { default as default323 } from "../icons/clipboard-copy";
import { default as default324 } from "../icons/clipboard-list";
import { default as default325 } from "../icons/clipboard-minus";
import { default as default326 } from "../icons/clipboard-paste";
import { default as default327 } from "../icons/clipboard-plus";
import { default as default328 } from "../icons/clipboard-x";
import { default as default329 } from "../icons/clipboard-type";
import { default as default330 } from "../icons/clipboard";
import { default as default331 } from "../icons/clock-10";
import { default as default332 } from "../icons/clock-1";
import { default as default333 } from "../icons/clock-11";
import { default as default334 } from "../icons/clock-12";
import { default as default335 } from "../icons/clock-2";
import { default as default336 } from "../icons/clock-3";
import { default as default337 } from "../icons/clock-4";
import { default as default338 } from "../icons/clock-5";
import { default as default339 } from "../icons/clock-6";
import { default as default340 } from "../icons/clock-7";
import { default as default341 } from "../icons/clock-8";
import { default as default342 } from "../icons/clock-9";
import { default as default343 } from "../icons/clock-alert";
import { default as default344 } from "../icons/clock-arrow-down";
import { default as default345 } from "../icons/clock-arrow-up";
import { default as default346 } from "../icons/clock-fading";
import { default as default347 } from "../icons/clock-plus";
import { default as default348 } from "../icons/clock";
import { default as default349 } from "../icons/cloud-alert";
import { default as default350 } from "../icons/cloud-check";
import { default as default351 } from "../icons/cloud-cog";
import { default as default352 } from "../icons/cloud-drizzle";
import { default as default353 } from "../icons/cloud-fog";
import { default as default354 } from "../icons/cloud-hail";
import { default as default355 } from "../icons/cloud-lightning";
import { default as default356 } from "../icons/cloud-moon-rain";
import { default as default357 } from "../icons/cloud-moon";
import { default as default358 } from "../icons/cloud-off";
import { default as default359 } from "../icons/cloud-rain-wind";
import { default as default360 } from "../icons/cloud-rain";
import { default as default361 } from "../icons/cloud-snow";
import { default as default362 } from "../icons/cloud-sun-rain";
import { default as default363 } from "../icons/cloud-sun";
import { default as default364 } from "../icons/cloud";
import { default as default365 } from "../icons/cloudy";
import { default as default366 } from "../icons/club";
import { default as default367 } from "../icons/code";
import { default as default368 } from "../icons/clover";
import { default as default369 } from "../icons/codepen";
import { default as default370 } from "../icons/codesandbox";
import { default as default371 } from "../icons/coffee";
import { default as default372 } from "../icons/cog";
import { default as default373 } from "../icons/coins";
import { default as default374 } from "../icons/columns-4";
import { default as default375 } from "../icons/command";
import { default as default376 } from "../icons/combine";
import { default as default377 } from "../icons/compass";
import { default as default378 } from "../icons/computer";
import { default as default379 } from "../icons/component";
import { default as default380 } from "../icons/concierge-bell";
import { default as default381 } from "../icons/construction";
import { default as default382 } from "../icons/cone";
import { default as default383 } from "../icons/contact";
import { default as default384 } from "../icons/contrast";
import { default as default385 } from "../icons/container";
import { default as default386 } from "../icons/cookie";
import { default as default387 } from "../icons/cooking-pot";
import { default as default388 } from "../icons/copy-check";
import { default as default389 } from "../icons/copy-minus";
import { default as default390 } from "../icons/copy-plus";
import { default as default391 } from "../icons/copy-slash";
import { default as default392 } from "../icons/copy-x";
import { default as default393 } from "../icons/copy";
import { default as default394 } from "../icons/copyleft";
import { default as default395 } from "../icons/copyright";
import { default as default396 } from "../icons/corner-down-left";
import { default as default397 } from "../icons/corner-down-right";
import { default as default398 } from "../icons/corner-left-down";
import { default as default399 } from "../icons/corner-left-up";
import { default as default400 } from "../icons/corner-right-down";
import { default as default401 } from "../icons/corner-right-up";
import { default as default402 } from "../icons/corner-up-right";
import { default as default403 } from "../icons/cpu";
import { default as default404 } from "../icons/corner-up-left";
import { default as default405 } from "../icons/creative-commons";
import { default as default406 } from "../icons/croissant";
import { default as default407 } from "../icons/credit-card";
import { default as default408 } from "../icons/crop";
import { default as default409 } from "../icons/cross";
import { default as default410 } from "../icons/crosshair";
import { default as default411 } from "../icons/crown";
import { default as default412 } from "../icons/cuboid";
import { default as default413 } from "../icons/cup-soda";
import { default as default414 } from "../icons/currency";
import { default as default415 } from "../icons/cylinder";
import { default as default416 } from "../icons/dam";
import { default as default417 } from "../icons/database-backup";
import { default as default418 } from "../icons/database-zap";
import { default as default419 } from "../icons/database";
import { default as default420 } from "../icons/decimals-arrow-left";
import { default as default421 } from "../icons/delete";
import { default as default422 } from "../icons/decimals-arrow-right";
import { default as default423 } from "../icons/dessert";
import { default as default424 } from "../icons/diameter";
import { default as default425 } from "../icons/diamond-minus";
import { default as default426 } from "../icons/diamond-plus";
import { default as default427 } from "../icons/diamond";
import { default as default428 } from "../icons/dice-1";
import { default as default429 } from "../icons/dice-2";
import { default as default430 } from "../icons/dice-3";
import { default as default431 } from "../icons/dice-4";
import { default as default432 } from "../icons/dice-5";
import { default as default433 } from "../icons/dice-6";
import { default as default434 } from "../icons/dices";
import { default as default435 } from "../icons/diff";
import { default as default436 } from "../icons/disc-2";
import { default as default437 } from "../icons/disc-3";
import { default as default438 } from "../icons/disc-album";
import { default as default439 } from "../icons/disc";
import { default as default440 } from "../icons/divide";
import { default as default441 } from "../icons/dna-off";
import { default as default442 } from "../icons/dna";
import { default as default443 } from "../icons/dock";
import { default as default444 } from "../icons/dog";
import { default as default445 } from "../icons/dollar-sign";
import { default as default446 } from "../icons/donut";
import { default as default447 } from "../icons/door-closed-locked";
import { default as default448 } from "../icons/door-closed";
import { default as default449 } from "../icons/door-open";
import { default as default450 } from "../icons/dot";
import { default as default451 } from "../icons/download";
import { default as default452 } from "../icons/drafting-compass";
import { default as default453 } from "../icons/drama";
import { default as default454 } from "../icons/dribbble";
import { default as default455 } from "../icons/drill";
import { default as default456 } from "../icons/droplet-off";
import { default as default457 } from "../icons/droplet";
import { default as default458 } from "../icons/droplets";
import { default as default459 } from "../icons/drum";
import { default as default460 } from "../icons/drumstick";
import { default as default461 } from "../icons/dumbbell";
import { default as default462 } from "../icons/ear-off";
import { default as default463 } from "../icons/ear";
import { default as default464 } from "../icons/earth-lock";
import { default as default465 } from "../icons/eclipse";
import { default as default466 } from "../icons/egg-fried";
import { default as default467 } from "../icons/egg-off";
import { default as default468 } from "../icons/egg";
import { default as default469 } from "../icons/equal-approximately";
import { default as default470 } from "../icons/equal";
import { default as default471 } from "../icons/equal-not";
import { default as default472 } from "../icons/eraser";
import { default as default473 } from "../icons/ethernet-port";
import { default as default474 } from "../icons/euro";
import { default as default475 } from "../icons/expand";
import { default as default476 } from "../icons/external-link";
import { default as default477 } from "../icons/eye-closed";
import { default as default478 } from "../icons/eye-off";
import { default as default479 } from "../icons/eye";
import { default as default480 } from "../icons/facebook";
import { default as default481 } from "../icons/factory";
import { default as default482 } from "../icons/fan";
import { default as default483 } from "../icons/fast-forward";
import { default as default484 } from "../icons/feather";
import { default as default485 } from "../icons/fence";
import { default as default486 } from "../icons/ferris-wheel";
import { default as default487 } from "../icons/figma";
import { default as default488 } from "../icons/file-archive";
import { default as default489 } from "../icons/file-audio-2";
import { default as default490 } from "../icons/file-audio";
import { default as default491 } from "../icons/file-badge-2";
import { default as default492 } from "../icons/file-badge";
import { default as default493 } from "../icons/file-box";
import { default as default494 } from "../icons/file-check-2";
import { default as default495 } from "../icons/file-check";
import { default as default496 } from "../icons/file-clock";
import { default as default497 } from "../icons/file-code-2";
import { default as default498 } from "../icons/file-code";
import { default as default499 } from "../icons/file-diff";
import { default as default500 } from "../icons/file-digit";
import { default as default501 } from "../icons/file-down";
import { default as default502 } from "../icons/file-heart";
import { default as default503 } from "../icons/file-image";
import { default as default504 } from "../icons/file-input";
import { default as default505 } from "../icons/file-json-2";
import { default as default506 } from "../icons/file-json";
import { default as default507 } from "../icons/file-key-2";
import { default as default508 } from "../icons/file-key";
import { default as default509 } from "../icons/file-lock-2";
import { default as default510 } from "../icons/file-lock";
import { default as default511 } from "../icons/file-minus-2";
import { default as default512 } from "../icons/file-minus";
import { default as default513 } from "../icons/file-music";
import { default as default514 } from "../icons/file-output";
import { default as default515 } from "../icons/file-plus-2";
import { default as default516 } from "../icons/file-plus";
import { default as default517 } from "../icons/file-question";
import { default as default518 } from "../icons/file-scan";
import { default as default519 } from "../icons/file-search-2";
import { default as default520 } from "../icons/file-search";
import { default as default521 } from "../icons/file-sliders";
import { default as default522 } from "../icons/file-spreadsheet";
import { default as default523 } from "../icons/file-stack";
import { default as default524 } from "../icons/file-symlink";
import { default as default525 } from "../icons/file-text";
import { default as default526 } from "../icons/file-terminal";
import { default as default527 } from "../icons/file-type-2";
import { default as default528 } from "../icons/file-type";
import { default as default529 } from "../icons/file-up";
import { default as default530 } from "../icons/file-user";
import { default as default531 } from "../icons/file-video-2";
import { default as default532 } from "../icons/file-video";
import { default as default533 } from "../icons/file-volume-2";
import { default as default534 } from "../icons/file-volume";
import { default as default535 } from "../icons/file-warning";
import { default as default536 } from "../icons/file-x-2";
import { default as default537 } from "../icons/file-x";
import { default as default538 } from "../icons/file";
import { default as default539 } from "../icons/files";
import { default as default540 } from "../icons/film";
import { default as default541 } from "../icons/fingerprint";
import { default as default542 } from "../icons/fire-extinguisher";
import { default as default543 } from "../icons/fish-off";
import { default as default544 } from "../icons/fish-symbol";
import { default as default545 } from "../icons/fish";
import { default as default546 } from "../icons/flag-off";
import { default as default547 } from "../icons/flag-triangle-left";
import { default as default548 } from "../icons/flag-triangle-right";
import { default as default549 } from "../icons/flag";
import { default as default550 } from "../icons/flame-kindling";
import { default as default551 } from "../icons/flame";
import { default as default552 } from "../icons/flashlight-off";
import { default as default553 } from "../icons/flashlight";
import { default as default554 } from "../icons/flask-conical-off";
import { default as default555 } from "../icons/flask-conical";
import { default as default556 } from "../icons/flask-round";
import { default as default557 } from "../icons/flip-horizontal";
import { default as default558 } from "../icons/flip-horizontal-2";
import { default as default559 } from "../icons/flip-vertical-2";
import { default as default560 } from "../icons/flip-vertical";
import { default as default561 } from "../icons/flower-2";
import { default as default562 } from "../icons/flower";
import { default as default563 } from "../icons/focus";
import { default as default564 } from "../icons/fold-horizontal";
import { default as default565 } from "../icons/fold-vertical";
import { default as default566 } from "../icons/folder-archive";
import { default as default567 } from "../icons/folder-check";
import { default as default568 } from "../icons/folder-clock";
import { default as default569 } from "../icons/folder-closed";
import { default as default570 } from "../icons/folder-code";
import { default as default571 } from "../icons/folder-dot";
import { default as default572 } from "../icons/folder-down";
import { default as default573 } from "../icons/folder-git-2";
import { default as default574 } from "../icons/folder-git";
import { default as default575 } from "../icons/folder-heart";
import { default as default576 } from "../icons/folder-input";
import { default as default577 } from "../icons/folder-kanban";
import { default as default578 } from "../icons/folder-key";
import { default as default579 } from "../icons/folder-lock";
import { default as default580 } from "../icons/folder-minus";
import { default as default581 } from "../icons/folder-open-dot";
import { default as default582 } from "../icons/folder-open";
import { default as default583 } from "../icons/folder-output";
import { default as default584 } from "../icons/folder-plus";
import { default as default585 } from "../icons/folder-root";
import { default as default586 } from "../icons/folder-search-2";
import { default as default587 } from "../icons/folder-search";
import { default as default588 } from "../icons/folder-symlink";
import { default as default589 } from "../icons/folder-sync";
import { default as default590 } from "../icons/folder-tree";
import { default as default591 } from "../icons/folder-up";
import { default as default592 } from "../icons/folder-x";
import { default as default593 } from "../icons/folder";
import { default as default594 } from "../icons/folders";
import { default as default595 } from "../icons/forklift";
import { default as default596 } from "../icons/footprints";
import { default as default597 } from "../icons/forward";
import { default as default598 } from "../icons/framer";
import { default as default599 } from "../icons/frame";
import { default as default600 } from "../icons/fuel";
import { default as default601 } from "../icons/frown";
import { default as default602 } from "../icons/fullscreen";
import { default as default603 } from "../icons/funnel-plus";
import { default as default604 } from "../icons/gallery-horizontal-end";
import { default as default605 } from "../icons/gallery-horizontal";
import { default as default606 } from "../icons/gallery-thumbnails";
import { default as default607 } from "../icons/gallery-vertical-end";
import { default as default608 } from "../icons/gallery-vertical";
import { default as default609 } from "../icons/gamepad-2";
import { default as default610 } from "../icons/gamepad";
import { default as default611 } from "../icons/gauge";
import { default as default612 } from "../icons/gavel";
import { default as default613 } from "../icons/gem";
import { default as default614 } from "../icons/ghost";
import { default as default615 } from "../icons/gift";
import { default as default616 } from "../icons/git-branch";
import { default as default617 } from "../icons/git-branch-plus";
import { default as default618 } from "../icons/git-commit-vertical";
import { default as default619 } from "../icons/git-compare-arrows";
import { default as default620 } from "../icons/git-compare";
import { default as default621 } from "../icons/git-fork";
import { default as default622 } from "../icons/git-graph";
import { default as default623 } from "../icons/git-merge";
import { default as default624 } from "../icons/git-pull-request-arrow";
import { default as default625 } from "../icons/git-pull-request-closed";
import { default as default626 } from "../icons/git-pull-request-create";
import { default as default627 } from "../icons/git-pull-request-create-arrow";
import { default as default628 } from "../icons/git-pull-request-draft";
import { default as default629 } from "../icons/git-pull-request";
import { default as default630 } from "../icons/github";
import { default as default631 } from "../icons/gitlab";
import { default as default632 } from "../icons/glass-water";
import { default as default633 } from "../icons/glasses";
import { default as default634 } from "../icons/globe-lock";
import { default as default635 } from "../icons/globe";
import { default as default636 } from "../icons/goal";
import { default as default637 } from "../icons/gpu";
import { default as default638 } from "../icons/grab";
import { default as default639 } from "../icons/graduation-cap";
import { default as default640 } from "../icons/grape";
import { default as default641 } from "../icons/grid-3x2";
import { default as default642 } from "../icons/grip-horizontal";
import { default as default643 } from "../icons/grip-vertical";
import { default as default644 } from "../icons/grip";
import { default as default645 } from "../icons/group";
import { default as default646 } from "../icons/guitar";
import { default as default647 } from "../icons/ham";
import { default as default648 } from "../icons/hamburger";
import { default as default649 } from "../icons/hammer";
import { default as default650 } from "../icons/hand-coins";
import { default as default651 } from "../icons/hand-heart";
import { default as default652 } from "../icons/hand-metal";
import { default as default653 } from "../icons/hand-platter";
import { default as default654 } from "../icons/hand";
import { default as default655 } from "../icons/hard-drive-download";
import { default as default656 } from "../icons/handshake";
import { default as default657 } from "../icons/hard-drive-upload";
import { default as default658 } from "../icons/hard-drive";
import { default as default659 } from "../icons/hard-hat";
import { default as default660 } from "../icons/hash";
import { default as default661 } from "../icons/hdmi-port";
import { default as default662 } from "../icons/haze";
import { default as default663 } from "../icons/heading-1";
import { default as default664 } from "../icons/heading-2";
import { default as default665 } from "../icons/heading-3";
import { default as default666 } from "../icons/heading-4";
import { default as default667 } from "../icons/heading-5";
import { default as default668 } from "../icons/heading-6";
import { default as default669 } from "../icons/heading";
import { default as default670 } from "../icons/headphone-off";
import { default as default671 } from "../icons/headphones";
import { default as default672 } from "../icons/headset";
import { default as default673 } from "../icons/heart-crack";
import { default as default674 } from "../icons/heart-handshake";
import { default as default675 } from "../icons/heart-minus";
import { default as default676 } from "../icons/heart-off";
import { default as default677 } from "../icons/heart-plus";
import { default as default678 } from "../icons/heart-pulse";
import { default as default679 } from "../icons/heart";
import { default as default680 } from "../icons/heater";
import { default as default681 } from "../icons/hexagon";
import { default as default682 } from "../icons/highlighter";
import { default as default683 } from "../icons/history";
import { default as default684 } from "../icons/hop-off";
import { default as default685 } from "../icons/hop";
import { default as default686 } from "../icons/hospital";
import { default as default687 } from "../icons/hotel";
import { default as default688 } from "../icons/hourglass";
import { default as default689 } from "../icons/house-plug";
import { default as default690 } from "../icons/house-plus";
import { default as default691 } from "../icons/house-wifi";
import { default as default692 } from "../icons/id-card";
import { default as default693 } from "../icons/id-card-lanyard";
import { default as default694 } from "../icons/image-down";
import { default as default695 } from "../icons/image-minus";
import { default as default696 } from "../icons/image-off";
import { default as default697 } from "../icons/image-play";
import { default as default698 } from "../icons/image-plus";
import { default as default699 } from "../icons/image-up";
import { default as default700 } from "../icons/image-upscale";
import { default as default701 } from "../icons/image";
import { default as default702 } from "../icons/images";
import { default as default703 } from "../icons/inbox";
import { default as default704 } from "../icons/import";
import { default as default705 } from "../icons/indian-rupee";
import { default as default706 } from "../icons/infinity";
import { default as default707 } from "../icons/info";
import { default as default708 } from "../icons/inspection-panel";
import { default as default709 } from "../icons/instagram";
import { default as default710 } from "../icons/italic";
import { default as default711 } from "../icons/iteration-ccw";
import { default as default712 } from "../icons/iteration-cw";
import { default as default713 } from "../icons/japanese-yen";
import { default as default714 } from "../icons/joystick";
import { default as default715 } from "../icons/kanban";
import { default as default716 } from "../icons/key-round";
import { default as default717 } from "../icons/key-square";
import { default as default718 } from "../icons/key";
import { default as default719 } from "../icons/keyboard-music";
import { default as default720 } from "../icons/keyboard-off";
import { default as default721 } from "../icons/lamp-ceiling";
import { default as default722 } from "../icons/keyboard";
import { default as default723 } from "../icons/lamp-desk";
import { default as default724 } from "../icons/lamp-floor";
import { default as default725 } from "../icons/lamp-wall-down";
import { default as default726 } from "../icons/lamp-wall-up";
import { default as default727 } from "../icons/lamp";
import { default as default728 } from "../icons/land-plot";
import { default as default729 } from "../icons/landmark";
import { default as default730 } from "../icons/languages";
import { default as default731 } from "../icons/laptop-minimal-check";
import { default as default732 } from "../icons/laptop";
import { default as default733 } from "../icons/lasso-select";
import { default as default734 } from "../icons/lasso";
import { default as default735 } from "../icons/laugh";
import { default as default736 } from "../icons/layers-2";
import { default as default737 } from "../icons/layout-grid";
import { default as default738 } from "../icons/layout-dashboard";
import { default as default739 } from "../icons/layout-list";
import { default as default740 } from "../icons/layout-panel-top";
import { default as default741 } from "../icons/layout-panel-left";
import { default as default742 } from "../icons/layout-template";
import { default as default743 } from "../icons/leaf";
import { default as default744 } from "../icons/lectern";
import { default as default745 } from "../icons/leafy-green";
import { default as default746 } from "../icons/letter-text";
import { default as default747 } from "../icons/library-big";
import { default as default748 } from "../icons/library";
import { default as default749 } from "../icons/life-buoy";
import { default as default750 } from "../icons/ligature";
import { default as default751 } from "../icons/lightbulb-off";
import { default as default752 } from "../icons/lightbulb";
import { default as default753 } from "../icons/link-2-off";
import { default as default754 } from "../icons/link-2";
import { default as default755 } from "../icons/link";
import { default as default756 } from "../icons/linkedin";
import { default as default757 } from "../icons/list-check";
import { default as default758 } from "../icons/list-checks";
import { default as default759 } from "../icons/list-end";
import { default as default760 } from "../icons/list-collapse";
import { default as default761 } from "../icons/list-filter-plus";
import { default as default762 } from "../icons/list-filter";
import { default as default763 } from "../icons/list-minus";
import { default as default764 } from "../icons/list-music";
import { default as default765 } from "../icons/list-ordered";
import { default as default766 } from "../icons/list-plus";
import { default as default767 } from "../icons/list-restart";
import { default as default768 } from "../icons/list-start";
import { default as default769 } from "../icons/list-todo";
import { default as default770 } from "../icons/list-tree";
import { default as default771 } from "../icons/list-video";
import { default as default772 } from "../icons/list-x";
import { default as default773 } from "../icons/list";
import { default as default774 } from "../icons/loader-pinwheel";
import { default as default775 } from "../icons/loader";
import { default as default776 } from "../icons/locate-fixed";
import { default as default777 } from "../icons/locate-off";
import { default as default778 } from "../icons/locate";
import { default as default779 } from "../icons/location-edit";
import { default as default780 } from "../icons/lock-keyhole";
import { default as default781 } from "../icons/lock";
import { default as default782 } from "../icons/log-in";
import { default as default783 } from "../icons/log-out";
import { default as default784 } from "../icons/logs";
import { default as default785 } from "../icons/lollipop";
import { default as default786 } from "../icons/luggage";
import { default as default787 } from "../icons/magnet";
import { default as default788 } from "../icons/mail-check";
import { default as default789 } from "../icons/mail-minus";
import { default as default790 } from "../icons/mail-open";
import { default as default791 } from "../icons/mail-plus";
import { default as default792 } from "../icons/mail-question";
import { default as default793 } from "../icons/mail-search";
import { default as default794 } from "../icons/mail-warning";
import { default as default795 } from "../icons/mail-x";
import { default as default796 } from "../icons/mail";
import { default as default797 } from "../icons/mailbox";
import { default as default798 } from "../icons/mails";
import { default as default799 } from "../icons/map-pin-check-inside";
import { default as default800 } from "../icons/map-pin-check";
import { default as default801 } from "../icons/map-pin-house";
import { default as default802 } from "../icons/map-pin-minus-inside";
import { default as default803 } from "../icons/map-pin-minus";
import { default as default804 } from "../icons/map-pin-off";
import { default as default805 } from "../icons/map-pin-plus-inside";
import { default as default806 } from "../icons/map-pin-plus";
import { default as default807 } from "../icons/map-pin-x-inside";
import { default as default808 } from "../icons/map-pin-x";
import { default as default809 } from "../icons/map-pin";
import { default as default810 } from "../icons/map-pinned";
import { default as default811 } from "../icons/map-plus";
import { default as default812 } from "../icons/map";
import { default as default813 } from "../icons/mars-stroke";
import { default as default814 } from "../icons/mars";
import { default as default815 } from "../icons/martini";
import { default as default816 } from "../icons/maximize-2";
import { default as default817 } from "../icons/maximize";
import { default as default818 } from "../icons/medal";
import { default as default819 } from "../icons/megaphone-off";
import { default as default820 } from "../icons/megaphone";
import { default as default821 } from "../icons/meh";
import { default as default822 } from "../icons/memory-stick";
import { default as default823 } from "../icons/menu";
import { default as default824 } from "../icons/merge";
import { default as default825 } from "../icons/message-circle-heart";
import { default as default826 } from "../icons/message-circle-code";
import { default as default827 } from "../icons/message-circle-more";
import { default as default828 } from "../icons/message-circle-dashed";
import { default as default829 } from "../icons/message-circle-off";
import { default as default830 } from "../icons/message-circle-plus";
import { default as default831 } from "../icons/message-circle-question";
import { default as default832 } from "../icons/message-circle-reply";
import { default as default833 } from "../icons/message-circle-warning";
import { default as default834 } from "../icons/message-circle-x";
import { default as default835 } from "../icons/message-circle";
import { default as default836 } from "../icons/message-square-code";
import { default as default837 } from "../icons/message-square-dashed";
import { default as default838 } from "../icons/message-square-diff";
import { default as default839 } from "../icons/message-square-dot";
import { default as default840 } from "../icons/message-square-heart";
import { default as default841 } from "../icons/message-square-lock";
import { default as default842 } from "../icons/message-square-more";
import { default as default843 } from "../icons/message-square-off";
import { default as default844 } from "../icons/message-square-quote";
import { default as default845 } from "../icons/message-square-plus";
import { default as default846 } from "../icons/message-square-reply";
import { default as default847 } from "../icons/message-square-share";
import { default as default848 } from "../icons/message-square-text";
import { default as default849 } from "../icons/message-square-warning";
import { default as default850 } from "../icons/message-square-x";
import { default as default851 } from "../icons/message-square";
import { default as default852 } from "../icons/messages-square";
import { default as default853 } from "../icons/mic-off";
import { default as default854 } from "../icons/mic";
import { default as default855 } from "../icons/microchip";
import { default as default856 } from "../icons/microscope";
import { default as default857 } from "../icons/microwave";
import { default as default858 } from "../icons/milestone";
import { default as default859 } from "../icons/milk";
import { default as default860 } from "../icons/milk-off";
import { default as default861 } from "../icons/minimize-2";
import { default as default862 } from "../icons/minimize";
import { default as default863 } from "../icons/minus";
import { default as default864 } from "../icons/monitor-check";
import { default as default865 } from "../icons/monitor-dot";
import { default as default866 } from "../icons/monitor-cog";
import { default as default867 } from "../icons/monitor-down";
import { default as default868 } from "../icons/monitor-off";
import { default as default869 } from "../icons/monitor-pause";
import { default as default870 } from "../icons/monitor-play";
import { default as default871 } from "../icons/monitor-smartphone";
import { default as default872 } from "../icons/monitor-speaker";
import { default as default873 } from "../icons/monitor-stop";
import { default as default874 } from "../icons/monitor-up";
import { default as default875 } from "../icons/monitor-x";
import { default as default876 } from "../icons/monitor";
import { default as default877 } from "../icons/moon-star";
import { default as default878 } from "../icons/moon";
import { default as default879 } from "../icons/mountain-snow";
import { default as default880 } from "../icons/mountain";
import { default as default881 } from "../icons/mouse-off";
import { default as default882 } from "../icons/mouse-pointer-2";
import { default as default883 } from "../icons/mouse-pointer-ban";
import { default as default884 } from "../icons/mouse-pointer-click";
import { default as default885 } from "../icons/mouse-pointer";
import { default as default886 } from "../icons/mouse";
import { default as default887 } from "../icons/move-diagonal-2";
import { default as default888 } from "../icons/move-diagonal";
import { default as default889 } from "../icons/move-down-left";
import { default as default890 } from "../icons/move-down-right";
import { default as default891 } from "../icons/move-down";
import { default as default892 } from "../icons/move-left";
import { default as default893 } from "../icons/move-horizontal";
import { default as default894 } from "../icons/move-up-left";
import { default as default895 } from "../icons/move-right";
import { default as default896 } from "../icons/move-up-right";
import { default as default897 } from "../icons/move-up";
import { default as default898 } from "../icons/move";
import { default as default899 } from "../icons/move-vertical";
import { default as default900 } from "../icons/music-2";
import { default as default901 } from "../icons/music-3";
import { default as default902 } from "../icons/music-4";
import { default as default903 } from "../icons/music";
import { default as default904 } from "../icons/navigation-2-off";
import { default as default905 } from "../icons/navigation-2";
import { default as default906 } from "../icons/navigation-off";
import { default as default907 } from "../icons/navigation";
import { default as default908 } from "../icons/network";
import { default as default909 } from "../icons/newspaper";
import { default as default910 } from "../icons/nfc";
import { default as default911 } from "../icons/non-binary";
import { default as default912 } from "../icons/notebook-pen";
import { default as default913 } from "../icons/notebook-tabs";
import { default as default914 } from "../icons/notebook-text";
import { default as default915 } from "../icons/notepad-text-dashed";
import { default as default916 } from "../icons/notebook";
import { default as default917 } from "../icons/notepad-text";
import { default as default918 } from "../icons/nut-off";
import { default as default919 } from "../icons/octagon-minus";
import { default as default920 } from "../icons/nut";
import { default as default921 } from "../icons/octagon";
import { default as default922 } from "../icons/option";
import { default as default923 } from "../icons/omega";
import { default as default924 } from "../icons/orbit";
import { default as default925 } from "../icons/origami";
import { default as default926 } from "../icons/package-2";
import { default as default927 } from "../icons/package-check";
import { default as default928 } from "../icons/package-minus";
import { default as default929 } from "../icons/package-open";
import { default as default930 } from "../icons/package-plus";
import { default as default931 } from "../icons/package-x";
import { default as default932 } from "../icons/package";
import { default as default933 } from "../icons/package-search";
import { default as default934 } from "../icons/paint-bucket";
import { default as default935 } from "../icons/paintbrush";
import { default as default936 } from "../icons/paint-roller";
import { default as default937 } from "../icons/palette";
import { default as default938 } from "../icons/panda";
import { default as default939 } from "../icons/panel-bottom-close";
import { default as default940 } from "../icons/panel-bottom-open";
import { default as default941 } from "../icons/panel-bottom";
import { default as default942 } from "../icons/panel-right-close";
import { default as default943 } from "../icons/panel-right-open";
import { default as default944 } from "../icons/panel-right";
import { default as default945 } from "../icons/panel-top-close";
import { default as default946 } from "../icons/panel-top";
import { default as default947 } from "../icons/panel-top-open";
import { default as default948 } from "../icons/panels-left-bottom";
import { default as default949 } from "../icons/panels-right-bottom";
import { default as default950 } from "../icons/paperclip";
import { default as default951 } from "../icons/parentheses";
import { default as default952 } from "../icons/parking-meter";
import { default as default953 } from "../icons/party-popper";
import { default as default954 } from "../icons/pause";
import { default as default955 } from "../icons/pc-case";
import { default as default956 } from "../icons/pen-off";
import { default as default957 } from "../icons/paw-print";
import { default as default958 } from "../icons/pen-tool";
import { default as default959 } from "../icons/pencil-line";
import { default as default960 } from "../icons/pencil-off";
import { default as default961 } from "../icons/pencil-ruler";
import { default as default962 } from "../icons/pencil";
import { default as default963 } from "../icons/pentagon";
import { default as default964 } from "../icons/percent";
import { default as default965 } from "../icons/person-standing";
import { default as default966 } from "../icons/philippine-peso";
import { default as default967 } from "../icons/phone-call";
import { default as default968 } from "../icons/phone-forwarded";
import { default as default969 } from "../icons/phone-incoming";
import { default as default970 } from "../icons/phone-missed";
import { default as default971 } from "../icons/phone-off";
import { default as default972 } from "../icons/phone";
import { default as default973 } from "../icons/phone-outgoing";
import { default as default974 } from "../icons/pi";
import { default as default975 } from "../icons/piano";
import { default as default976 } from "../icons/pickaxe";
import { default as default977 } from "../icons/picture-in-picture-2";
import { default as default978 } from "../icons/picture-in-picture";
import { default as default979 } from "../icons/piggy-bank";
import { default as default980 } from "../icons/pilcrow-left";
import { default as default981 } from "../icons/pilcrow-right";
import { default as default982 } from "../icons/pilcrow";
import { default as default983 } from "../icons/pill-bottle";
import { default as default984 } from "../icons/pill";
import { default as default985 } from "../icons/pin-off";
import { default as default986 } from "../icons/pin";
import { default as default987 } from "../icons/pipette";
import { default as default988 } from "../icons/pizza";
import { default as default989 } from "../icons/plane-landing";
import { default as default990 } from "../icons/plane";
import { default as default991 } from "../icons/plane-takeoff";
import { default as default992 } from "../icons/play";
import { default as default993 } from "../icons/plug";
import { default as default994 } from "../icons/plug-2";
import { default as default995 } from "../icons/plus";
import { default as default996 } from "../icons/pocket";
import { default as default997 } from "../icons/pocket-knife";
import { default as default998 } from "../icons/podcast";
import { default as default999 } from "../icons/pointer-off";
import { default as default1000 } from "../icons/pointer";
import { default as default1001 } from "../icons/popcorn";
import { default as default1002 } from "../icons/popsicle";
import { default as default1003 } from "../icons/pound-sterling";
import { default as default1004 } from "../icons/power-off";
import { default as default1005 } from "../icons/power";
import { default as default1006 } from "../icons/presentation";
import { default as default1007 } from "../icons/printer";
import { default as default1008 } from "../icons/printer-check";
import { default as default1009 } from "../icons/projector";
import { default as default1010 } from "../icons/puzzle";
import { default as default1011 } from "../icons/proportions";
import { default as default1012 } from "../icons/pyramid";
import { default as default1013 } from "../icons/qr-code";
import { default as default1014 } from "../icons/quote";
import { default as default1015 } from "../icons/rabbit";
import { default as default1016 } from "../icons/radar";
import { default as default1017 } from "../icons/radiation";
import { default as default1018 } from "../icons/radical";
import { default as default1019 } from "../icons/radio-receiver";
import { default as default1020 } from "../icons/radio-tower";
import { default as default1021 } from "../icons/radio";
import { default as default1022 } from "../icons/radius";
import { default as default1023 } from "../icons/rail-symbol";
import { default as default1024 } from "../icons/rainbow";
import { default as default1025 } from "../icons/rat";
import { default as default1026 } from "../icons/ratio";
import { default as default1027 } from "../icons/receipt-cent";
import { default as default1028 } from "../icons/receipt-euro";
import { default as default1029 } from "../icons/receipt-indian-rupee";
import { default as default1030 } from "../icons/receipt-japanese-yen";
import { default as default1031 } from "../icons/receipt-pound-sterling";
import { default as default1032 } from "../icons/receipt-russian-ruble";
import { default as default1033 } from "../icons/receipt-swiss-franc";
import { default as default1034 } from "../icons/receipt-text";
import { default as default1035 } from "../icons/receipt";
import { default as default1036 } from "../icons/rectangle-goggles";
import { default as default1037 } from "../icons/rectangle-horizontal";
import { default as default1038 } from "../icons/rectangle-vertical";
import { default as default1039 } from "../icons/recycle";
import { default as default1040 } from "../icons/redo-2";
import { default as default1041 } from "../icons/redo-dot";
import { default as default1042 } from "../icons/refresh-ccw-dot";
import { default as default1043 } from "../icons/redo";
import { default as default1044 } from "../icons/refresh-ccw";
import { default as default1045 } from "../icons/refresh-cw-off";
import { default as default1046 } from "../icons/refresh-cw";
import { default as default1047 } from "../icons/refrigerator";
import { default as default1048 } from "../icons/regex";
import { default as default1049 } from "../icons/repeat-1";
import { default as default1050 } from "../icons/repeat-2";
import { default as default1051 } from "../icons/remove-formatting";
import { default as default1052 } from "../icons/repeat";
import { default as default1053 } from "../icons/replace-all";
import { default as default1054 } from "../icons/replace";
import { default as default1055 } from "../icons/reply-all";
import { default as default1056 } from "../icons/reply";
import { default as default1057 } from "../icons/rewind";
import { default as default1058 } from "../icons/ribbon";
import { default as default1059 } from "../icons/rocket";
import { default as default1060 } from "../icons/rocking-chair";
import { default as default1061 } from "../icons/rotate-ccw-key";
import { default as default1062 } from "../icons/roller-coaster";
import { default as default1063 } from "../icons/rotate-ccw-square";
import { default as default1064 } from "../icons/rotate-ccw";
import { default as default1065 } from "../icons/rotate-cw-square";
import { default as default1066 } from "../icons/rotate-cw";
import { default as default1067 } from "../icons/route-off";
import { default as default1068 } from "../icons/route";
import { default as default1069 } from "../icons/router";
import { default as default1070 } from "../icons/rows-4";
import { default as default1071 } from "../icons/rss";
import { default as default1072 } from "../icons/ruler-dimension-line";
import { default as default1073 } from "../icons/ruler";
import { default as default1074 } from "../icons/russian-ruble";
import { default as default1075 } from "../icons/sailboat";
import { default as default1076 } from "../icons/salad";
import { default as default1077 } from "../icons/satellite-dish";
import { default as default1078 } from "../icons/sandwich";
import { default as default1079 } from "../icons/satellite";
import { default as default1080 } from "../icons/saudi-riyal";
import { default as default1081 } from "../icons/save-all";
import { default as default1082 } from "../icons/save-off";
import { default as default1083 } from "../icons/save";
import { default as default1084 } from "../icons/scale";
import { default as default1085 } from "../icons/scaling";
import { default as default1086 } from "../icons/scan-barcode";
import { default as default1087 } from "../icons/scan-eye";
import { default as default1088 } from "../icons/scan-face";
import { default as default1089 } from "../icons/scan-qr-code";
import { default as default1090 } from "../icons/scan-heart";
import { default as default1091 } from "../icons/scan-line";
import { default as default1092 } from "../icons/scan-search";
import { default as default1093 } from "../icons/scan-text";
import { default as default1094 } from "../icons/scan";
import { default as default1095 } from "../icons/school";
import { default as default1096 } from "../icons/scissors-line-dashed";
import { default as default1097 } from "../icons/scissors";
import { default as default1098 } from "../icons/screen-share-off";
import { default as default1099 } from "../icons/screen-share";
import { default as default1100 } from "../icons/scroll-text";
import { default as default1101 } from "../icons/scroll";
import { default as default1102 } from "../icons/search-check";
import { default as default1103 } from "../icons/search-code";
import { default as default1104 } from "../icons/search-slash";
import { default as default1105 } from "../icons/search-x";
import { default as default1106 } from "../icons/search";
import { default as default1107 } from "../icons/section";
import { default as default1108 } from "../icons/send-to-back";
import { default as default1109 } from "../icons/send";
import { default as default1110 } from "../icons/separator-horizontal";
import { default as default1111 } from "../icons/separator-vertical";
import { default as default1112 } from "../icons/server-cog";
import { default as default1113 } from "../icons/server-crash";
import { default as default1114 } from "../icons/server-off";
import { default as default1115 } from "../icons/server";
import { default as default1116 } from "../icons/settings-2";
import { default as default1117 } from "../icons/settings";
import { default as default1118 } from "../icons/shapes";
import { default as default1119 } from "../icons/share-2";
import { default as default1120 } from "../icons/share";
import { default as default1121 } from "../icons/sheet";
import { default as default1122 } from "../icons/shell";
import { default as default1123 } from "../icons/shield-alert";
import { default as default1124 } from "../icons/shield-ban";
import { default as default1125 } from "../icons/shield-check";
import { default as default1126 } from "../icons/shield-ellipsis";
import { default as default1127 } from "../icons/shield-half";
import { default as default1128 } from "../icons/shield-off";
import { default as default1129 } from "../icons/shield-minus";
import { default as default1130 } from "../icons/shield-plus";
import { default as default1131 } from "../icons/shield-question";
import { default as default1132 } from "../icons/shield-user";
import { default as default1133 } from "../icons/shield";
import { default as default1134 } from "../icons/ship-wheel";
import { default as default1135 } from "../icons/shirt";
import { default as default1136 } from "../icons/ship";
import { default as default1137 } from "../icons/shopping-bag";
import { default as default1138 } from "../icons/shopping-basket";
import { default as default1139 } from "../icons/shovel";
import { default as default1140 } from "../icons/shopping-cart";
import { default as default1141 } from "../icons/shower-head";
import { default as default1142 } from "../icons/shredder";
import { default as default1143 } from "../icons/shrimp";
import { default as default1144 } from "../icons/shrink";
import { default as default1145 } from "../icons/shuffle";
import { default as default1146 } from "../icons/shrub";
import { default as default1147 } from "../icons/sigma";
import { default as default1148 } from "../icons/signal-high";
import { default as default1149 } from "../icons/signal-medium";
import { default as default1150 } from "../icons/signal-low";
import { default as default1151 } from "../icons/signal-zero";
import { default as default1152 } from "../icons/signal";
import { default as default1153 } from "../icons/signature";
import { default as default1154 } from "../icons/signpost-big";
import { default as default1155 } from "../icons/signpost";
import { default as default1156 } from "../icons/siren";
import { default as default1157 } from "../icons/skip-forward";
import { default as default1158 } from "../icons/skip-back";
import { default as default1159 } from "../icons/skull";
import { default as default1160 } from "../icons/slack";
import { default as default1161 } from "../icons/slash";
import { default as default1162 } from "../icons/slice";
import { default as default1163 } from "../icons/smartphone-charging";
import { default as default1164 } from "../icons/sliders-horizontal";
import { default as default1165 } from "../icons/smartphone-nfc";
import { default as default1166 } from "../icons/smartphone";
import { default as default1167 } from "../icons/smile-plus";
import { default as default1168 } from "../icons/smile";
import { default as default1169 } from "../icons/snail";
import { default as default1170 } from "../icons/snowflake";
import { default as default1171 } from "../icons/soap-dispenser-droplet";
import { default as default1172 } from "../icons/sofa";
import { default as default1173 } from "../icons/soup";
import { default as default1174 } from "../icons/space";
import { default as default1175 } from "../icons/spade";
import { default as default1176 } from "../icons/sparkle";
import { default as default1177 } from "../icons/speaker";
import { default as default1178 } from "../icons/speech";
import { default as default1179 } from "../icons/spell-check-2";
import { default as default1180 } from "../icons/spell-check";
import { default as default1181 } from "../icons/spline-pointer";
import { default as default1182 } from "../icons/spline";
import { default as default1183 } from "../icons/split";
import { default as default1184 } from "../icons/spray-can";
import { default as default1185 } from "../icons/sprout";
import { default as default1186 } from "../icons/square-dashed-bottom-code";
import { default as default1187 } from "../icons/square-dashed-bottom";
import { default as default1188 } from "../icons/square-dashed-top-solid";
import { default as default1189 } from "../icons/square-radical";
import { default as default1190 } from "../icons/square-round-corner";
import { default as default1191 } from "../icons/square-square";
import { default as default1192 } from "../icons/square-stack";
import { default as default1193 } from "../icons/square";
import { default as default1194 } from "../icons/squares-exclude";
import { default as default1195 } from "../icons/squares-subtract";
import { default as default1196 } from "../icons/squares-intersect";
import { default as default1197 } from "../icons/squares-unite";
import { default as default1198 } from "../icons/squircle";
import { default as default1199 } from "../icons/stamp";
import { default as default1200 } from "../icons/squirrel";
import { default as default1201 } from "../icons/star-half";
import { default as default1202 } from "../icons/star-off";
import { default as default1203 } from "../icons/star";
import { default as default1204 } from "../icons/step-forward";
import { default as default1205 } from "../icons/step-back";
import { default as default1206 } from "../icons/stethoscope";
import { default as default1207 } from "../icons/sticker";
import { default as default1208 } from "../icons/sticky-note";
import { default as default1209 } from "../icons/store";
import { default as default1210 } from "../icons/stretch-vertical";
import { default as default1211 } from "../icons/stretch-horizontal";
import { default as default1212 } from "../icons/strikethrough";
import { default as default1213 } from "../icons/subscript";
import { default as default1214 } from "../icons/sun-medium";
import { default as default1215 } from "../icons/sun-moon";
import { default as default1216 } from "../icons/sun-dim";
import { default as default1217 } from "../icons/sun-snow";
import { default as default1218 } from "../icons/sunrise";
import { default as default1219 } from "../icons/sunset";
import { default as default1220 } from "../icons/sun";
import { default as default1221 } from "../icons/superscript";
import { default as default1222 } from "../icons/swatch-book";
import { default as default1223 } from "../icons/switch-camera";
import { default as default1224 } from "../icons/swiss-franc";
import { default as default1225 } from "../icons/swords";
import { default as default1226 } from "../icons/sword";
import { default as default1227 } from "../icons/syringe";
import { default as default1228 } from "../icons/table-2";
import { default as default1229 } from "../icons/table-cells-split";
import { default as default1230 } from "../icons/table-cells-merge";
import { default as default1231 } from "../icons/table-properties";
import { default as default1232 } from "../icons/table-columns-split";
import { default as default1233 } from "../icons/table-of-contents";
import { default as default1234 } from "../icons/table-rows-split";
import { default as default1235 } from "../icons/tablet-smartphone";
import { default as default1236 } from "../icons/table";
import { default as default1237 } from "../icons/tablets";
import { default as default1238 } from "../icons/tablet";
import { default as default1239 } from "../icons/tags";
import { default as default1240 } from "../icons/tag";
import { default as default1241 } from "../icons/tally-1";
import { default as default1242 } from "../icons/tally-2";
import { default as default1243 } from "../icons/tally-3";
import { default as default1244 } from "../icons/tally-4";
import { default as default1245 } from "../icons/tally-5";
import { default as default1246 } from "../icons/tangent";
import { default as default1247 } from "../icons/target";
import { default as default1248 } from "../icons/telescope";
import { default as default1249 } from "../icons/tent";
import { default as default1250 } from "../icons/terminal";
import { default as default1251 } from "../icons/tent-tree";
import { default as default1252 } from "../icons/test-tube";
import { default as default1253 } from "../icons/test-tubes";
import { default as default1254 } from "../icons/text-cursor";
import { default as default1255 } from "../icons/text-cursor-input";
import { default as default1256 } from "../icons/text-search";
import { default as default1257 } from "../icons/text-quote";
import { default as default1258 } from "../icons/text";
import { default as default1259 } from "../icons/theater";
import { default as default1260 } from "../icons/thermometer-snowflake";
import { default as default1261 } from "../icons/thermometer-sun";
import { default as default1262 } from "../icons/thumbs-down";
import { default as default1263 } from "../icons/thermometer";
import { default as default1264 } from "../icons/thumbs-up";
import { default as default1265 } from "../icons/ticket-check";
import { default as default1266 } from "../icons/ticket-minus";
import { default as default1267 } from "../icons/ticket-percent";
import { default as default1268 } from "../icons/ticket-plus";
import { default as default1269 } from "../icons/ticket-slash";
import { default as default1270 } from "../icons/ticket-x";
import { default as default1271 } from "../icons/ticket";
import { default as default1272 } from "../icons/tickets-plane";
import { default as default1273 } from "../icons/tickets";
import { default as default1274 } from "../icons/timer-off";
import { default as default1275 } from "../icons/timer-reset";
import { default as default1276 } from "../icons/timer";
import { default as default1277 } from "../icons/toggle-left";
import { default as default1278 } from "../icons/toggle-right";
import { default as default1279 } from "../icons/toilet";
import { default as default1280 } from "../icons/tornado";
import { default as default1281 } from "../icons/torus";
import { default as default1282 } from "../icons/touchpad-off";
import { default as default1283 } from "../icons/touchpad";
import { default as default1284 } from "../icons/tower-control";
import { default as default1285 } from "../icons/toy-brick";
import { default as default1286 } from "../icons/tractor";
import { default as default1287 } from "../icons/traffic-cone";
import { default as default1288 } from "../icons/train-front-tunnel";
import { default as default1289 } from "../icons/train-front";
import { default as default1290 } from "../icons/train-track";
import { default as default1291 } from "../icons/transgender";
import { default as default1292 } from "../icons/trash-2";
import { default as default1293 } from "../icons/trash";
import { default as default1294 } from "../icons/tree-deciduous";
import { default as default1295 } from "../icons/tree-pine";
import { default as default1296 } from "../icons/trees";
import { default as default1297 } from "../icons/trello";
import { default as default1298 } from "../icons/trending-down";
import { default as default1299 } from "../icons/trending-up-down";
import { default as default1300 } from "../icons/trending-up";
import { default as default1301 } from "../icons/triangle-right";
import { default as default1302 } from "../icons/triangle-dashed";
import { default as default1303 } from "../icons/triangle";
import { default as default1304 } from "../icons/trophy";
import { default as default1305 } from "../icons/truck";
import { default as default1306 } from "../icons/truck-electric";
import { default as default1307 } from "../icons/turtle";
import { default as default1308 } from "../icons/tv-minimal-play";
import { default as default1309 } from "../icons/twitch";
import { default as default1310 } from "../icons/tv";
import { default as default1311 } from "../icons/twitter";
import { default as default1312 } from "../icons/type-outline";
import { default as default1313 } from "../icons/umbrella-off";
import { default as default1314 } from "../icons/type";
import { default as default1315 } from "../icons/umbrella";
import { default as default1316 } from "../icons/underline";
import { default as default1317 } from "../icons/undo-2";
import { default as default1318 } from "../icons/undo-dot";
import { default as default1319 } from "../icons/undo";
import { default as default1320 } from "../icons/unfold-horizontal";
import { default as default1321 } from "../icons/ungroup";
import { default as default1322 } from "../icons/unfold-vertical";
import { default as default1323 } from "../icons/unlink-2";
import { default as default1324 } from "../icons/unlink";
import { default as default1325 } from "../icons/unplug";
import { default as default1326 } from "../icons/upload";
import { default as default1327 } from "../icons/usb";
import { default as default1328 } from "../icons/user-cog";
import { default as default1329 } from "../icons/user-check";
import { default as default1330 } from "../icons/user-lock";
import { default as default1331 } from "../icons/user-minus";
import { default as default1332 } from "../icons/user-pen";
import { default as default1333 } from "../icons/user-plus";
import { default as default1334 } from "../icons/user-round-pen";
import { default as default1335 } from "../icons/user-round-search";
import { default as default1336 } from "../icons/user-search";
import { default as default1337 } from "../icons/user-x";
import { default as default1338 } from "../icons/user";
import { default as default1339 } from "../icons/users";
import { default as default1340 } from "../icons/utility-pole";
import { default as default1341 } from "../icons/variable";
import { default as default1342 } from "../icons/vault";
import { default as default1343 } from "../icons/vegan";
import { default as default1344 } from "../icons/venus-and-mars";
import { default as default1345 } from "../icons/venetian-mask";
import { default as default1346 } from "../icons/venus";
import { default as default1347 } from "../icons/vibrate-off";
import { default as default1348 } from "../icons/vibrate";
import { default as default1349 } from "../icons/video-off";
import { default as default1350 } from "../icons/video";
import { default as default1351 } from "../icons/videotape";
import { default as default1352 } from "../icons/view";
import { default as default1353 } from "../icons/voicemail";
import { default as default1354 } from "../icons/volleyball";
import { default as default1355 } from "../icons/volume-1";
import { default as default1356 } from "../icons/volume-2";
import { default as default1357 } from "../icons/volume-off";
import { default as default1358 } from "../icons/volume-x";
import { default as default1359 } from "../icons/volume";
import { default as default1360 } from "../icons/vote";
import { default as default1361 } from "../icons/wallet-cards";
import { default as default1362 } from "../icons/wallet";
import { default as default1363 } from "../icons/wallpaper";
import { default as default1364 } from "../icons/wand";
import { default as default1365 } from "../icons/warehouse";
import { default as default1366 } from "../icons/washing-machine";
import { default as default1367 } from "../icons/watch";
import { default as default1368 } from "../icons/waves-ladder";
import { default as default1369 } from "../icons/waves";
import { default as default1370 } from "../icons/waypoints";
import { default as default1371 } from "../icons/webcam";
import { default as default1372 } from "../icons/webhook";
import { default as default1373 } from "../icons/weight";
import { default as default1374 } from "../icons/wheat-off";
import { default as default1375 } from "../icons/webhook-off";
import { default as default1376 } from "../icons/wheat";
import { default as default1377 } from "../icons/whole-word";
import { default as default1378 } from "../icons/wifi-high";
import { default as default1379 } from "../icons/wifi-low";
import { default as default1380 } from "../icons/wifi-off";
import { default as default1381 } from "../icons/wifi-pen";
import { default as default1382 } from "../icons/wifi-zero";
import { default as default1383 } from "../icons/wifi";
import { default as default1384 } from "../icons/wind-arrow-down";
import { default as default1385 } from "../icons/wind";
import { default as default1386 } from "../icons/wine-off";
import { default as default1387 } from "../icons/workflow";
import { default as default1388 } from "../icons/wine";
import { default as default1389 } from "../icons/worm";
import { default as default1390 } from "../icons/wrap-text";
import { default as default1391 } from "../icons/wrench";
import { default as default1392 } from "../icons/x";
import { default as default1393 } from "../icons/youtube";
import { default as default1394 } from "../icons/zap-off";
import { default as default1395 } from "../icons/zap";
import { default as default1396 } from "../icons/zoom-in";
import { default as default1397 } from "../icons/zoom-out";
import { default as default1398 } from "../icons/arrow-down-0-1";
import { default as default1399 } from "../icons/arrow-down-1-0";
import { default as default1400 } from "../icons/arrow-up-0-1";
import { default as default1401 } from "../icons/arrow-up-1-0";
import { default as default1402 } from "../icons/alarm-clock-check";
import { default as default1403 } from "../icons/alarm-check";
import { default as default1404 } from "../icons/alarm-clock-minus";
import { default as default1405 } from "../icons/alarm-minus";
import { default as default1406 } from "../icons/alarm-clock-plus";
import { default as default1407 } from "../icons/alarm-plus";
import { default as default1408 } from "../icons/arrow-down-a-z";
import { default as default1409 } from "../icons/arrow-down-az";
import { default as default1410 } from "../icons/arrow-down-wide-narrow";
import { default as default1411 } from "../icons/sort-desc";
import { default as default1412 } from "../icons/arrow-down-z-a";
import { default as default1413 } from "../icons/arrow-down-za";
import { default as default1414 } from "../icons/arrow-up-a-z";
import { default as default1415 } from "../icons/arrow-up-az";
import { default as default1416 } from "../icons/arrow-up-narrow-wide";
import { default as default1417 } from "../icons/sort-asc";
import { default as default1418 } from "../icons/arrow-up-z-a";
import { default as default1419 } from "../icons/arrow-up-za";
import { default as default1420 } from "../icons/axis-3d";
import { default as default1421 } from "../icons/axis-3-d";
import { default as default1422 } from "../icons/badge-check";
import { default as default1423 } from "../icons/verified";
import { default as default1424 } from "../icons/between-horizontal-end";
import {
  default as default1425
} from "../icons/between-horizonal-end";
import { default as default1426 } from "../icons/between-horizontal-start";
import {
  default as default1427
} from "../icons/between-horizonal-start";
import { default as default1428 } from "../icons/book-dashed";
import { default as default1429 } from "../icons/book-template";
import { default as default1430 } from "../icons/braces";
import { default as default1431 } from "../icons/curly-braces";
import { default as default1432 } from "../icons/captions";
import { default as default1433 } from "../icons/subtitles";
import { default as default1434 } from "../icons/chart-area";
import {
  default as default1435
} from "../icons/area-chart";
import { default as default1436 } from "../icons/chart-bar-big";
import {
  default as default1437
} from "../icons/bar-chart-horizontal-big";
import { default as default1438 } from "../icons/chart-bar";
import {
  default as default1439
} from "../icons/bar-chart-horizontal";
import { default as default1440 } from "../icons/chart-candlestick";
import {
  default as default1441
} from "../icons/candlestick-chart";
import { default as default1442 } from "../icons/chart-column-big";
import {
  default as default1443
} from "../icons/bar-chart-big";
import { default as default1444 } from "../icons/chart-column-increasing";
import {
  default as default1445
} from "../icons/bar-chart-4";
import { default as default1446 } from "../icons/chart-column";
import {
  default as default1447
} from "../icons/bar-chart-3";
import { default as default1448 } from "../icons/chart-line";
import {
  default as default1449
} from "../icons/line-chart";
import { default as default1450 } from "../icons/chart-no-axes-column-increasing";
import {
  default as default1451
} from "../icons/bar-chart";
import { default as default1452 } from "../icons/chart-no-axes-column";
import {
  default as default1453
} from "../icons/bar-chart-2";
import { default as default1454 } from "../icons/chart-no-axes-gantt";
import {
  default as default1455
} from "../icons/gantt-chart";
import { default as default1456 } from "../icons/chart-pie";
import {
  default as default1457
} from "../icons/pie-chart";
import { default as default1458 } from "../icons/chart-scatter";
import {
  default as default1459
} from "../icons/scatter-chart";
import { default as default1460 } from "../icons/circle-alert";
import { default as default1461 } from "../icons/alert-circle";
import { default as default1462 } from "../icons/circle-arrow-down";
import { default as default1463 } from "../icons/arrow-down-circle";
import { default as default1464 } from "../icons/circle-arrow-left";
import { default as default1465 } from "../icons/arrow-left-circle";
import { default as default1466 } from "../icons/circle-arrow-out-down-right";
import { default as default1467 } from "../icons/arrow-down-right-from-circle";
import { default as default1468 } from "../icons/circle-arrow-out-down-left";
import { default as default1469 } from "../icons/arrow-down-left-from-circle";
import { default as default1470 } from "../icons/circle-arrow-out-up-left";
import { default as default1471 } from "../icons/arrow-up-left-from-circle";
import { default as default1472 } from "../icons/circle-arrow-out-up-right";
import { default as default1473 } from "../icons/arrow-up-right-from-circle";
import { default as default1474 } from "../icons/circle-arrow-right";
import { default as default1475 } from "../icons/arrow-right-circle";
import { default as default1476 } from "../icons/circle-arrow-up";
import { default as default1477 } from "../icons/arrow-up-circle";
import { default as default1478 } from "../icons/circle-check-big";
import { default as default1479 } from "../icons/check-circle";
import { default as default1480 } from "../icons/circle-check";
import { default as default1481 } from "../icons/check-circle-2";
import { default as default1482 } from "../icons/circle-chevron-down";
import { default as default1483 } from "../icons/chevron-down-circle";
import { default as default1484 } from "../icons/circle-chevron-left";
import { default as default1485 } from "../icons/chevron-left-circle";
import { default as default1486 } from "../icons/circle-chevron-right";
import { default as default1487 } from "../icons/chevron-right-circle";
import { default as default1488 } from "../icons/circle-chevron-up";
import { default as default1489 } from "../icons/chevron-up-circle";
import { default as default1490 } from "../icons/circle-divide";
import { default as default1491 } from "../icons/divide-circle";
import { default as default1492 } from "../icons/circle-gauge";
import { default as default1493 } from "../icons/gauge-circle";
import { default as default1494 } from "../icons/circle-help";
import { default as default1495 } from "../icons/help-circle";
import { default as default1496 } from "../icons/circle-minus";
import { default as default1497 } from "../icons/minus-circle";
import { default as default1498 } from "../icons/circle-parking-off";
import { default as default1499 } from "../icons/parking-circle-off";
import { default as default1500 } from "../icons/circle-parking";
import { default as default1501 } from "../icons/parking-circle";
import { default as default1502 } from "../icons/circle-percent";
import { default as default1503 } from "../icons/percent-circle";
import { default as default1504 } from "../icons/circle-pause";
import { default as default1505 } from "../icons/pause-circle";
import { default as default1506 } from "../icons/circle-play";
import { default as default1507 } from "../icons/play-circle";
import { default as default1508 } from "../icons/circle-plus";
import { default as default1509 } from "../icons/plus-circle";
import { default as default1510 } from "../icons/circle-power";
import { default as default1511 } from "../icons/power-circle";
import { default as default1512 } from "../icons/circle-slash-2";
import { default as default1513 } from "../icons/circle-slashed";
import { default as default1514 } from "../icons/circle-user-round";
import {
  default as default1515
} from "../icons/user-circle-2";
import { default as default1516 } from "../icons/circle-stop";
import { default as default1517 } from "../icons/stop-circle";
import { default as default1518 } from "../icons/circle-user";
import { default as default1519 } from "../icons/user-circle";
import { default as default1520 } from "../icons/circle-x";
import { default as default1521 } from "../icons/x-circle";
import { default as default1522 } from "../icons/clipboard-pen-line";
import { default as default1523 } from "../icons/clipboard-signature";
import { default as default1524 } from "../icons/clipboard-pen";
import { default as default1525 } from "../icons/clipboard-edit";
import { default as default1526 } from "../icons/cloud-download";
import { default as default1527 } from "../icons/download-cloud";
import { default as default1528 } from "../icons/cloud-upload";
import { default as default1529 } from "../icons/upload-cloud";
import { default as default1530 } from "../icons/code-xml";
import {
  default as default1531
} from "../icons/code-2";
import { default as default1532 } from "../icons/columns-2";
import { default as default1533 } from "../icons/columns";
import { default as default1534 } from "../icons/columns-3-cog";
import { default as default1535 } from "../icons/columns-settings";
import { default as default1536 } from "../icons/table-config";
import { default as default1537 } from "../icons/columns-3";
import { default as default1538 } from "../icons/panels-left-right";
import { default as default1539 } from "../icons/contact-round";
import {
  default as default1540
} from "../icons/contact-2";
import { default as default1541 } from "../icons/diamond-percent";
import { default as default1542 } from "../icons/percent-diamond";
import { default as default1543 } from "../icons/earth";
import {
  default as default1544
} from "../icons/globe-2";
import { default as default1545 } from "../icons/ellipsis-vertical";
import { default as default1546 } from "../icons/more-vertical";
import { default as default1547 } from "../icons/ellipsis";
import { default as default1548 } from "../icons/more-horizontal";
import { default as default1549 } from "../icons/file-axis-3d";
import { default as default1550 } from "../icons/file-axis-3-d";
import { default as default1551 } from "../icons/file-chart-column-increasing";
import {
  default as default1552
} from "../icons/file-bar-chart";
import { default as default1553 } from "../icons/file-chart-line";
import {
  default as default1554
} from "../icons/file-line-chart";
import { default as default1555 } from "../icons/file-chart-column";
import {
  default as default1556
} from "../icons/file-bar-chart-2";
import { default as default1557 } from "../icons/file-chart-pie";
import {
  default as default1558
} from "../icons/file-pie-chart";
import { default as default1559 } from "../icons/file-cog";
import {
  default as default1560
} from "../icons/file-cog-2";
import { default as default1561 } from "../icons/file-pen-line";
import { default as default1562 } from "../icons/file-signature";
import { default as default1563 } from "../icons/file-pen";
import { default as default1564 } from "../icons/file-edit";
import { default as default1565 } from "../icons/folder-cog";
import {
  default as default1566
} from "../icons/folder-cog-2";
import { default as default1567 } from "../icons/folder-pen";
import { default as default1568 } from "../icons/folder-edit";
import { default as default1569 } from "../icons/funnel-x";
import {
  default as default1570
} from "../icons/filter-x";
import { default as default1571 } from "../icons/funnel";
import {
  default as default1572
} from "../icons/filter";
import { default as default1573 } from "../icons/git-commit-horizontal";
import { default as default1574 } from "../icons/git-commit";
import { default as default1575 } from "../icons/grid-2x2-check";
import { default as default1576 } from "../icons/grid-2-x-2-check";
import { default as default1577 } from "../icons/grid-2x2-plus";
import { default as default1578 } from "../icons/grid-2-x-2-plus";
import { default as default1579 } from "../icons/grid-2x2-x";
import { default as default1580 } from "../icons/grid-2-x-2-x";
import { default as default1581 } from "../icons/grid-2x2";
import { default as default1582 } from "../icons/grid-2-x-2";
import { default as default1583 } from "../icons/grid-3x3";
import { default as default1584 } from "../icons/grid";
import { default as default1585 } from "../icons/grid-3-x-3";
import { default as default1586 } from "../icons/hand-helping";
import { default as default1587 } from "../icons/helping-hand";
import { default as default1588 } from "../icons/ice-cream-bowl";
import {
  default as default1589
} from "../icons/ice-cream-2";
import { default as default1590 } from "../icons/house";
import { default as default1591 } from "../icons/home";
import { default as default1592 } from "../icons/ice-cream-cone";
import { default as default1593 } from "../icons/ice-cream";
import { default as default1594 } from "../icons/indent-decrease";
import { default as default1595 } from "../icons/outdent";
import { default as default1596 } from "../icons/indent-increase";
import { default as default1597 } from "../icons/indent";
import { default as default1598 } from "../icons/laptop-minimal";
import {
  default as default1599
} from "../icons/laptop-2";
import { default as default1600 } from "../icons/layers";
import { default as default1601 } from "../icons/layers-3";
import { default as default1602 } from "../icons/loader-circle";
import { default as default1603 } from "../icons/loader-2";
import { default as default1604 } from "../icons/lock-keyhole-open";
import { default as default1605 } from "../icons/unlock-keyhole";
import { default as default1606 } from "../icons/lock-open";
import { default as default1607 } from "../icons/unlock";
import { default as default1608 } from "../icons/mic-vocal";
import {
  default as default1609
} from "../icons/mic-2";
import { default as default1610 } from "../icons/move-3d";
import { default as default1611 } from "../icons/move-3-d";
import { default as default1612 } from "../icons/octagon-alert";
import { default as default1613 } from "../icons/alert-octagon";
import { default as default1614 } from "../icons/octagon-pause";
import { default as default1615 } from "../icons/pause-octagon";
import { default as default1616 } from "../icons/octagon-x";
import { default as default1617 } from "../icons/x-octagon";
import { default as default1618 } from "../icons/paintbrush-vertical";
import { default as default1619 } from "../icons/paintbrush-2";
import { default as default1620 } from "../icons/panel-bottom-dashed";
import { default as default1621 } from "../icons/panel-bottom-inactive";
import { default as default1622 } from "../icons/panel-left-dashed";
import { default as default1623 } from "../icons/panel-left-inactive";
import { default as default1624 } from "../icons/panel-left-close";
import { default as default1625 } from "../icons/sidebar-close";
import { default as default1626 } from "../icons/panel-left-open";
import { default as default1627 } from "../icons/sidebar-open";
import { default as default1628 } from "../icons/panel-left";
import { default as default1629 } from "../icons/sidebar";
import { default as default1630 } from "../icons/panel-right-dashed";
import { default as default1631 } from "../icons/panel-right-inactive";
import { default as default1632 } from "../icons/panel-top-dashed";
import { default as default1633 } from "../icons/panel-top-inactive";
import { default as default1634 } from "../icons/panels-top-left";
import { default as default1635 } from "../icons/layout";
import { default as default1636 } from "../icons/pen-line";
import {
  default as default1637
} from "../icons/edit-3";
import { default as default1638 } from "../icons/pen";
import {
  default as default1639
} from "../icons/edit-2";
import { default as default1640 } from "../icons/plug-zap";
import { default as default1641 } from "../icons/plug-zap-2";
import { default as default1642 } from "../icons/rectangle-ellipsis";
import { default as default1643 } from "../icons/form-input";
import { default as default1644 } from "../icons/rotate-3d";
import { default as default1645 } from "../icons/rotate-3-d";
import { default as default1646 } from "../icons/rows-2";
import { default as default1647 } from "../icons/rows";
import { default as default1648 } from "../icons/rows-3";
import { default as default1649 } from "../icons/panels-top-bottom";
import { default as default1650 } from "../icons/scale-3d";
import { default as default1651 } from "../icons/scale-3-d";
import { default as default1652 } from "../icons/send-horizontal";
import {
  default as default1653
} from "../icons/send-horizonal";
import { default as default1654 } from "../icons/shield-x";
import { default as default1655 } from "../icons/shield-close";
import { default as default1656 } from "../icons/sliders-vertical";
import { default as default1657 } from "../icons/sliders";
import { default as default1658 } from "../icons/sparkles";
import { default as default1659 } from "../icons/stars";
import { default as default1660 } from "../icons/square-activity";
import { default as default1661 } from "../icons/activity-square";
import { default as default1662 } from "../icons/square-arrow-down-left";
import { default as default1663 } from "../icons/arrow-down-left-square";
import { default as default1664 } from "../icons/square-arrow-down-right";
import { default as default1665 } from "../icons/arrow-down-right-square";
import { default as default1666 } from "../icons/square-arrow-left";
import { default as default1667 } from "../icons/arrow-left-square";
import { default as default1668 } from "../icons/square-arrow-down";
import { default as default1669 } from "../icons/arrow-down-square";
import { default as default1670 } from "../icons/square-arrow-out-down-left";
import { default as default1671 } from "../icons/arrow-down-left-from-square";
import { default as default1672 } from "../icons/square-arrow-out-down-right";
import { default as default1673 } from "../icons/arrow-down-right-from-square";
import { default as default1674 } from "../icons/square-arrow-out-up-left";
import { default as default1675 } from "../icons/arrow-up-left-from-square";
import { default as default1676 } from "../icons/square-arrow-out-up-right";
import { default as default1677 } from "../icons/arrow-up-right-from-square";
import { default as default1678 } from "../icons/square-arrow-right";
import { default as default1679 } from "../icons/arrow-right-square";
import { default as default1680 } from "../icons/square-arrow-up-right";
import { default as default1681 } from "../icons/arrow-up-right-square";
import { default as default1682 } from "../icons/square-arrow-up-left";
import { default as default1683 } from "../icons/arrow-up-left-square";
import { default as default1684 } from "../icons/square-arrow-up";
import { default as default1685 } from "../icons/arrow-up-square";
import { default as default1686 } from "../icons/square-asterisk";
import { default as default1687 } from "../icons/asterisk-square";
import { default as default1688 } from "../icons/square-bottom-dashed-scissors";
import { default as default1689 } from "../icons/scissors-square-dashed-bottom";
import { default as default1690 } from "../icons/square-chart-gantt";
import { default as default1691 } from "../icons/gantt-chart-square";
import { default as default1692 } from "../icons/square-gantt-chart";
import { default as default1693 } from "../icons/square-check";
import { default as default1694 } from "../icons/check-square-2";
import { default as default1695 } from "../icons/square-chevron-down";
import { default as default1696 } from "../icons/chevron-down-square";
import { default as default1697 } from "../icons/square-chevron-left";
import { default as default1698 } from "../icons/chevron-left-square";
import { default as default1699 } from "../icons/square-check-big";
import { default as default1700 } from "../icons/check-square";
import { default as default1701 } from "../icons/square-chevron-right";
import { default as default1702 } from "../icons/chevron-right-square";
import { default as default1703 } from "../icons/square-chevron-up";
import { default as default1704 } from "../icons/chevron-up-square";
import { default as default1705 } from "../icons/square-code";
import { default as default1706 } from "../icons/code-square";
import { default as default1707 } from "../icons/square-dashed-kanban";
import { default as default1708 } from "../icons/kanban-square-dashed";
import { default as default1709 } from "../icons/square-dashed-mouse-pointer";
import { default as default1710 } from "../icons/mouse-pointer-square-dashed";
import { default as default1711 } from "../icons/square-dashed";
import { default as default1712 } from "../icons/box-select";
import { default as default1713 } from "../icons/square-divide";
import { default as default1714 } from "../icons/divide-square";
import { default as default1715 } from "../icons/square-dot";
import { default as default1716 } from "../icons/dot-square";
import { default as default1717 } from "../icons/square-equal";
import { default as default1718 } from "../icons/equal-square";
import { default as default1719 } from "../icons/square-function";
import { default as default1720 } from "../icons/function-square";
import { default as default1721 } from "../icons/square-kanban";
import { default as default1722 } from "../icons/kanban-square";
import { default as default1723 } from "../icons/square-m";
import { default as default1724 } from "../icons/m-square";
import { default as default1725 } from "../icons/square-library";
import { default as default1726 } from "../icons/library-square";
import { default as default1727 } from "../icons/square-menu";
import { default as default1728 } from "../icons/menu-square";
import { default as default1729 } from "../icons/square-mouse-pointer";
import { default as default1730 } from "../icons/inspect";
import { default as default1731 } from "../icons/square-minus";
import { default as default1732 } from "../icons/minus-square";
import { default as default1733 } from "../icons/square-parking-off";
import { default as default1734 } from "../icons/parking-square-off";
import { default as default1735 } from "../icons/square-parking";
import { default as default1736 } from "../icons/parking-square";
import { default as default1737 } from "../icons/square-pen";
import { default as default1738 } from "../icons/pen-box";
import { default as default1739 } from "../icons/pen-square";
import { default as default1740 } from "../icons/edit";
import { default as default1741 } from "../icons/square-percent";
import { default as default1742 } from "../icons/percent-square";
import { default as default1743 } from "../icons/square-pi";
import { default as default1744 } from "../icons/pi-square";
import { default as default1745 } from "../icons/square-pilcrow";
import { default as default1746 } from "../icons/pilcrow-square";
import { default as default1747 } from "../icons/square-play";
import { default as default1748 } from "../icons/play-square";
import { default as default1749 } from "../icons/square-plus";
import { default as default1750 } from "../icons/plus-square";
import { default as default1751 } from "../icons/square-power";
import { default as default1752 } from "../icons/power-square";
import { default as default1753 } from "../icons/square-scissors";
import { default as default1754 } from "../icons/scissors-square";
import { default as default1755 } from "../icons/square-slash";
import { default as default1756 } from "../icons/slash-square";
import { default as default1757 } from "../icons/square-sigma";
import { default as default1758 } from "../icons/sigma-square";
import { default as default1759 } from "../icons/square-split-horizontal";
import { default as default1760 } from "../icons/split-square-horizontal";
import { default as default1761 } from "../icons/square-split-vertical";
import { default as default1762 } from "../icons/split-square-vertical";
import { default as default1763 } from "../icons/square-user-round";
import {
  default as default1764
} from "../icons/user-square-2";
import { default as default1765 } from "../icons/square-terminal";
import { default as default1766 } from "../icons/terminal-square";
import { default as default1767 } from "../icons/square-user";
import { default as default1768 } from "../icons/user-square";
import { default as default1769 } from "../icons/square-x";
import { default as default1770 } from "../icons/x-square";
import { default as default1771 } from "../icons/test-tube-diagonal";
import { default as default1772 } from "../icons/test-tube-2";
import { default as default1773 } from "../icons/text-select";
import { default as default1774 } from "../icons/text-selection";
import { default as default1775 } from "../icons/tram-front";
import { default as default1776 } from "../icons/train";
import { default as default1777 } from "../icons/tree-palm";
import { default as default1778 } from "../icons/palmtree";
import { default as default1779 } from "../icons/triangle-alert";
import { default as default1780 } from "../icons/alert-triangle";
import { default as default1781 } from "../icons/tv-minimal";
import {
  default as default1782
} from "../icons/tv-2";
import { default as default1783 } from "../icons/university";
import {
  default as default1784
} from "../icons/school-2";
import { default as default1785 } from "../icons/user-round-check";
import {
  default as default1786
} from "../icons/user-check-2";
import { default as default1787 } from "../icons/user-round-cog";
import {
  default as default1788
} from "../icons/user-cog-2";
import { default as default1789 } from "../icons/user-round-minus";
import {
  default as default1790
} from "../icons/user-minus-2";
import { default as default1791 } from "../icons/user-round-plus";
import {
  default as default1792
} from "../icons/user-plus-2";
import { default as default1793 } from "../icons/user-round-x";
import {
  default as default1794
} from "../icons/user-x-2";
import { default as default1795 } from "../icons/user-round";
import {
  default as default1796
} from "../icons/user-2";
import { default as default1797 } from "../icons/users-round";
import {
  default as default1798
} from "../icons/users-2";
import { default as default1799 } from "../icons/utensils-crossed";
import { default as default1800 } from "../icons/fork-knife-crossed";
import { default as default1801 } from "../icons/utensils";
import { default as default1802 } from "../icons/fork-knife";
import { default as default1803 } from "../icons/wallet-minimal";
import {
  default as default1804
} from "../icons/wallet-2";
import { default as default1805 } from "../icons/wand-sparkles";
import {
  default as default1806
} from "../icons/wand-2";
export {
  default2 as LucideAArrowDown,
  default3 as LucideAArrowUp,
  default4 as LucideALargeSmall,
  default5 as LucideAccessibility,
  default6 as LucideActivity,
  default1661 as LucideActivitySquare,
  default7 as LucideAirVent,
  default8 as LucideAirplay,
  default1403 as LucideAlarmCheck,
  default10 as LucideAlarmClock,
  default1402 as LucideAlarmClockCheck,
  default1404 as LucideAlarmClockMinus,
  default9 as LucideAlarmClockOff,
  default1406 as LucideAlarmClockPlus,
  default1405 as LucideAlarmMinus,
  default1407 as LucideAlarmPlus,
  default11 as LucideAlarmSmoke,
  default12 as LucideAlbum,
  default1461 as LucideAlertCircle,
  default1613 as LucideAlertOctagon,
  default1780 as LucideAlertTriangle,
  default15 as LucideAlignCenter,
  default13 as LucideAlignCenterHorizontal,
  default14 as LucideAlignCenterVertical,
  default16 as LucideAlignEndHorizontal,
  default17 as LucideAlignEndVertical,
  default18 as LucideAlignHorizontalDistributeCenter,
  default19 as LucideAlignHorizontalDistributeEnd,
  default20 as LucideAlignHorizontalDistributeStart,
  default21 as LucideAlignHorizontalJustifyCenter,
  default22 as LucideAlignHorizontalJustifyEnd,
  default23 as LucideAlignHorizontalJustifyStart,
  default24 as LucideAlignHorizontalSpaceAround,
  default25 as LucideAlignHorizontalSpaceBetween,
  default26 as LucideAlignJustify,
  default27 as LucideAlignLeft,
  default28 as LucideAlignRight,
  default29 as LucideAlignStartHorizontal,
  default30 as LucideAlignStartVertical,
  default31 as LucideAlignVerticalDistributeCenter,
  default32 as LucideAlignVerticalDistributeEnd,
  default33 as LucideAlignVerticalDistributeStart,
  default34 as LucideAlignVerticalJustifyCenter,
  default37 as LucideAlignVerticalJustifyEnd,
  default35 as LucideAlignVerticalJustifyStart,
  default36 as LucideAlignVerticalSpaceAround,
  default38 as LucideAlignVerticalSpaceBetween,
  default39 as LucideAmbulance,
  default40 as LucideAmpersand,
  default41 as LucideAmpersands,
  default42 as LucideAmphora,
  default43 as LucideAnchor,
  default44 as LucideAngry,
  default46 as LucideAnnoyed,
  default45 as LucideAntenna,
  default47 as LucideAnvil,
  default48 as LucideAperture,
  default52 as LucideAppWindow,
  default49 as LucideAppWindowMac,
  default50 as LucideApple,
  default54 as LucideArchive,
  default51 as LucideArchiveRestore,
  default53 as LucideArchiveX,
  default1435 as LucideAreaChart,
  default56 as LucideArmchair,
  default58 as LucideArrowBigDown,
  default55 as LucideArrowBigDownDash,
  default60 as LucideArrowBigLeft,
  default57 as LucideArrowBigLeftDash,
  default61 as LucideArrowBigRight,
  default59 as LucideArrowBigRightDash,
  default63 as LucideArrowBigUp,
  default62 as LucideArrowBigUpDash,
  default71 as LucideArrowDown,
  default1398 as LucideArrowDown01,
  default1399 as LucideArrowDown10,
  default1408 as LucideArrowDownAZ,
  default1409 as LucideArrowDownAz,
  default1463 as LucideArrowDownCircle,
  default64 as LucideArrowDownFromLine,
  default65 as LucideArrowDownLeft,
  default1469 as LucideArrowDownLeftFromCircle,
  default1671 as LucideArrowDownLeftFromSquare,
  default1663 as LucideArrowDownLeftSquare,
  default66 as LucideArrowDownNarrowWide,
  default68 as LucideArrowDownRight,
  default1467 as LucideArrowDownRightFromCircle,
  default1673 as LucideArrowDownRightFromSquare,
  default1665 as LucideArrowDownRightSquare,
  default1669 as LucideArrowDownSquare,
  default67 as LucideArrowDownToDot,
  default70 as LucideArrowDownToLine,
  default69 as LucideArrowDownUp,
  default1410 as LucideArrowDownWideNarrow,
  default1412 as LucideArrowDownZA,
  default1413 as LucideArrowDownZa,
  default75 as LucideArrowLeft,
  default1465 as LucideArrowLeftCircle,
  default72 as LucideArrowLeftFromLine,
  default73 as LucideArrowLeftRight,
  default1667 as LucideArrowLeftSquare,
  default74 as LucideArrowLeftToLine,
  default79 as LucideArrowRight,
  default1475 as LucideArrowRightCircle,
  default76 as LucideArrowRightFromLine,
  default77 as LucideArrowRightLeft,
  default1679 as LucideArrowRightSquare,
  default78 as LucideArrowRightToLine,
  default87 as LucideArrowUp,
  default1400 as LucideArrowUp01,
  default1401 as LucideArrowUp10,
  default1414 as LucideArrowUpAZ,
  default1415 as LucideArrowUpAz,
  default1477 as LucideArrowUpCircle,
  default80 as LucideArrowUpDown,
  default81 as LucideArrowUpFromDot,
  default82 as LucideArrowUpFromLine,
  default83 as LucideArrowUpLeft,
  default1471 as LucideArrowUpLeftFromCircle,
  default1675 as LucideArrowUpLeftFromSquare,
  default1683 as LucideArrowUpLeftSquare,
  default1416 as LucideArrowUpNarrowWide,
  default84 as LucideArrowUpRight,
  default1473 as LucideArrowUpRightFromCircle,
  default1677 as LucideArrowUpRightFromSquare,
  default1681 as LucideArrowUpRightSquare,
  default1685 as LucideArrowUpSquare,
  default85 as LucideArrowUpToLine,
  default86 as LucideArrowUpWideNarrow,
  default1418 as LucideArrowUpZA,
  default1419 as LucideArrowUpZa,
  default88 as LucideArrowsUpFromLine,
  default89 as LucideAsterisk,
  default1687 as LucideAsteriskSquare,
  default90 as LucideAtSign,
  default91 as LucideAtom,
  default93 as LucideAudioLines,
  default92 as LucideAudioWaveform,
  default94 as LucideAward,
  default95 as LucideAxe,
  default1421 as LucideAxis3D,
  default1420 as LucideAxis3d,
  default96 as LucideBaby,
  default97 as LucideBackpack,
  default113 as LucideBadge,
  default98 as LucideBadgeAlert,
  default99 as LucideBadgeCent,
  default1422 as LucideBadgeCheck,
  default100 as LucideBadgeDollarSign,
  default101 as LucideBadgeEuro,
  default102 as LucideBadgeHelp,
  default103 as LucideBadgeIndianRupee,
  default104 as LucideBadgeInfo,
  default105 as LucideBadgeJapaneseYen,
  default106 as LucideBadgeMinus,
  default108 as LucideBadgePercent,
  default107 as LucideBadgePlus,
  default109 as LucideBadgePoundSterling,
  default110 as LucideBadgeRussianRuble,
  default111 as LucideBadgeSwissFranc,
  default112 as LucideBadgeX,
  default114 as LucideBaggageClaim,
  default115 as LucideBan,
  default116 as LucideBanana,
  default117 as LucideBandage,
  default121 as LucideBanknote,
  default118 as LucideBanknoteArrowDown,
  default119 as LucideBanknoteArrowUp,
  default120 as LucideBanknoteX,
  default1451 as LucideBarChart,
  default1453 as LucideBarChart2,
  default1447 as LucideBarChart3,
  default1445 as LucideBarChart4,
  default1443 as LucideBarChartBig,
  default1439 as LucideBarChartHorizontal,
  default1437 as LucideBarChartHorizontalBig,
  default122 as LucideBarcode,
  default123 as LucideBaseline,
  default124 as LucideBath,
  default130 as LucideBattery,
  default125 as LucideBatteryCharging,
  default126 as LucideBatteryFull,
  default127 as LucideBatteryLow,
  default128 as LucideBatteryMedium,
  default131 as LucideBatteryPlus,
  default129 as LucideBatteryWarning,
  default132 as LucideBeaker,
  default134 as LucideBean,
  default133 as LucideBeanOff,
  default137 as LucideBed,
  default135 as LucideBedDouble,
  default136 as LucideBedSingle,
  default138 as LucideBeef,
  default140 as LucideBeer,
  default139 as LucideBeerOff,
  default147 as LucideBell,
  default141 as LucideBellDot,
  default142 as LucideBellElectric,
  default143 as LucideBellMinus,
  default144 as LucideBellOff,
  default145 as LucideBellPlus,
  default146 as LucideBellRing,
  default1425 as LucideBetweenHorizonalEnd,
  default1427 as LucideBetweenHorizonalStart,
  default1424 as LucideBetweenHorizontalEnd,
  default1426 as LucideBetweenHorizontalStart,
  default148 as LucideBetweenVerticalEnd,
  default149 as LucideBetweenVerticalStart,
  default150 as LucideBicepsFlexed,
  default151 as LucideBike,
  default152 as LucideBinary,
  default153 as LucideBinoculars,
  default154 as LucideBiohazard,
  default155 as LucideBird,
  default157 as LucideBitcoin,
  default156 as LucideBlend,
  default158 as LucideBlinds,
  default159 as LucideBlocks,
  default163 as LucideBluetooth,
  default160 as LucideBluetoothConnected,
  default161 as LucideBluetoothOff,
  default162 as LucideBluetoothSearching,
  default164 as LucideBold,
  default166 as LucideBolt,
  default165 as LucideBomb,
  default167 as LucideBone,
  default191 as LucideBook,
  default168 as LucideBookA,
  default169 as LucideBookAlert,
  default170 as LucideBookAudio,
  default171 as LucideBookCheck,
  default172 as LucideBookCopy,
  default1428 as LucideBookDashed,
  default173 as LucideBookDown,
  default174 as LucideBookHeadphones,
  default175 as LucideBookHeart,
  default176 as LucideBookImage,
  default177 as LucideBookKey,
  default178 as LucideBookLock,
  default179 as LucideBookMarked,
  default180 as LucideBookMinus,
  default183 as LucideBookOpen,
  default181 as LucideBookOpenCheck,
  default182 as LucideBookOpenText,
  default184 as LucideBookPlus,
  default1429 as LucideBookTemplate,
  default185 as LucideBookText,
  default186 as LucideBookType,
  default188 as LucideBookUp,
  default187 as LucideBookUp2,
  default189 as LucideBookUser,
  default190 as LucideBookX,
  default196 as LucideBookmark,
  default192 as LucideBookmarkCheck,
  default193 as LucideBookmarkMinus,
  default194 as LucideBookmarkPlus,
  default195 as LucideBookmarkX,
  default197 as LucideBoomBox,
  default200 as LucideBot,
  default198 as LucideBotMessageSquare,
  default199 as LucideBotOff,
  default202 as LucideBowArrow,
  default201 as LucideBox,
  default1712 as LucideBoxSelect,
  default203 as LucideBoxes,
  default1430 as LucideBraces,
  default205 as LucideBrackets,
  default207 as LucideBrain,
  default204 as LucideBrainCircuit,
  default206 as LucideBrainCog,
  default209 as LucideBrickWall,
  default208 as LucideBrickWallFire,
  default213 as LucideBriefcase,
  default210 as LucideBriefcaseBusiness,
  default211 as LucideBriefcaseConveyorBelt,
  default212 as LucideBriefcaseMedical,
  default214 as LucideBringToFront,
  default216 as LucideBrush,
  default215 as LucideBrushCleaning,
  default217 as LucideBubbles,
  default219 as LucideBug,
  default218 as LucideBugOff,
  default220 as LucideBugPlay,
  default222 as LucideBuilding,
  default221 as LucideBuilding2,
  default224 as LucideBus,
  default223 as LucideBusFront,
  default226 as LucideCable,
  default225 as LucideCableCar,
  default228 as LucideCake,
  default227 as LucideCakeSlice,
  default229 as LucideCalculator,
  default250 as LucideCalendar,
  default231 as LucideCalendar1,
  default230 as LucideCalendarArrowDown,
  default233 as LucideCalendarArrowUp,
  default234 as LucideCalendarCheck,
  default232 as LucideCalendarCheck2,
  default235 as LucideCalendarClock,
  default236 as LucideCalendarCog,
  default237 as LucideCalendarDays,
  default238 as LucideCalendarFold,
  default239 as LucideCalendarHeart,
  default241 as LucideCalendarMinus,
  default240 as LucideCalendarMinus2,
  default242 as LucideCalendarOff,
  default244 as LucideCalendarPlus,
  default243 as LucideCalendarPlus2,
  default245 as LucideCalendarRange,
  default246 as LucideCalendarSearch,
  default247 as LucideCalendarSync,
  default249 as LucideCalendarX,
  default248 as LucideCalendarX2,
  default252 as LucideCamera,
  default251 as LucideCameraOff,
  default1441 as LucideCandlestickChart,
  default255 as LucideCandy,
  default253 as LucideCandyCane,
  default254 as LucideCandyOff,
  default256 as LucideCannabis,
  default1432 as LucideCaptions,
  default257 as LucideCaptionsOff,
  default260 as LucideCar,
  default258 as LucideCarFront,
  default259 as LucideCarTaxiFront,
  default261 as LucideCaravan,
  default262 as LucideCardSim,
  default263 as LucideCarrot,
  default264 as LucideCaseLower,
  default265 as LucideCaseSensitive,
  default266 as LucideCaseUpper,
  default267 as LucideCassetteTape,
  default268 as LucideCast,
  default270 as LucideCastle,
  default269 as LucideCat,
  default271 as LucideCctv,
  default1434 as LucideChartArea,
  default1438 as LucideChartBar,
  default1436 as LucideChartBarBig,
  default272 as LucideChartBarDecreasing,
  default273 as LucideChartBarIncreasing,
  default274 as LucideChartBarStacked,
  default1440 as LucideChartCandlestick,
  default1446 as LucideChartColumn,
  default1442 as LucideChartColumnBig,
  default275 as LucideChartColumnDecreasing,
  default1444 as LucideChartColumnIncreasing,
  default276 as LucideChartColumnStacked,
  default277 as LucideChartGantt,
  default1448 as LucideChartLine,
  default278 as LucideChartNetwork,
  default1452 as LucideChartNoAxesColumn,
  default279 as LucideChartNoAxesColumnDecreasing,
  default1450 as LucideChartNoAxesColumnIncreasing,
  default280 as LucideChartNoAxesCombined,
  default1454 as LucideChartNoAxesGantt,
  default1456 as LucideChartPie,
  default1458 as LucideChartScatter,
  default281 as LucideChartSpline,
  default284 as LucideCheck,
  default282 as LucideCheckCheck,
  default1479 as LucideCheckCircle,
  default1481 as LucideCheckCircle2,
  default283 as LucideCheckLine,
  default1700 as LucideCheckSquare,
  default1694 as LucideCheckSquare2,
  default285 as LucideChefHat,
  default287 as LucideCherry,
  default286 as LucideChevronDown,
  default1483 as LucideChevronDownCircle,
  default1696 as LucideChevronDownSquare,
  default288 as LucideChevronFirst,
  default289 as LucideChevronLast,
  default291 as LucideChevronLeft,
  default1485 as LucideChevronLeftCircle,
  default1698 as LucideChevronLeftSquare,
  default290 as LucideChevronRight,
  default1487 as LucideChevronRightCircle,
  default1702 as LucideChevronRightSquare,
  default292 as LucideChevronUp,
  default1489 as LucideChevronUpCircle,
  default1704 as LucideChevronUpSquare,
  default295 as LucideChevronsDown,
  default293 as LucideChevronsDownUp,
  default297 as LucideChevronsLeft,
  default296 as LucideChevronsLeftRight,
  default294 as LucideChevronsLeftRightEllipsis,
  default299 as LucideChevronsRight,
  default298 as LucideChevronsRightLeft,
  default302 as LucideChevronsUp,
  default300 as LucideChevronsUpDown,
  default301 as LucideChrome,
  default303 as LucideChurch,
  default305 as LucideCigarette,
  default304 as LucideCigaretteOff,
  default318 as LucideCircle,
  default1460 as LucideCircleAlert,
  default1462 as LucideCircleArrowDown,
  default1464 as LucideCircleArrowLeft,
  default1468 as LucideCircleArrowOutDownLeft,
  default1466 as LucideCircleArrowOutDownRight,
  default1470 as LucideCircleArrowOutUpLeft,
  default1472 as LucideCircleArrowOutUpRight,
  default1474 as LucideCircleArrowRight,
  default1476 as LucideCircleArrowUp,
  default1480 as LucideCircleCheck,
  default1478 as LucideCircleCheckBig,
  default1482 as LucideCircleChevronDown,
  default1484 as LucideCircleChevronLeft,
  default1486 as LucideCircleChevronRight,
  default1488 as LucideCircleChevronUp,
  default306 as LucideCircleDashed,
  default1490 as LucideCircleDivide,
  default308 as LucideCircleDollarSign,
  default309 as LucideCircleDot,
  default307 as LucideCircleDotDashed,
  default310 as LucideCircleEllipsis,
  default311 as LucideCircleEqual,
  default312 as LucideCircleFadingArrowUp,
  default313 as LucideCircleFadingPlus,
  default1492 as LucideCircleGauge,
  default1494 as LucideCircleHelp,
  default1496 as LucideCircleMinus,
  default314 as LucideCircleOff,
  default1500 as LucideCircleParking,
  default1498 as LucideCircleParkingOff,
  default1504 as LucideCirclePause,
  default1502 as LucideCirclePercent,
  default1506 as LucideCirclePlay,
  default1508 as LucideCirclePlus,
  default315 as LucideCirclePoundSterling,
  default1510 as LucideCirclePower,
  default316 as LucideCircleSlash,
  default1512 as LucideCircleSlash2,
  default1513 as LucideCircleSlashed,
  default317 as LucideCircleSmall,
  default1516 as LucideCircleStop,
  default1518 as LucideCircleUser,
  default1514 as LucideCircleUserRound,
  default1520 as LucideCircleX,
  default319 as LucideCircuitBoard,
  default320 as LucideCitrus,
  default321 as LucideClapperboard,
  default330 as LucideClipboard,
  default322 as LucideClipboardCheck,
  default323 as LucideClipboardCopy,
  default1525 as LucideClipboardEdit,
  default324 as LucideClipboardList,
  default325 as LucideClipboardMinus,
  default326 as LucideClipboardPaste,
  default1524 as LucideClipboardPen,
  default1522 as LucideClipboardPenLine,
  default327 as LucideClipboardPlus,
  default1523 as LucideClipboardSignature,
  default329 as LucideClipboardType,
  default328 as LucideClipboardX,
  default348 as LucideClock,
  default332 as LucideClock1,
  default331 as LucideClock10,
  default333 as LucideClock11,
  default334 as LucideClock12,
  default335 as LucideClock2,
  default336 as LucideClock3,
  default337 as LucideClock4,
  default338 as LucideClock5,
  default339 as LucideClock6,
  default340 as LucideClock7,
  default341 as LucideClock8,
  default342 as LucideClock9,
  default343 as LucideClockAlert,
  default344 as LucideClockArrowDown,
  default345 as LucideClockArrowUp,
  default346 as LucideClockFading,
  default347 as LucideClockPlus,
  default364 as LucideCloud,
  default349 as LucideCloudAlert,
  default350 as LucideCloudCheck,
  default351 as LucideCloudCog,
  default1526 as LucideCloudDownload,
  default352 as LucideCloudDrizzle,
  default353 as LucideCloudFog,
  default354 as LucideCloudHail,
  default355 as LucideCloudLightning,
  default357 as LucideCloudMoon,
  default356 as LucideCloudMoonRain,
  default358 as LucideCloudOff,
  default360 as LucideCloudRain,
  default359 as LucideCloudRainWind,
  default361 as LucideCloudSnow,
  default363 as LucideCloudSun,
  default362 as LucideCloudSunRain,
  default1528 as LucideCloudUpload,
  default365 as LucideCloudy,
  default368 as LucideClover,
  default366 as LucideClub,
  default367 as LucideCode,
  default1531 as LucideCode2,
  default1706 as LucideCodeSquare,
  default1530 as LucideCodeXml,
  default369 as LucideCodepen,
  default370 as LucideCodesandbox,
  default371 as LucideCoffee,
  default372 as LucideCog,
  default373 as LucideCoins,
  default1533 as LucideColumns,
  default1532 as LucideColumns2,
  default1537 as LucideColumns3,
  default1534 as LucideColumns3Cog,
  default374 as LucideColumns4,
  default1535 as LucideColumnsSettings,
  default376 as LucideCombine,
  default375 as LucideCommand,
  default377 as LucideCompass,
  default379 as LucideComponent,
  default378 as LucideComputer,
  default380 as LucideConciergeBell,
  default382 as LucideCone,
  default381 as LucideConstruction,
  default383 as LucideContact,
  default1540 as LucideContact2,
  default1539 as LucideContactRound,
  default385 as LucideContainer,
  default384 as LucideContrast,
  default386 as LucideCookie,
  default387 as LucideCookingPot,
  default393 as LucideCopy,
  default388 as LucideCopyCheck,
  default389 as LucideCopyMinus,
  default390 as LucideCopyPlus,
  default391 as LucideCopySlash,
  default392 as LucideCopyX,
  default394 as LucideCopyleft,
  default395 as LucideCopyright,
  default396 as LucideCornerDownLeft,
  default397 as LucideCornerDownRight,
  default398 as LucideCornerLeftDown,
  default399 as LucideCornerLeftUp,
  default400 as LucideCornerRightDown,
  default401 as LucideCornerRightUp,
  default404 as LucideCornerUpLeft,
  default402 as LucideCornerUpRight,
  default403 as LucideCpu,
  default405 as LucideCreativeCommons,
  default407 as LucideCreditCard,
  default406 as LucideCroissant,
  default408 as LucideCrop,
  default409 as LucideCross,
  default410 as LucideCrosshair,
  default411 as LucideCrown,
  default412 as LucideCuboid,
  default413 as LucideCupSoda,
  default1431 as LucideCurlyBraces,
  default414 as LucideCurrency,
  default415 as LucideCylinder,
  default416 as LucideDam,
  default419 as LucideDatabase,
  default417 as LucideDatabaseBackup,
  default418 as LucideDatabaseZap,
  default420 as LucideDecimalsArrowLeft,
  default422 as LucideDecimalsArrowRight,
  default421 as LucideDelete,
  default423 as LucideDessert,
  default424 as LucideDiameter,
  default427 as LucideDiamond,
  default425 as LucideDiamondMinus,
  default1541 as LucideDiamondPercent,
  default426 as LucideDiamondPlus,
  default428 as LucideDice1,
  default429 as LucideDice2,
  default430 as LucideDice3,
  default431 as LucideDice4,
  default432 as LucideDice5,
  default433 as LucideDice6,
  default434 as LucideDices,
  default435 as LucideDiff,
  default439 as LucideDisc,
  default436 as LucideDisc2,
  default437 as LucideDisc3,
  default438 as LucideDiscAlbum,
  default440 as LucideDivide,
  default1491 as LucideDivideCircle,
  default1714 as LucideDivideSquare,
  default442 as LucideDna,
  default441 as LucideDnaOff,
  default443 as LucideDock,
  default444 as LucideDog,
  default445 as LucideDollarSign,
  default446 as LucideDonut,
  default448 as LucideDoorClosed,
  default447 as LucideDoorClosedLocked,
  default449 as LucideDoorOpen,
  default450 as LucideDot,
  default1716 as LucideDotSquare,
  default451 as LucideDownload,
  default1527 as LucideDownloadCloud,
  default452 as LucideDraftingCompass,
  default453 as LucideDrama,
  default454 as LucideDribbble,
  default455 as LucideDrill,
  default457 as LucideDroplet,
  default456 as LucideDropletOff,
  default458 as LucideDroplets,
  default459 as LucideDrum,
  default460 as LucideDrumstick,
  default461 as LucideDumbbell,
  default463 as LucideEar,
  default462 as LucideEarOff,
  default1543 as LucideEarth,
  default464 as LucideEarthLock,
  default465 as LucideEclipse,
  default1740 as LucideEdit,
  default1639 as LucideEdit2,
  default1637 as LucideEdit3,
  default468 as LucideEgg,
  default466 as LucideEggFried,
  default467 as LucideEggOff,
  default1547 as LucideEllipsis,
  default1545 as LucideEllipsisVertical,
  default470 as LucideEqual,
  default469 as LucideEqualApproximately,
  default471 as LucideEqualNot,
  default1718 as LucideEqualSquare,
  default472 as LucideEraser,
  default473 as LucideEthernetPort,
  default474 as LucideEuro,
  default475 as LucideExpand,
  default476 as LucideExternalLink,
  default479 as LucideEye,
  default477 as LucideEyeClosed,
  default478 as LucideEyeOff,
  default480 as LucideFacebook,
  default481 as LucideFactory,
  default482 as LucideFan,
  default483 as LucideFastForward,
  default484 as LucideFeather,
  default485 as LucideFence,
  default486 as LucideFerrisWheel,
  default487 as LucideFigma,
  default538 as LucideFile,
  default488 as LucideFileArchive,
  default490 as LucideFileAudio,
  default489 as LucideFileAudio2,
  default1550 as LucideFileAxis3D,
  default1549 as LucideFileAxis3d,
  default492 as LucideFileBadge,
  default491 as LucideFileBadge2,
  default1552 as LucideFileBarChart,
  default1556 as LucideFileBarChart2,
  default493 as LucideFileBox,
  default1555 as LucideFileChartColumn,
  default1551 as LucideFileChartColumnIncreasing,
  default1553 as LucideFileChartLine,
  default1557 as LucideFileChartPie,
  default495 as LucideFileCheck,
  default494 as LucideFileCheck2,
  default496 as LucideFileClock,
  default498 as LucideFileCode,
  default497 as LucideFileCode2,
  default1559 as LucideFileCog,
  default1560 as LucideFileCog2,
  default499 as LucideFileDiff,
  default500 as LucideFileDigit,
  default501 as LucideFileDown,
  default1564 as LucideFileEdit,
  default502 as LucideFileHeart,
  default503 as LucideFileImage,
  default504 as LucideFileInput,
  default506 as LucideFileJson,
  default505 as LucideFileJson2,
  default508 as LucideFileKey,
  default507 as LucideFileKey2,
  default1554 as LucideFileLineChart,
  default510 as LucideFileLock,
  default509 as LucideFileLock2,
  default512 as LucideFileMinus,
  default511 as LucideFileMinus2,
  default513 as LucideFileMusic,
  default514 as LucideFileOutput,
  default1563 as LucideFilePen,
  default1561 as LucideFilePenLine,
  default1558 as LucideFilePieChart,
  default516 as LucideFilePlus,
  default515 as LucideFilePlus2,
  default517 as LucideFileQuestion,
  default518 as LucideFileScan,
  default520 as LucideFileSearch,
  default519 as LucideFileSearch2,
  default1562 as LucideFileSignature,
  default521 as LucideFileSliders,
  default522 as LucideFileSpreadsheet,
  default523 as LucideFileStack,
  default524 as LucideFileSymlink,
  default526 as LucideFileTerminal,
  default525 as LucideFileText,
  default528 as LucideFileType,
  default527 as LucideFileType2,
  default529 as LucideFileUp,
  default530 as LucideFileUser,
  default532 as LucideFileVideo,
  default531 as LucideFileVideo2,
  default534 as LucideFileVolume,
  default533 as LucideFileVolume2,
  default535 as LucideFileWarning,
  default537 as LucideFileX,
  default536 as LucideFileX2,
  default539 as LucideFiles,
  default540 as LucideFilm,
  default1572 as LucideFilter,
  default1570 as LucideFilterX,
  default541 as LucideFingerprint,
  default542 as LucideFireExtinguisher,
  default545 as LucideFish,
  default543 as LucideFishOff,
  default544 as LucideFishSymbol,
  default549 as LucideFlag,
  default546 as LucideFlagOff,
  default547 as LucideFlagTriangleLeft,
  default548 as LucideFlagTriangleRight,
  default551 as LucideFlame,
  default550 as LucideFlameKindling,
  default553 as LucideFlashlight,
  default552 as LucideFlashlightOff,
  default555 as LucideFlaskConical,
  default554 as LucideFlaskConicalOff,
  default556 as LucideFlaskRound,
  default557 as LucideFlipHorizontal,
  default558 as LucideFlipHorizontal2,
  default560 as LucideFlipVertical,
  default559 as LucideFlipVertical2,
  default562 as LucideFlower,
  default561 as LucideFlower2,
  default563 as LucideFocus,
  default564 as LucideFoldHorizontal,
  default565 as LucideFoldVertical,
  default593 as LucideFolder,
  default566 as LucideFolderArchive,
  default567 as LucideFolderCheck,
  default568 as LucideFolderClock,
  default569 as LucideFolderClosed,
  default570 as LucideFolderCode,
  default1565 as LucideFolderCog,
  default1566 as LucideFolderCog2,
  default571 as LucideFolderDot,
  default572 as LucideFolderDown,
  default1568 as LucideFolderEdit,
  default574 as LucideFolderGit,
  default573 as LucideFolderGit2,
  default575 as LucideFolderHeart,
  default576 as LucideFolderInput,
  default577 as LucideFolderKanban,
  default578 as LucideFolderKey,
  default579 as LucideFolderLock,
  default580 as LucideFolderMinus,
  default582 as LucideFolderOpen,
  default581 as LucideFolderOpenDot,
  default583 as LucideFolderOutput,
  default1567 as LucideFolderPen,
  default584 as LucideFolderPlus,
  default585 as LucideFolderRoot,
  default587 as LucideFolderSearch,
  default586 as LucideFolderSearch2,
  default588 as LucideFolderSymlink,
  default589 as LucideFolderSync,
  default590 as LucideFolderTree,
  default591 as LucideFolderUp,
  default592 as LucideFolderX,
  default594 as LucideFolders,
  default596 as LucideFootprints,
  default1802 as LucideForkKnife,
  default1800 as LucideForkKnifeCrossed,
  default595 as LucideForklift,
  default1643 as LucideFormInput,
  default597 as LucideForward,
  default599 as LucideFrame,
  default598 as LucideFramer,
  default601 as LucideFrown,
  default600 as LucideFuel,
  default602 as LucideFullscreen,
  default1720 as LucideFunctionSquare,
  default1571 as LucideFunnel,
  default603 as LucideFunnelPlus,
  default1569 as LucideFunnelX,
  default605 as LucideGalleryHorizontal,
  default604 as LucideGalleryHorizontalEnd,
  default606 as LucideGalleryThumbnails,
  default608 as LucideGalleryVertical,
  default607 as LucideGalleryVerticalEnd,
  default610 as LucideGamepad,
  default609 as LucideGamepad2,
  default1455 as LucideGanttChart,
  default1691 as LucideGanttChartSquare,
  default611 as LucideGauge,
  default1493 as LucideGaugeCircle,
  default612 as LucideGavel,
  default613 as LucideGem,
  default614 as LucideGhost,
  default615 as LucideGift,
  default616 as LucideGitBranch,
  default617 as LucideGitBranchPlus,
  default1574 as LucideGitCommit,
  default1573 as LucideGitCommitHorizontal,
  default618 as LucideGitCommitVertical,
  default620 as LucideGitCompare,
  default619 as LucideGitCompareArrows,
  default621 as LucideGitFork,
  default622 as LucideGitGraph,
  default623 as LucideGitMerge,
  default629 as LucideGitPullRequest,
  default624 as LucideGitPullRequestArrow,
  default625 as LucideGitPullRequestClosed,
  default626 as LucideGitPullRequestCreate,
  default627 as LucideGitPullRequestCreateArrow,
  default628 as LucideGitPullRequestDraft,
  default630 as LucideGithub,
  default631 as LucideGitlab,
  default632 as LucideGlassWater,
  default633 as LucideGlasses,
  default635 as LucideGlobe,
  default1544 as LucideGlobe2,
  default634 as LucideGlobeLock,
  default636 as LucideGoal,
  default637 as LucideGpu,
  default638 as LucideGrab,
  default639 as LucideGraduationCap,
  default640 as LucideGrape,
  default1584 as LucideGrid,
  default1582 as LucideGrid2X2,
  default1576 as LucideGrid2X2Check,
  default1578 as LucideGrid2X2Plus,
  default1580 as LucideGrid2X2X,
  default1581 as LucideGrid2x2,
  default1575 as LucideGrid2x2Check,
  default1577 as LucideGrid2x2Plus,
  default1579 as LucideGrid2x2X,
  default1585 as LucideGrid3X3,
  default641 as LucideGrid3x2,
  default1583 as LucideGrid3x3,
  default644 as LucideGrip,
  default642 as LucideGripHorizontal,
  default643 as LucideGripVertical,
  default645 as LucideGroup,
  default646 as LucideGuitar,
  default647 as LucideHam,
  default648 as LucideHamburger,
  default649 as LucideHammer,
  default654 as LucideHand,
  default650 as LucideHandCoins,
  default651 as LucideHandHeart,
  default1586 as LucideHandHelping,
  default652 as LucideHandMetal,
  default653 as LucideHandPlatter,
  default656 as LucideHandshake,
  default658 as LucideHardDrive,
  default655 as LucideHardDriveDownload,
  default657 as LucideHardDriveUpload,
  default659 as LucideHardHat,
  default660 as LucideHash,
  default662 as LucideHaze,
  default661 as LucideHdmiPort,
  default669 as LucideHeading,
  default663 as LucideHeading1,
  default664 as LucideHeading2,
  default665 as LucideHeading3,
  default666 as LucideHeading4,
  default667 as LucideHeading5,
  default668 as LucideHeading6,
  default670 as LucideHeadphoneOff,
  default671 as LucideHeadphones,
  default672 as LucideHeadset,
  default679 as LucideHeart,
  default673 as LucideHeartCrack,
  default674 as LucideHeartHandshake,
  default675 as LucideHeartMinus,
  default676 as LucideHeartOff,
  default677 as LucideHeartPlus,
  default678 as LucideHeartPulse,
  default680 as LucideHeater,
  default1495 as LucideHelpCircle,
  default1587 as LucideHelpingHand,
  default681 as LucideHexagon,
  default682 as LucideHighlighter,
  default683 as LucideHistory,
  default1591 as LucideHome,
  default685 as LucideHop,
  default684 as LucideHopOff,
  default686 as LucideHospital,
  default687 as LucideHotel,
  default688 as LucideHourglass,
  default1590 as LucideHouse,
  default689 as LucideHousePlug,
  default690 as LucideHousePlus,
  default691 as LucideHouseWifi,
  default1593 as LucideIceCream,
  default1589 as LucideIceCream2,
  default1588 as LucideIceCreamBowl,
  default1592 as LucideIceCreamCone,
  default692 as LucideIdCard,
  default693 as LucideIdCardLanyard,
  default701 as LucideImage,
  default694 as LucideImageDown,
  default695 as LucideImageMinus,
  default696 as LucideImageOff,
  default697 as LucideImagePlay,
  default698 as LucideImagePlus,
  default699 as LucideImageUp,
  default700 as LucideImageUpscale,
  default702 as LucideImages,
  default704 as LucideImport,
  default703 as LucideInbox,
  default1597 as LucideIndent,
  default1594 as LucideIndentDecrease,
  default1596 as LucideIndentIncrease,
  default705 as LucideIndianRupee,
  default706 as LucideInfinity,
  default707 as LucideInfo,
  default1730 as LucideInspect,
  default708 as LucideInspectionPanel,
  default709 as LucideInstagram,
  default710 as LucideItalic,
  default711 as LucideIterationCcw,
  default712 as LucideIterationCw,
  default713 as LucideJapaneseYen,
  default714 as LucideJoystick,
  default715 as LucideKanban,
  default1722 as LucideKanbanSquare,
  default1708 as LucideKanbanSquareDashed,
  default718 as LucideKey,
  default716 as LucideKeyRound,
  default717 as LucideKeySquare,
  default722 as LucideKeyboard,
  default719 as LucideKeyboardMusic,
  default720 as LucideKeyboardOff,
  default727 as LucideLamp,
  default721 as LucideLampCeiling,
  default723 as LucideLampDesk,
  default724 as LucideLampFloor,
  default725 as LucideLampWallDown,
  default726 as LucideLampWallUp,
  default728 as LucideLandPlot,
  default729 as LucideLandmark,
  default730 as LucideLanguages,
  default732 as LucideLaptop,
  default1599 as LucideLaptop2,
  default1598 as LucideLaptopMinimal,
  default731 as LucideLaptopMinimalCheck,
  default734 as LucideLasso,
  default733 as LucideLassoSelect,
  default735 as LucideLaugh,
  default1600 as LucideLayers,
  default736 as LucideLayers2,
  default1601 as LucideLayers3,
  default1635 as LucideLayout,
  default738 as LucideLayoutDashboard,
  default737 as LucideLayoutGrid,
  default739 as LucideLayoutList,
  default741 as LucideLayoutPanelLeft,
  default740 as LucideLayoutPanelTop,
  default742 as LucideLayoutTemplate,
  default743 as LucideLeaf,
  default745 as LucideLeafyGreen,
  default744 as LucideLectern,
  default746 as LucideLetterText,
  default748 as LucideLibrary,
  default747 as LucideLibraryBig,
  default1726 as LucideLibrarySquare,
  default749 as LucideLifeBuoy,
  default750 as LucideLigature,
  default752 as LucideLightbulb,
  default751 as LucideLightbulbOff,
  default1449 as LucideLineChart,
  default755 as LucideLink,
  default754 as LucideLink2,
  default753 as LucideLink2Off,
  default756 as LucideLinkedin,
  default773 as LucideList,
  default757 as LucideListCheck,
  default758 as LucideListChecks,
  default760 as LucideListCollapse,
  default759 as LucideListEnd,
  default762 as LucideListFilter,
  default761 as LucideListFilterPlus,
  default763 as LucideListMinus,
  default764 as LucideListMusic,
  default765 as LucideListOrdered,
  default766 as LucideListPlus,
  default767 as LucideListRestart,
  default768 as LucideListStart,
  default769 as LucideListTodo,
  default770 as LucideListTree,
  default771 as LucideListVideo,
  default772 as LucideListX,
  default775 as LucideLoader,
  default1603 as LucideLoader2,
  default1602 as LucideLoaderCircle,
  default774 as LucideLoaderPinwheel,
  default778 as LucideLocate,
  default776 as LucideLocateFixed,
  default777 as LucideLocateOff,
  default779 as LucideLocationEdit,
  default781 as LucideLock,
  default780 as LucideLockKeyhole,
  default1604 as LucideLockKeyholeOpen,
  default1606 as LucideLockOpen,
  default782 as LucideLogIn,
  default783 as LucideLogOut,
  default784 as LucideLogs,
  default785 as LucideLollipop,
  default786 as LucideLuggage,
  default1724 as LucideMSquare,
  default787 as LucideMagnet,
  default796 as LucideMail,
  default788 as LucideMailCheck,
  default789 as LucideMailMinus,
  default790 as LucideMailOpen,
  default791 as LucideMailPlus,
  default792 as LucideMailQuestion,
  default793 as LucideMailSearch,
  default794 as LucideMailWarning,
  default795 as LucideMailX,
  default797 as LucideMailbox,
  default798 as LucideMails,
  default812 as LucideMap,
  default809 as LucideMapPin,
  default800 as LucideMapPinCheck,
  default799 as LucideMapPinCheckInside,
  default801 as LucideMapPinHouse,
  default803 as LucideMapPinMinus,
  default802 as LucideMapPinMinusInside,
  default804 as LucideMapPinOff,
  default806 as LucideMapPinPlus,
  default805 as LucideMapPinPlusInside,
  default808 as LucideMapPinX,
  default807 as LucideMapPinXInside,
  default810 as LucideMapPinned,
  default811 as LucideMapPlus,
  default814 as LucideMars,
  default813 as LucideMarsStroke,
  default815 as LucideMartini,
  default817 as LucideMaximize,
  default816 as LucideMaximize2,
  default818 as LucideMedal,
  default820 as LucideMegaphone,
  default819 as LucideMegaphoneOff,
  default821 as LucideMeh,
  default822 as LucideMemoryStick,
  default823 as LucideMenu,
  default1728 as LucideMenuSquare,
  default824 as LucideMerge,
  default835 as LucideMessageCircle,
  default826 as LucideMessageCircleCode,
  default828 as LucideMessageCircleDashed,
  default825 as LucideMessageCircleHeart,
  default827 as LucideMessageCircleMore,
  default829 as LucideMessageCircleOff,
  default830 as LucideMessageCirclePlus,
  default831 as LucideMessageCircleQuestion,
  default832 as LucideMessageCircleReply,
  default833 as LucideMessageCircleWarning,
  default834 as LucideMessageCircleX,
  default851 as LucideMessageSquare,
  default836 as LucideMessageSquareCode,
  default837 as LucideMessageSquareDashed,
  default838 as LucideMessageSquareDiff,
  default839 as LucideMessageSquareDot,
  default840 as LucideMessageSquareHeart,
  default841 as LucideMessageSquareLock,
  default842 as LucideMessageSquareMore,
  default843 as LucideMessageSquareOff,
  default845 as LucideMessageSquarePlus,
  default844 as LucideMessageSquareQuote,
  default846 as LucideMessageSquareReply,
  default847 as LucideMessageSquareShare,
  default848 as LucideMessageSquareText,
  default849 as LucideMessageSquareWarning,
  default850 as LucideMessageSquareX,
  default852 as LucideMessagesSquare,
  default854 as LucideMic,
  default1609 as LucideMic2,
  default853 as LucideMicOff,
  default1608 as LucideMicVocal,
  default855 as LucideMicrochip,
  default856 as LucideMicroscope,
  default857 as LucideMicrowave,
  default858 as LucideMilestone,
  default859 as LucideMilk,
  default860 as LucideMilkOff,
  default862 as LucideMinimize,
  default861 as LucideMinimize2,
  default863 as LucideMinus,
  default1497 as LucideMinusCircle,
  default1732 as LucideMinusSquare,
  default876 as LucideMonitor,
  default864 as LucideMonitorCheck,
  default866 as LucideMonitorCog,
  default865 as LucideMonitorDot,
  default867 as LucideMonitorDown,
  default868 as LucideMonitorOff,
  default869 as LucideMonitorPause,
  default870 as LucideMonitorPlay,
  default871 as LucideMonitorSmartphone,
  default872 as LucideMonitorSpeaker,
  default873 as LucideMonitorStop,
  default874 as LucideMonitorUp,
  default875 as LucideMonitorX,
  default878 as LucideMoon,
  default877 as LucideMoonStar,
  default1548 as LucideMoreHorizontal,
  default1546 as LucideMoreVertical,
  default880 as LucideMountain,
  default879 as LucideMountainSnow,
  default886 as LucideMouse,
  default881 as LucideMouseOff,
  default885 as LucideMousePointer,
  default882 as LucideMousePointer2,
  default883 as LucideMousePointerBan,
  default884 as LucideMousePointerClick,
  default1710 as LucideMousePointerSquareDashed,
  default898 as LucideMove,
  default1611 as LucideMove3D,
  default1610 as LucideMove3d,
  default888 as LucideMoveDiagonal,
  default887 as LucideMoveDiagonal2,
  default891 as LucideMoveDown,
  default889 as LucideMoveDownLeft,
  default890 as LucideMoveDownRight,
  default893 as LucideMoveHorizontal,
  default892 as LucideMoveLeft,
  default895 as LucideMoveRight,
  default897 as LucideMoveUp,
  default894 as LucideMoveUpLeft,
  default896 as LucideMoveUpRight,
  default899 as LucideMoveVertical,
  default903 as LucideMusic,
  default900 as LucideMusic2,
  default901 as LucideMusic3,
  default902 as LucideMusic4,
  default907 as LucideNavigation,
  default905 as LucideNavigation2,
  default904 as LucideNavigation2Off,
  default906 as LucideNavigationOff,
  default908 as LucideNetwork,
  default909 as LucideNewspaper,
  default910 as LucideNfc,
  default911 as LucideNonBinary,
  default916 as LucideNotebook,
  default912 as LucideNotebookPen,
  default913 as LucideNotebookTabs,
  default914 as LucideNotebookText,
  default917 as LucideNotepadText,
  default915 as LucideNotepadTextDashed,
  default920 as LucideNut,
  default918 as LucideNutOff,
  default921 as LucideOctagon,
  default1612 as LucideOctagonAlert,
  default919 as LucideOctagonMinus,
  default1614 as LucideOctagonPause,
  default1616 as LucideOctagonX,
  default923 as LucideOmega,
  default922 as LucideOption,
  default924 as LucideOrbit,
  default925 as LucideOrigami,
  default1595 as LucideOutdent,
  default932 as LucidePackage,
  default926 as LucidePackage2,
  default927 as LucidePackageCheck,
  default928 as LucidePackageMinus,
  default929 as LucidePackageOpen,
  default930 as LucidePackagePlus,
  default933 as LucidePackageSearch,
  default931 as LucidePackageX,
  default934 as LucidePaintBucket,
  default936 as LucidePaintRoller,
  default935 as LucidePaintbrush,
  default1619 as LucidePaintbrush2,
  default1618 as LucidePaintbrushVertical,
  default937 as LucidePalette,
  default1778 as LucidePalmtree,
  default938 as LucidePanda,
  default941 as LucidePanelBottom,
  default939 as LucidePanelBottomClose,
  default1620 as LucidePanelBottomDashed,
  default1621 as LucidePanelBottomInactive,
  default940 as LucidePanelBottomOpen,
  default1628 as LucidePanelLeft,
  default1624 as LucidePanelLeftClose,
  default1622 as LucidePanelLeftDashed,
  default1623 as LucidePanelLeftInactive,
  default1626 as LucidePanelLeftOpen,
  default944 as LucidePanelRight,
  default942 as LucidePanelRightClose,
  default1630 as LucidePanelRightDashed,
  default1631 as LucidePanelRightInactive,
  default943 as LucidePanelRightOpen,
  default946 as LucidePanelTop,
  default945 as LucidePanelTopClose,
  default1632 as LucidePanelTopDashed,
  default1633 as LucidePanelTopInactive,
  default947 as LucidePanelTopOpen,
  default948 as LucidePanelsLeftBottom,
  default1538 as LucidePanelsLeftRight,
  default949 as LucidePanelsRightBottom,
  default1649 as LucidePanelsTopBottom,
  default1634 as LucidePanelsTopLeft,
  default950 as LucidePaperclip,
  default951 as LucideParentheses,
  default1501 as LucideParkingCircle,
  default1499 as LucideParkingCircleOff,
  default952 as LucideParkingMeter,
  default1736 as LucideParkingSquare,
  default1734 as LucideParkingSquareOff,
  default953 as LucidePartyPopper,
  default954 as LucidePause,
  default1505 as LucidePauseCircle,
  default1615 as LucidePauseOctagon,
  default957 as LucidePawPrint,
  default955 as LucidePcCase,
  default1638 as LucidePen,
  default1738 as LucidePenBox,
  default1636 as LucidePenLine,
  default956 as LucidePenOff,
  default1739 as LucidePenSquare,
  default958 as LucidePenTool,
  default962 as LucidePencil,
  default959 as LucidePencilLine,
  default960 as LucidePencilOff,
  default961 as LucidePencilRuler,
  default963 as LucidePentagon,
  default964 as LucidePercent,
  default1503 as LucidePercentCircle,
  default1542 as LucidePercentDiamond,
  default1742 as LucidePercentSquare,
  default965 as LucidePersonStanding,
  default966 as LucidePhilippinePeso,
  default972 as LucidePhone,
  default967 as LucidePhoneCall,
  default968 as LucidePhoneForwarded,
  default969 as LucidePhoneIncoming,
  default970 as LucidePhoneMissed,
  default971 as LucidePhoneOff,
  default973 as LucidePhoneOutgoing,
  default974 as LucidePi,
  default1744 as LucidePiSquare,
  default975 as LucidePiano,
  default976 as LucidePickaxe,
  default978 as LucidePictureInPicture,
  default977 as LucidePictureInPicture2,
  default1457 as LucidePieChart,
  default979 as LucidePiggyBank,
  default982 as LucidePilcrow,
  default980 as LucidePilcrowLeft,
  default981 as LucidePilcrowRight,
  default1746 as LucidePilcrowSquare,
  default984 as LucidePill,
  default983 as LucidePillBottle,
  default986 as LucidePin,
  default985 as LucidePinOff,
  default987 as LucidePipette,
  default988 as LucidePizza,
  default990 as LucidePlane,
  default989 as LucidePlaneLanding,
  default991 as LucidePlaneTakeoff,
  default992 as LucidePlay,
  default1507 as LucidePlayCircle,
  default1748 as LucidePlaySquare,
  default993 as LucidePlug,
  default994 as LucidePlug2,
  default1640 as LucidePlugZap,
  default1641 as LucidePlugZap2,
  default995 as LucidePlus,
  default1509 as LucidePlusCircle,
  default1750 as LucidePlusSquare,
  default996 as LucidePocket,
  default997 as LucidePocketKnife,
  default998 as LucidePodcast,
  default1000 as LucidePointer,
  default999 as LucidePointerOff,
  default1001 as LucidePopcorn,
  default1002 as LucidePopsicle,
  default1003 as LucidePoundSterling,
  default1005 as LucidePower,
  default1511 as LucidePowerCircle,
  default1004 as LucidePowerOff,
  default1752 as LucidePowerSquare,
  default1006 as LucidePresentation,
  default1007 as LucidePrinter,
  default1008 as LucidePrinterCheck,
  default1009 as LucideProjector,
  default1011 as LucideProportions,
  default1010 as LucidePuzzle,
  default1012 as LucidePyramid,
  default1013 as LucideQrCode,
  default1014 as LucideQuote,
  default1015 as LucideRabbit,
  default1016 as LucideRadar,
  default1017 as LucideRadiation,
  default1018 as LucideRadical,
  default1021 as LucideRadio,
  default1019 as LucideRadioReceiver,
  default1020 as LucideRadioTower,
  default1022 as LucideRadius,
  default1023 as LucideRailSymbol,
  default1024 as LucideRainbow,
  default1025 as LucideRat,
  default1026 as LucideRatio,
  default1035 as LucideReceipt,
  default1027 as LucideReceiptCent,
  default1028 as LucideReceiptEuro,
  default1029 as LucideReceiptIndianRupee,
  default1030 as LucideReceiptJapaneseYen,
  default1031 as LucideReceiptPoundSterling,
  default1032 as LucideReceiptRussianRuble,
  default1033 as LucideReceiptSwissFranc,
  default1034 as LucideReceiptText,
  default1642 as LucideRectangleEllipsis,
  default1036 as LucideRectangleGoggles,
  default1037 as LucideRectangleHorizontal,
  default1038 as LucideRectangleVertical,
  default1039 as LucideRecycle,
  default1043 as LucideRedo,
  default1040 as LucideRedo2,
  default1041 as LucideRedoDot,
  default1044 as LucideRefreshCcw,
  default1042 as LucideRefreshCcwDot,
  default1046 as LucideRefreshCw,
  default1045 as LucideRefreshCwOff,
  default1047 as LucideRefrigerator,
  default1048 as LucideRegex,
  default1051 as LucideRemoveFormatting,
  default1052 as LucideRepeat,
  default1049 as LucideRepeat1,
  default1050 as LucideRepeat2,
  default1054 as LucideReplace,
  default1053 as LucideReplaceAll,
  default1056 as LucideReply,
  default1055 as LucideReplyAll,
  default1057 as LucideRewind,
  default1058 as LucideRibbon,
  default1059 as LucideRocket,
  default1060 as LucideRockingChair,
  default1062 as LucideRollerCoaster,
  default1645 as LucideRotate3D,
  default1644 as LucideRotate3d,
  default1064 as LucideRotateCcw,
  default1061 as LucideRotateCcwKey,
  default1063 as LucideRotateCcwSquare,
  default1066 as LucideRotateCw,
  default1065 as LucideRotateCwSquare,
  default1068 as LucideRoute,
  default1067 as LucideRouteOff,
  default1069 as LucideRouter,
  default1647 as LucideRows,
  default1646 as LucideRows2,
  default1648 as LucideRows3,
  default1070 as LucideRows4,
  default1071 as LucideRss,
  default1073 as LucideRuler,
  default1072 as LucideRulerDimensionLine,
  default1074 as LucideRussianRuble,
  default1075 as LucideSailboat,
  default1076 as LucideSalad,
  default1078 as LucideSandwich,
  default1079 as LucideSatellite,
  default1077 as LucideSatelliteDish,
  default1080 as LucideSaudiRiyal,
  default1083 as LucideSave,
  default1081 as LucideSaveAll,
  default1082 as LucideSaveOff,
  default1084 as LucideScale,
  default1651 as LucideScale3D,
  default1650 as LucideScale3d,
  default1085 as LucideScaling,
  default1094 as LucideScan,
  default1086 as LucideScanBarcode,
  default1087 as LucideScanEye,
  default1088 as LucideScanFace,
  default1090 as LucideScanHeart,
  default1091 as LucideScanLine,
  default1089 as LucideScanQrCode,
  default1092 as LucideScanSearch,
  default1093 as LucideScanText,
  default1459 as LucideScatterChart,
  default1095 as LucideSchool,
  default1784 as LucideSchool2,
  default1097 as LucideScissors,
  default1096 as LucideScissorsLineDashed,
  default1754 as LucideScissorsSquare,
  default1689 as LucideScissorsSquareDashedBottom,
  default1099 as LucideScreenShare,
  default1098 as LucideScreenShareOff,
  default1101 as LucideScroll,
  default1100 as LucideScrollText,
  default1106 as LucideSearch,
  default1102 as LucideSearchCheck,
  default1103 as LucideSearchCode,
  default1104 as LucideSearchSlash,
  default1105 as LucideSearchX,
  default1107 as LucideSection,
  default1109 as LucideSend,
  default1653 as LucideSendHorizonal,
  default1652 as LucideSendHorizontal,
  default1108 as LucideSendToBack,
  default1110 as LucideSeparatorHorizontal,
  default1111 as LucideSeparatorVertical,
  default1115 as LucideServer,
  default1112 as LucideServerCog,
  default1113 as LucideServerCrash,
  default1114 as LucideServerOff,
  default1117 as LucideSettings,
  default1116 as LucideSettings2,
  default1118 as LucideShapes,
  default1120 as LucideShare,
  default1119 as LucideShare2,
  default1121 as LucideSheet,
  default1122 as LucideShell,
  default1133 as LucideShield,
  default1123 as LucideShieldAlert,
  default1124 as LucideShieldBan,
  default1125 as LucideShieldCheck,
  default1655 as LucideShieldClose,
  default1126 as LucideShieldEllipsis,
  default1127 as LucideShieldHalf,
  default1129 as LucideShieldMinus,
  default1128 as LucideShieldOff,
  default1130 as LucideShieldPlus,
  default1131 as LucideShieldQuestion,
  default1132 as LucideShieldUser,
  default1654 as LucideShieldX,
  default1136 as LucideShip,
  default1134 as LucideShipWheel,
  default1135 as LucideShirt,
  default1137 as LucideShoppingBag,
  default1138 as LucideShoppingBasket,
  default1140 as LucideShoppingCart,
  default1139 as LucideShovel,
  default1141 as LucideShowerHead,
  default1142 as LucideShredder,
  default1143 as LucideShrimp,
  default1144 as LucideShrink,
  default1146 as LucideShrub,
  default1145 as LucideShuffle,
  default1629 as LucideSidebar,
  default1625 as LucideSidebarClose,
  default1627 as LucideSidebarOpen,
  default1147 as LucideSigma,
  default1758 as LucideSigmaSquare,
  default1152 as LucideSignal,
  default1148 as LucideSignalHigh,
  default1150 as LucideSignalLow,
  default1149 as LucideSignalMedium,
  default1151 as LucideSignalZero,
  default1153 as LucideSignature,
  default1155 as LucideSignpost,
  default1154 as LucideSignpostBig,
  default1156 as LucideSiren,
  default1158 as LucideSkipBack,
  default1157 as LucideSkipForward,
  default1159 as LucideSkull,
  default1160 as LucideSlack,
  default1161 as LucideSlash,
  default1756 as LucideSlashSquare,
  default1162 as LucideSlice,
  default1657 as LucideSliders,
  default1164 as LucideSlidersHorizontal,
  default1656 as LucideSlidersVertical,
  default1166 as LucideSmartphone,
  default1163 as LucideSmartphoneCharging,
  default1165 as LucideSmartphoneNfc,
  default1168 as LucideSmile,
  default1167 as LucideSmilePlus,
  default1169 as LucideSnail,
  default1170 as LucideSnowflake,
  default1171 as LucideSoapDispenserDroplet,
  default1172 as LucideSofa,
  default1417 as LucideSortAsc,
  default1411 as LucideSortDesc,
  default1173 as LucideSoup,
  default1174 as LucideSpace,
  default1175 as LucideSpade,
  default1176 as LucideSparkle,
  default1658 as LucideSparkles,
  default1177 as LucideSpeaker,
  default1178 as LucideSpeech,
  default1180 as LucideSpellCheck,
  default1179 as LucideSpellCheck2,
  default1182 as LucideSpline,
  default1181 as LucideSplinePointer,
  default1183 as LucideSplit,
  default1760 as LucideSplitSquareHorizontal,
  default1762 as LucideSplitSquareVertical,
  default1184 as LucideSprayCan,
  default1185 as LucideSprout,
  default1193 as LucideSquare,
  default1660 as LucideSquareActivity,
  default1668 as LucideSquareArrowDown,
  default1662 as LucideSquareArrowDownLeft,
  default1664 as LucideSquareArrowDownRight,
  default1666 as LucideSquareArrowLeft,
  default1670 as LucideSquareArrowOutDownLeft,
  default1672 as LucideSquareArrowOutDownRight,
  default1674 as LucideSquareArrowOutUpLeft,
  default1676 as LucideSquareArrowOutUpRight,
  default1678 as LucideSquareArrowRight,
  default1684 as LucideSquareArrowUp,
  default1682 as LucideSquareArrowUpLeft,
  default1680 as LucideSquareArrowUpRight,
  default1686 as LucideSquareAsterisk,
  default1688 as LucideSquareBottomDashedScissors,
  default1690 as LucideSquareChartGantt,
  default1693 as LucideSquareCheck,
  default1699 as LucideSquareCheckBig,
  default1695 as LucideSquareChevronDown,
  default1697 as LucideSquareChevronLeft,
  default1701 as LucideSquareChevronRight,
  default1703 as LucideSquareChevronUp,
  default1705 as LucideSquareCode,
  default1711 as LucideSquareDashed,
  default1187 as LucideSquareDashedBottom,
  default1186 as LucideSquareDashedBottomCode,
  default1707 as LucideSquareDashedKanban,
  default1709 as LucideSquareDashedMousePointer,
  default1188 as LucideSquareDashedTopSolid,
  default1713 as LucideSquareDivide,
  default1715 as LucideSquareDot,
  default1717 as LucideSquareEqual,
  default1719 as LucideSquareFunction,
  default1692 as LucideSquareGanttChart,
  default1721 as LucideSquareKanban,
  default1725 as LucideSquareLibrary,
  default1723 as LucideSquareM,
  default1727 as LucideSquareMenu,
  default1731 as LucideSquareMinus,
  default1729 as LucideSquareMousePointer,
  default1735 as LucideSquareParking,
  default1733 as LucideSquareParkingOff,
  default1737 as LucideSquarePen,
  default1741 as LucideSquarePercent,
  default1743 as LucideSquarePi,
  default1745 as LucideSquarePilcrow,
  default1747 as LucideSquarePlay,
  default1749 as LucideSquarePlus,
  default1751 as LucideSquarePower,
  default1189 as LucideSquareRadical,
  default1190 as LucideSquareRoundCorner,
  default1753 as LucideSquareScissors,
  default1757 as LucideSquareSigma,
  default1755 as LucideSquareSlash,
  default1759 as LucideSquareSplitHorizontal,
  default1761 as LucideSquareSplitVertical,
  default1191 as LucideSquareSquare,
  default1192 as LucideSquareStack,
  default1765 as LucideSquareTerminal,
  default1767 as LucideSquareUser,
  default1763 as LucideSquareUserRound,
  default1769 as LucideSquareX,
  default1194 as LucideSquaresExclude,
  default1196 as LucideSquaresIntersect,
  default1195 as LucideSquaresSubtract,
  default1197 as LucideSquaresUnite,
  default1198 as LucideSquircle,
  default1200 as LucideSquirrel,
  default1199 as LucideStamp,
  default1203 as LucideStar,
  default1201 as LucideStarHalf,
  default1202 as LucideStarOff,
  default1659 as LucideStars,
  default1205 as LucideStepBack,
  default1204 as LucideStepForward,
  default1206 as LucideStethoscope,
  default1207 as LucideSticker,
  default1208 as LucideStickyNote,
  default1517 as LucideStopCircle,
  default1209 as LucideStore,
  default1211 as LucideStretchHorizontal,
  default1210 as LucideStretchVertical,
  default1212 as LucideStrikethrough,
  default1213 as LucideSubscript,
  default1433 as LucideSubtitles,
  default1220 as LucideSun,
  default1216 as LucideSunDim,
  default1214 as LucideSunMedium,
  default1215 as LucideSunMoon,
  default1217 as LucideSunSnow,
  default1218 as LucideSunrise,
  default1219 as LucideSunset,
  default1221 as LucideSuperscript,
  default1222 as LucideSwatchBook,
  default1224 as LucideSwissFranc,
  default1223 as LucideSwitchCamera,
  default1226 as LucideSword,
  default1225 as LucideSwords,
  default1227 as LucideSyringe,
  default1236 as LucideTable,
  default1228 as LucideTable2,
  default1230 as LucideTableCellsMerge,
  default1229 as LucideTableCellsSplit,
  default1232 as LucideTableColumnsSplit,
  default1536 as LucideTableConfig,
  default1233 as LucideTableOfContents,
  default1231 as LucideTableProperties,
  default1234 as LucideTableRowsSplit,
  default1238 as LucideTablet,
  default1235 as LucideTabletSmartphone,
  default1237 as LucideTablets,
  default1240 as LucideTag,
  default1239 as LucideTags,
  default1241 as LucideTally1,
  default1242 as LucideTally2,
  default1243 as LucideTally3,
  default1244 as LucideTally4,
  default1245 as LucideTally5,
  default1246 as LucideTangent,
  default1247 as LucideTarget,
  default1248 as LucideTelescope,
  default1249 as LucideTent,
  default1251 as LucideTentTree,
  default1250 as LucideTerminal,
  default1766 as LucideTerminalSquare,
  default1252 as LucideTestTube,
  default1772 as LucideTestTube2,
  default1771 as LucideTestTubeDiagonal,
  default1253 as LucideTestTubes,
  default1258 as LucideText,
  default1254 as LucideTextCursor,
  default1255 as LucideTextCursorInput,
  default1257 as LucideTextQuote,
  default1256 as LucideTextSearch,
  default1773 as LucideTextSelect,
  default1774 as LucideTextSelection,
  default1259 as LucideTheater,
  default1263 as LucideThermometer,
  default1260 as LucideThermometerSnowflake,
  default1261 as LucideThermometerSun,
  default1262 as LucideThumbsDown,
  default1264 as LucideThumbsUp,
  default1271 as LucideTicket,
  default1265 as LucideTicketCheck,
  default1266 as LucideTicketMinus,
  default1267 as LucideTicketPercent,
  default1268 as LucideTicketPlus,
  default1269 as LucideTicketSlash,
  default1270 as LucideTicketX,
  default1273 as LucideTickets,
  default1272 as LucideTicketsPlane,
  default1276 as LucideTimer,
  default1274 as LucideTimerOff,
  default1275 as LucideTimerReset,
  default1277 as LucideToggleLeft,
  default1278 as LucideToggleRight,
  default1279 as LucideToilet,
  default1280 as LucideTornado,
  default1281 as LucideTorus,
  default1283 as LucideTouchpad,
  default1282 as LucideTouchpadOff,
  default1284 as LucideTowerControl,
  default1285 as LucideToyBrick,
  default1286 as LucideTractor,
  default1287 as LucideTrafficCone,
  default1776 as LucideTrain,
  default1289 as LucideTrainFront,
  default1288 as LucideTrainFrontTunnel,
  default1290 as LucideTrainTrack,
  default1775 as LucideTramFront,
  default1291 as LucideTransgender,
  default1293 as LucideTrash,
  default1292 as LucideTrash2,
  default1294 as LucideTreeDeciduous,
  default1777 as LucideTreePalm,
  default1295 as LucideTreePine,
  default1296 as LucideTrees,
  default1297 as LucideTrello,
  default1298 as LucideTrendingDown,
  default1300 as LucideTrendingUp,
  default1299 as LucideTrendingUpDown,
  default1303 as LucideTriangle,
  default1779 as LucideTriangleAlert,
  default1302 as LucideTriangleDashed,
  default1301 as LucideTriangleRight,
  default1304 as LucideTrophy,
  default1305 as LucideTruck,
  default1306 as LucideTruckElectric,
  default1307 as LucideTurtle,
  default1310 as LucideTv,
  default1782 as LucideTv2,
  default1781 as LucideTvMinimal,
  default1308 as LucideTvMinimalPlay,
  default1309 as LucideTwitch,
  default1311 as LucideTwitter,
  default1314 as LucideType,
  default1312 as LucideTypeOutline,
  default1315 as LucideUmbrella,
  default1313 as LucideUmbrellaOff,
  default1316 as LucideUnderline,
  default1319 as LucideUndo,
  default1317 as LucideUndo2,
  default1318 as LucideUndoDot,
  default1320 as LucideUnfoldHorizontal,
  default1322 as LucideUnfoldVertical,
  default1321 as LucideUngroup,
  default1783 as LucideUniversity,
  default1324 as LucideUnlink,
  default1323 as LucideUnlink2,
  default1607 as LucideUnlock,
  default1605 as LucideUnlockKeyhole,
  default1325 as LucideUnplug,
  default1326 as LucideUpload,
  default1529 as LucideUploadCloud,
  default1327 as LucideUsb,
  default1338 as LucideUser,
  default1796 as LucideUser2,
  default1329 as LucideUserCheck,
  default1786 as LucideUserCheck2,
  default1519 as LucideUserCircle,
  default1515 as LucideUserCircle2,
  default1328 as LucideUserCog,
  default1788 as LucideUserCog2,
  default1330 as LucideUserLock,
  default1331 as LucideUserMinus,
  default1790 as LucideUserMinus2,
  default1332 as LucideUserPen,
  default1333 as LucideUserPlus,
  default1792 as LucideUserPlus2,
  default1795 as LucideUserRound,
  default1785 as LucideUserRoundCheck,
  default1787 as LucideUserRoundCog,
  default1789 as LucideUserRoundMinus,
  default1334 as LucideUserRoundPen,
  default1791 as LucideUserRoundPlus,
  default1335 as LucideUserRoundSearch,
  default1793 as LucideUserRoundX,
  default1336 as LucideUserSearch,
  default1768 as LucideUserSquare,
  default1764 as LucideUserSquare2,
  default1337 as LucideUserX,
  default1794 as LucideUserX2,
  default1339 as LucideUsers,
  default1798 as LucideUsers2,
  default1797 as LucideUsersRound,
  default1801 as LucideUtensils,
  default1799 as LucideUtensilsCrossed,
  default1340 as LucideUtilityPole,
  default1341 as LucideVariable,
  default1342 as LucideVault,
  default1343 as LucideVegan,
  default1345 as LucideVenetianMask,
  default1346 as LucideVenus,
  default1344 as LucideVenusAndMars,
  default1423 as LucideVerified,
  default1348 as LucideVibrate,
  default1347 as LucideVibrateOff,
  default1350 as LucideVideo,
  default1349 as LucideVideoOff,
  default1351 as LucideVideotape,
  default1352 as LucideView,
  default1353 as LucideVoicemail,
  default1354 as LucideVolleyball,
  default1359 as LucideVolume,
  default1355 as LucideVolume1,
  default1356 as LucideVolume2,
  default1357 as LucideVolumeOff,
  default1358 as LucideVolumeX,
  default1360 as LucideVote,
  default1362 as LucideWallet,
  default1804 as LucideWallet2,
  default1361 as LucideWalletCards,
  default1803 as LucideWalletMinimal,
  default1363 as LucideWallpaper,
  default1364 as LucideWand,
  default1806 as LucideWand2,
  default1805 as LucideWandSparkles,
  default1365 as LucideWarehouse,
  default1366 as LucideWashingMachine,
  default1367 as LucideWatch,
  default1369 as LucideWaves,
  default1368 as LucideWavesLadder,
  default1370 as LucideWaypoints,
  default1371 as LucideWebcam,
  default1372 as LucideWebhook,
  default1375 as LucideWebhookOff,
  default1373 as LucideWeight,
  default1376 as LucideWheat,
  default1374 as LucideWheatOff,
  default1377 as LucideWholeWord,
  default1383 as LucideWifi,
  default1378 as LucideWifiHigh,
  default1379 as LucideWifiLow,
  default1380 as LucideWifiOff,
  default1381 as LucideWifiPen,
  default1382 as LucideWifiZero,
  default1385 as LucideWind,
  default1384 as LucideWindArrowDown,
  default1388 as LucideWine,
  default1386 as LucideWineOff,
  default1387 as LucideWorkflow,
  default1389 as LucideWorm,
  default1390 as LucideWrapText,
  default1391 as LucideWrench,
  default1392 as LucideX,
  default1521 as LucideXCircle,
  default1617 as LucideXOctagon,
  default1770 as LucideXSquare,
  default1393 as LucideYoutube,
  default1395 as LucideZap,
  default1394 as LucideZapOff,
  default1396 as LucideZoomIn,
  default1397 as LucideZoomOut
};
