// .ignored/lucide-solid/dist/source/aliases/suffixed.jsx
/**
* @license lucide-solid v0.514.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/suffixed.ts
import { default as default2 } from "../icons/a-arrow-down";
import { default as default3 } from "../icons/a-arrow-up";
import { default as default4 } from "../icons/a-large-small";
import { default as default5 } from "../icons/accessibility";
import { default as default6 } from "../icons/activity";
import { default as default7 } from "../icons/air-vent";
import { default as default8 } from "../icons/airplay";
import { default as default9 } from "../icons/alarm-clock-off";
import { default as default10 } from "../icons/alarm-clock";
import { default as default11 } from "../icons/alarm-smoke";
import { default as default12 } from "../icons/album";
import { default as default13 } from "../icons/align-center-horizontal";
import { default as default14 } from "../icons/align-center-vertical";
import { default as default15 } from "../icons/align-center";
import { default as default16 } from "../icons/align-end-horizontal";
import { default as default17 } from "../icons/align-end-vertical";
import { default as default18 } from "../icons/align-horizontal-distribute-center";
import { default as default19 } from "../icons/align-horizontal-distribute-end";
import { default as default20 } from "../icons/align-horizontal-justify-center";
import { default as default21 } from "../icons/align-horizontal-distribute-start";
import { default as default22 } from "../icons/align-horizontal-justify-end";
import { default as default23 } from "../icons/align-horizontal-justify-start";
import { default as default24 } from "../icons/align-horizontal-space-around";
import { default as default25 } from "../icons/align-horizontal-space-between";
import { default as default26 } from "../icons/align-justify";
import { default as default27 } from "../icons/align-left";
import { default as default28 } from "../icons/align-right";
import { default as default29 } from "../icons/align-start-horizontal";
import { default as default30 } from "../icons/align-start-vertical";
import { default as default31 } from "../icons/align-vertical-distribute-center";
import { default as default32 } from "../icons/align-vertical-distribute-end";
import { default as default33 } from "../icons/align-vertical-distribute-start";
import { default as default34 } from "../icons/align-vertical-justify-center";
import { default as default35 } from "../icons/align-vertical-justify-start";
import { default as default36 } from "../icons/align-vertical-space-around";
import { default as default37 } from "../icons/align-vertical-justify-end";
import { default as default38 } from "../icons/align-vertical-space-between";
import { default as default39 } from "../icons/ambulance";
import { default as default40 } from "../icons/ampersand";
import { default as default41 } from "../icons/ampersands";
import { default as default42 } from "../icons/amphora";
import { default as default43 } from "../icons/anchor";
import { default as default44 } from "../icons/angry";
import { default as default45 } from "../icons/antenna";
import { default as default46 } from "../icons/annoyed";
import { default as default47 } from "../icons/anvil";
import { default as default48 } from "../icons/aperture";
import { default as default49 } from "../icons/app-window-mac";
import { default as default50 } from "../icons/apple";
import { default as default51 } from "../icons/archive-restore";
import { default as default52 } from "../icons/app-window";
import { default as default53 } from "../icons/archive-x";
import { default as default54 } from "../icons/archive";
import { default as default55 } from "../icons/arrow-big-down-dash";
import { default as default56 } from "../icons/armchair";
import { default as default57 } from "../icons/arrow-big-left-dash";
import { default as default58 } from "../icons/arrow-big-down";
import { default as default59 } from "../icons/arrow-big-right-dash";
import { default as default60 } from "../icons/arrow-big-left";
import { default as default61 } from "../icons/arrow-big-right";
import { default as default62 } from "../icons/arrow-big-up-dash";
import { default as default63 } from "../icons/arrow-big-up";
import { default as default64 } from "../icons/arrow-down-from-line";
import { default as default65 } from "../icons/arrow-down-left";
import { default as default66 } from "../icons/arrow-down-narrow-wide";
import { default as default67 } from "../icons/arrow-down-right";
import { default as default68 } from "../icons/arrow-down-up";
import { default as default69 } from "../icons/arrow-down-to-dot";
import { default as default70 } from "../icons/arrow-down-to-line";
import { default as default71 } from "../icons/arrow-down";
import { default as default72 } from "../icons/arrow-left-from-line";
import { default as default73 } from "../icons/arrow-left-right";
import { default as default74 } from "../icons/arrow-left-to-line";
import { default as default75 } from "../icons/arrow-left";
import { default as default76 } from "../icons/arrow-right-from-line";
import { default as default77 } from "../icons/arrow-right-to-line";
import { default as default78 } from "../icons/arrow-right-left";
import { default as default79 } from "../icons/arrow-right";
import { default as default80 } from "../icons/arrow-up-down";
import { default as default81 } from "../icons/arrow-up-from-dot";
import { default as default82 } from "../icons/arrow-up-from-line";
import { default as default83 } from "../icons/arrow-up-left";
import { default as default84 } from "../icons/arrow-up-right";
import { default as default85 } from "../icons/arrow-up-to-line";
import { default as default86 } from "../icons/arrow-up-wide-narrow";
import { default as default87 } from "../icons/arrow-up";
import { default as default88 } from "../icons/arrows-up-from-line";
import { default as default89 } from "../icons/asterisk";
import { default as default90 } from "../icons/atom";
import { default as default91 } from "../icons/at-sign";
import { default as default92 } from "../icons/audio-waveform";
import { default as default93 } from "../icons/audio-lines";
import { default as default94 } from "../icons/award";
import { default as default95 } from "../icons/axe";
import { default as default96 } from "../icons/baby";
import { default as default97 } from "../icons/backpack";
import { default as default98 } from "../icons/badge-alert";
import { default as default99 } from "../icons/badge-dollar-sign";
import { default as default100 } from "../icons/badge-cent";
import { default as default101 } from "../icons/badge-euro";
import { default as default102 } from "../icons/badge-indian-rupee";
import { default as default103 } from "../icons/badge-info";
import { default as default104 } from "../icons/badge-help";
import { default as default105 } from "../icons/badge-japanese-yen";
import { default as default106 } from "../icons/badge-minus";
import { default as default107 } from "../icons/badge-plus";
import { default as default108 } from "../icons/badge-percent";
import { default as default109 } from "../icons/badge-pound-sterling";
import { default as default110 } from "../icons/badge-russian-ruble";
import { default as default111 } from "../icons/badge-swiss-franc";
import { default as default112 } from "../icons/badge-x";
import { default as default113 } from "../icons/badge";
import { default as default114 } from "../icons/baggage-claim";
import { default as default115 } from "../icons/ban";
import { default as default116 } from "../icons/banana";
import { default as default117 } from "../icons/bandage";
import { default as default118 } from "../icons/banknote-arrow-down";
import { default as default119 } from "../icons/banknote-arrow-up";
import { default as default120 } from "../icons/banknote-x";
import { default as default121 } from "../icons/banknote";
import { default as default122 } from "../icons/barcode";
import { default as default123 } from "../icons/baseline";
import { default as default124 } from "../icons/bath";
import { default as default125 } from "../icons/battery-charging";
import { default as default126 } from "../icons/battery-full";
import { default as default127 } from "../icons/battery-low";
import { default as default128 } from "../icons/battery-medium";
import { default as default129 } from "../icons/battery-warning";
import { default as default130 } from "../icons/battery";
import { default as default131 } from "../icons/battery-plus";
import { default as default132 } from "../icons/beaker";
import { default as default133 } from "../icons/bean-off";
import { default as default134 } from "../icons/bean";
import { default as default135 } from "../icons/bed-double";
import { default as default136 } from "../icons/bed";
import { default as default137 } from "../icons/bed-single";
import { default as default138 } from "../icons/beef";
import { default as default139 } from "../icons/beer-off";
import { default as default140 } from "../icons/beer";
import { default as default141 } from "../icons/bell-dot";
import { default as default142 } from "../icons/bell-electric";
import { default as default143 } from "../icons/bell-off";
import { default as default144 } from "../icons/bell-minus";
import { default as default145 } from "../icons/bell-plus";
import { default as default146 } from "../icons/bell-ring";
import { default as default147 } from "../icons/bell";
import { default as default148 } from "../icons/between-vertical-end";
import { default as default149 } from "../icons/between-vertical-start";
import { default as default150 } from "../icons/biceps-flexed";
import { default as default151 } from "../icons/bike";
import { default as default152 } from "../icons/binary";
import { default as default153 } from "../icons/binoculars";
import { default as default154 } from "../icons/bird";
import { default as default155 } from "../icons/blend";
import { default as default156 } from "../icons/biohazard";
import { default as default157 } from "../icons/bitcoin";
import { default as default158 } from "../icons/blinds";
import { default as default159 } from "../icons/bluetooth-connected";
import { default as default160 } from "../icons/blocks";
import { default as default161 } from "../icons/bluetooth-off";
import { default as default162 } from "../icons/bluetooth-searching";
import { default as default163 } from "../icons/bluetooth";
import { default as default164 } from "../icons/bomb";
import { default as default165 } from "../icons/bolt";
import { default as default166 } from "../icons/bold";
import { default as default167 } from "../icons/bone";
import { default as default168 } from "../icons/book-a";
import { default as default169 } from "../icons/book-alert";
import { default as default170 } from "../icons/book-audio";
import { default as default171 } from "../icons/book-check";
import { default as default172 } from "../icons/book-copy";
import { default as default173 } from "../icons/book-down";
import { default as default174 } from "../icons/book-headphones";
import { default as default175 } from "../icons/book-heart";
import { default as default176 } from "../icons/book-image";
import { default as default177 } from "../icons/book-key";
import { default as default178 } from "../icons/book-lock";
import { default as default179 } from "../icons/book-marked";
import { default as default180 } from "../icons/book-minus";
import { default as default181 } from "../icons/book-open-check";
import { default as default182 } from "../icons/book-open-text";
import { default as default183 } from "../icons/book-open";
import { default as default184 } from "../icons/book-plus";
import { default as default185 } from "../icons/book-text";
import { default as default186 } from "../icons/book-type";
import { default as default187 } from "../icons/book-up-2";
import { default as default188 } from "../icons/book-up";
import { default as default189 } from "../icons/book-user";
import { default as default190 } from "../icons/book-x";
import { default as default191 } from "../icons/book";
import { default as default192 } from "../icons/bookmark-check";
import { default as default193 } from "../icons/bookmark-minus";
import { default as default194 } from "../icons/bookmark-x";
import { default as default195 } from "../icons/bookmark-plus";
import { default as default196 } from "../icons/bookmark";
import { default as default197 } from "../icons/boom-box";
import { default as default198 } from "../icons/bot-message-square";
import { default as default199 } from "../icons/bot-off";
import { default as default200 } from "../icons/bot";
import { default as default201 } from "../icons/box";
import { default as default202 } from "../icons/bow-arrow";
import { default as default203 } from "../icons/boxes";
import { default as default204 } from "../icons/brain-circuit";
import { default as default205 } from "../icons/brackets";
import { default as default206 } from "../icons/brain-cog";
import { default as default207 } from "../icons/brain";
import { default as default208 } from "../icons/brick-wall-fire";
import { default as default209 } from "../icons/brick-wall";
import { default as default210 } from "../icons/briefcase-business";
import { default as default211 } from "../icons/briefcase-medical";
import { default as default212 } from "../icons/briefcase-conveyor-belt";
import { default as default213 } from "../icons/briefcase";
import { default as default214 } from "../icons/bring-to-front";
import { default as default215 } from "../icons/brush-cleaning";
import { default as default216 } from "../icons/brush";
import { default as default217 } from "../icons/bug-off";
import { default as default218 } from "../icons/bubbles";
import { default as default219 } from "../icons/bug";
import { default as default220 } from "../icons/bug-play";
import { default as default221 } from "../icons/building-2";
import { default as default222 } from "../icons/building";
import { default as default223 } from "../icons/bus";
import { default as default224 } from "../icons/bus-front";
import { default as default225 } from "../icons/cable-car";
import { default as default226 } from "../icons/cable";
import { default as default227 } from "../icons/cake-slice";
import { default as default228 } from "../icons/cake";
import { default as default229 } from "../icons/calculator";
import { default as default230 } from "../icons/calendar-arrow-down";
import { default as default231 } from "../icons/calendar-1";
import { default as default232 } from "../icons/calendar-check-2";
import { default as default233 } from "../icons/calendar-arrow-up";
import { default as default234 } from "../icons/calendar-check";
import { default as default235 } from "../icons/calendar-clock";
import { default as default236 } from "../icons/calendar-cog";
import { default as default237 } from "../icons/calendar-days";
import { default as default238 } from "../icons/calendar-fold";
import { default as default239 } from "../icons/calendar-heart";
import { default as default240 } from "../icons/calendar-minus-2";
import { default as default241 } from "../icons/calendar-minus";
import { default as default242 } from "../icons/calendar-off";
import { default as default243 } from "../icons/calendar-plus-2";
import { default as default244 } from "../icons/calendar-plus";
import { default as default245 } from "../icons/calendar-range";
import { default as default246 } from "../icons/calendar-search";
import { default as default247 } from "../icons/calendar-sync";
import { default as default248 } from "../icons/calendar-x-2";
import { default as default249 } from "../icons/calendar-x";
import { default as default250 } from "../icons/calendar";
import { default as default251 } from "../icons/camera-off";
import { default as default252 } from "../icons/camera";
import { default as default253 } from "../icons/candy-cane";
import { default as default254 } from "../icons/candy-off";
import { default as default255 } from "../icons/candy";
import { default as default256 } from "../icons/cannabis";
import { default as default257 } from "../icons/captions-off";
import { default as default258 } from "../icons/car-front";
import { default as default259 } from "../icons/car";
import { default as default260 } from "../icons/car-taxi-front";
import { default as default261 } from "../icons/caravan";
import { default as default262 } from "../icons/card-sim";
import { default as default263 } from "../icons/carrot";
import { default as default264 } from "../icons/case-lower";
import { default as default265 } from "../icons/case-sensitive";
import { default as default266 } from "../icons/case-upper";
import { default as default267 } from "../icons/cassette-tape";
import { default as default268 } from "../icons/cast";
import { default as default269 } from "../icons/cat";
import { default as default270 } from "../icons/castle";
import { default as default271 } from "../icons/cctv";
import { default as default272 } from "../icons/chart-bar-decreasing";
import { default as default273 } from "../icons/chart-bar-increasing";
import { default as default274 } from "../icons/chart-bar-stacked";
import { default as default275 } from "../icons/chart-column-decreasing";
import { default as default276 } from "../icons/chart-column-stacked";
import { default as default277 } from "../icons/chart-gantt";
import { default as default278 } from "../icons/chart-network";
import { default as default279 } from "../icons/chart-no-axes-column-decreasing";
import { default as default280 } from "../icons/chart-no-axes-combined";
import { default as default281 } from "../icons/chart-spline";
import { default as default282 } from "../icons/check-check";
import { default as default283 } from "../icons/check-line";
import { default as default284 } from "../icons/check";
import { default as default285 } from "../icons/chef-hat";
import { default as default286 } from "../icons/chevron-down";
import { default as default287 } from "../icons/cherry";
import { default as default288 } from "../icons/chevron-first";
import { default as default289 } from "../icons/chevron-last";
import { default as default290 } from "../icons/chevron-right";
import { default as default291 } from "../icons/chevron-left";
import { default as default292 } from "../icons/chevron-up";
import { default as default293 } from "../icons/chevrons-down-up";
import { default as default294 } from "../icons/chevrons-down";
import { default as default295 } from "../icons/chevrons-left-right-ellipsis";
import { default as default296 } from "../icons/chevrons-left-right";
import { default as default297 } from "../icons/chevrons-left";
import { default as default298 } from "../icons/chevrons-right-left";
import { default as default299 } from "../icons/chevrons-right";
import { default as default300 } from "../icons/chevrons-up-down";
import { default as default301 } from "../icons/chrome";
import { default as default302 } from "../icons/chevrons-up";
import { default as default303 } from "../icons/church";
import { default as default304 } from "../icons/cigarette-off";
import { default as default305 } from "../icons/cigarette";
import { default as default306 } from "../icons/circle-dashed";
import { default as default307 } from "../icons/circle-dot-dashed";
import { default as default308 } from "../icons/circle-dollar-sign";
import { default as default309 } from "../icons/circle-dot";
import { default as default310 } from "../icons/circle-ellipsis";
import { default as default311 } from "../icons/circle-equal";
import { default as default312 } from "../icons/circle-fading-arrow-up";
import { default as default313 } from "../icons/circle-fading-plus";
import { default as default314 } from "../icons/circle-off";
import { default as default315 } from "../icons/circle-pound-sterling";
import { default as default316 } from "../icons/circle-slash";
import { default as default317 } from "../icons/circle-small";
import { default as default318 } from "../icons/circle";
import { default as default319 } from "../icons/circuit-board";
import { default as default320 } from "../icons/citrus";
import { default as default321 } from "../icons/clapperboard";
import { default as default322 } from "../icons/clipboard-check";
import { default as default323 } from "../icons/clipboard-copy";
import { default as default324 } from "../icons/clipboard-list";
import { default as default325 } from "../icons/clipboard-minus";
import { default as default326 } from "../icons/clipboard-paste";
import { default as default327 } from "../icons/clipboard-plus";
import { default as default328 } from "../icons/clipboard-x";
import { default as default329 } from "../icons/clipboard-type";
import { default as default330 } from "../icons/clipboard";
import { default as default331 } from "../icons/clock-10";
import { default as default332 } from "../icons/clock-11";
import { default as default333 } from "../icons/clock-1";
import { default as default334 } from "../icons/clock-12";
import { default as default335 } from "../icons/clock-2";
import { default as default336 } from "../icons/clock-3";
import { default as default337 } from "../icons/clock-4";
import { default as default338 } from "../icons/clock-5";
import { default as default339 } from "../icons/clock-6";
import { default as default340 } from "../icons/clock-7";
import { default as default341 } from "../icons/clock-8";
import { default as default342 } from "../icons/clock-9";
import { default as default343 } from "../icons/clock-alert";
import { default as default344 } from "../icons/clock-arrow-down";
import { default as default345 } from "../icons/clock-arrow-up";
import { default as default346 } from "../icons/clock-plus";
import { default as default347 } from "../icons/clock-fading";
import { default as default348 } from "../icons/clock";
import { default as default349 } from "../icons/cloud-check";
import { default as default350 } from "../icons/cloud-cog";
import { default as default351 } from "../icons/cloud-alert";
import { default as default352 } from "../icons/cloud-drizzle";
import { default as default353 } from "../icons/cloud-fog";
import { default as default354 } from "../icons/cloud-hail";
import { default as default355 } from "../icons/cloud-lightning";
import { default as default356 } from "../icons/cloud-moon-rain";
import { default as default357 } from "../icons/cloud-moon";
import { default as default358 } from "../icons/cloud-off";
import { default as default359 } from "../icons/cloud-rain-wind";
import { default as default360 } from "../icons/cloud-snow";
import { default as default361 } from "../icons/cloud-sun-rain";
import { default as default362 } from "../icons/cloud-sun";
import { default as default363 } from "../icons/cloud-rain";
import { default as default364 } from "../icons/cloud";
import { default as default365 } from "../icons/cloudy";
import { default as default366 } from "../icons/club";
import { default as default367 } from "../icons/code";
import { default as default368 } from "../icons/clover";
import { default as default369 } from "../icons/codepen";
import { default as default370 } from "../icons/codesandbox";
import { default as default371 } from "../icons/coffee";
import { default as default372 } from "../icons/cog";
import { default as default373 } from "../icons/coins";
import { default as default374 } from "../icons/columns-4";
import { default as default375 } from "../icons/command";
import { default as default376 } from "../icons/combine";
import { default as default377 } from "../icons/compass";
import { default as default378 } from "../icons/computer";
import { default as default379 } from "../icons/component";
import { default as default380 } from "../icons/concierge-bell";
import { default as default381 } from "../icons/construction";
import { default as default382 } from "../icons/cone";
import { default as default383 } from "../icons/contact";
import { default as default384 } from "../icons/contrast";
import { default as default385 } from "../icons/container";
import { default as default386 } from "../icons/cookie";
import { default as default387 } from "../icons/cooking-pot";
import { default as default388 } from "../icons/copy-check";
import { default as default389 } from "../icons/copy-minus";
import { default as default390 } from "../icons/copy-plus";
import { default as default391 } from "../icons/copy-slash";
import { default as default392 } from "../icons/copy-x";
import { default as default393 } from "../icons/copy";
import { default as default394 } from "../icons/copyleft";
import { default as default395 } from "../icons/copyright";
import { default as default396 } from "../icons/corner-down-left";
import { default as default397 } from "../icons/corner-down-right";
import { default as default398 } from "../icons/corner-left-up";
import { default as default399 } from "../icons/corner-left-down";
import { default as default400 } from "../icons/corner-right-down";
import { default as default401 } from "../icons/corner-right-up";
import { default as default402 } from "../icons/corner-up-right";
import { default as default403 } from "../icons/cpu";
import { default as default404 } from "../icons/corner-up-left";
import { default as default405 } from "../icons/creative-commons";
import { default as default406 } from "../icons/croissant";
import { default as default407 } from "../icons/credit-card";
import { default as default408 } from "../icons/crop";
import { default as default409 } from "../icons/cross";
import { default as default410 } from "../icons/crosshair";
import { default as default411 } from "../icons/crown";
import { default as default412 } from "../icons/cuboid";
import { default as default413 } from "../icons/cup-soda";
import { default as default414 } from "../icons/currency";
import { default as default415 } from "../icons/cylinder";
import { default as default416 } from "../icons/dam";
import { default as default417 } from "../icons/database-backup";
import { default as default418 } from "../icons/database-zap";
import { default as default419 } from "../icons/database";
import { default as default420 } from "../icons/decimals-arrow-left";
import { default as default421 } from "../icons/delete";
import { default as default422 } from "../icons/decimals-arrow-right";
import { default as default423 } from "../icons/diameter";
import { default as default424 } from "../icons/dessert";
import { default as default425 } from "../icons/diamond-minus";
import { default as default426 } from "../icons/diamond-plus";
import { default as default427 } from "../icons/diamond";
import { default as default428 } from "../icons/dice-1";
import { default as default429 } from "../icons/dice-2";
import { default as default430 } from "../icons/dice-3";
import { default as default431 } from "../icons/dice-4";
import { default as default432 } from "../icons/dice-5";
import { default as default433 } from "../icons/dice-6";
import { default as default434 } from "../icons/dices";
import { default as default435 } from "../icons/disc-2";
import { default as default436 } from "../icons/diff";
import { default as default437 } from "../icons/disc-album";
import { default as default438 } from "../icons/disc-3";
import { default as default439 } from "../icons/disc";
import { default as default440 } from "../icons/divide";
import { default as default441 } from "../icons/dna";
import { default as default442 } from "../icons/dna-off";
import { default as default443 } from "../icons/dock";
import { default as default444 } from "../icons/dog";
import { default as default445 } from "../icons/dollar-sign";
import { default as default446 } from "../icons/door-closed-locked";
import { default as default447 } from "../icons/donut";
import { default as default448 } from "../icons/door-closed";
import { default as default449 } from "../icons/door-open";
import { default as default450 } from "../icons/dot";
import { default as default451 } from "../icons/download";
import { default as default452 } from "../icons/drafting-compass";
import { default as default453 } from "../icons/drama";
import { default as default454 } from "../icons/dribbble";
import { default as default455 } from "../icons/drill";
import { default as default456 } from "../icons/droplet-off";
import { default as default457 } from "../icons/droplet";
import { default as default458 } from "../icons/droplets";
import { default as default459 } from "../icons/drum";
import { default as default460 } from "../icons/drumstick";
import { default as default461 } from "../icons/dumbbell";
import { default as default462 } from "../icons/ear-off";
import { default as default463 } from "../icons/ear";
import { default as default464 } from "../icons/earth-lock";
import { default as default465 } from "../icons/eclipse";
import { default as default466 } from "../icons/egg-fried";
import { default as default467 } from "../icons/egg-off";
import { default as default468 } from "../icons/egg";
import { default as default469 } from "../icons/equal-approximately";
import { default as default470 } from "../icons/equal";
import { default as default471 } from "../icons/equal-not";
import { default as default472 } from "../icons/eraser";
import { default as default473 } from "../icons/ethernet-port";
import { default as default474 } from "../icons/euro";
import { default as default475 } from "../icons/expand";
import { default as default476 } from "../icons/external-link";
import { default as default477 } from "../icons/eye-closed";
import { default as default478 } from "../icons/eye-off";
import { default as default479 } from "../icons/eye";
import { default as default480 } from "../icons/facebook";
import { default as default481 } from "../icons/factory";
import { default as default482 } from "../icons/fan";
import { default as default483 } from "../icons/fast-forward";
import { default as default484 } from "../icons/feather";
import { default as default485 } from "../icons/fence";
import { default as default486 } from "../icons/ferris-wheel";
import { default as default487 } from "../icons/file-archive";
import { default as default488 } from "../icons/figma";
import { default as default489 } from "../icons/file-audio-2";
import { default as default490 } from "../icons/file-audio";
import { default as default491 } from "../icons/file-badge";
import { default as default492 } from "../icons/file-badge-2";
import { default as default493 } from "../icons/file-box";
import { default as default494 } from "../icons/file-check-2";
import { default as default495 } from "../icons/file-check";
import { default as default496 } from "../icons/file-clock";
import { default as default497 } from "../icons/file-code-2";
import { default as default498 } from "../icons/file-code";
import { default as default499 } from "../icons/file-diff";
import { default as default500 } from "../icons/file-digit";
import { default as default501 } from "../icons/file-down";
import { default as default502 } from "../icons/file-heart";
import { default as default503 } from "../icons/file-image";
import { default as default504 } from "../icons/file-input";
import { default as default505 } from "../icons/file-json-2";
import { default as default506 } from "../icons/file-json";
import { default as default507 } from "../icons/file-key-2";
import { default as default508 } from "../icons/file-key";
import { default as default509 } from "../icons/file-lock-2";
import { default as default510 } from "../icons/file-lock";
import { default as default511 } from "../icons/file-minus-2";
import { default as default512 } from "../icons/file-minus";
import { default as default513 } from "../icons/file-music";
import { default as default514 } from "../icons/file-output";
import { default as default515 } from "../icons/file-plus-2";
import { default as default516 } from "../icons/file-plus";
import { default as default517 } from "../icons/file-question";
import { default as default518 } from "../icons/file-scan";
import { default as default519 } from "../icons/file-search-2";
import { default as default520 } from "../icons/file-search";
import { default as default521 } from "../icons/file-spreadsheet";
import { default as default522 } from "../icons/file-sliders";
import { default as default523 } from "../icons/file-stack";
import { default as default524 } from "../icons/file-symlink";
import { default as default525 } from "../icons/file-text";
import { default as default526 } from "../icons/file-terminal";
import { default as default527 } from "../icons/file-type-2";
import { default as default528 } from "../icons/file-type";
import { default as default529 } from "../icons/file-up";
import { default as default530 } from "../icons/file-user";
import { default as default531 } from "../icons/file-video-2";
import { default as default532 } from "../icons/file-video";
import { default as default533 } from "../icons/file-volume-2";
import { default as default534 } from "../icons/file-volume";
import { default as default535 } from "../icons/file-warning";
import { default as default536 } from "../icons/file-x-2";
import { default as default537 } from "../icons/file-x";
import { default as default538 } from "../icons/file";
import { default as default539 } from "../icons/files";
import { default as default540 } from "../icons/film";
import { default as default541 } from "../icons/fingerprint";
import { default as default542 } from "../icons/fire-extinguisher";
import { default as default543 } from "../icons/fish-off";
import { default as default544 } from "../icons/fish-symbol";
import { default as default545 } from "../icons/fish";
import { default as default546 } from "../icons/flag-off";
import { default as default547 } from "../icons/flag-triangle-left";
import { default as default548 } from "../icons/flag-triangle-right";
import { default as default549 } from "../icons/flag";
import { default as default550 } from "../icons/flame-kindling";
import { default as default551 } from "../icons/flame";
import { default as default552 } from "../icons/flashlight-off";
import { default as default553 } from "../icons/flashlight";
import { default as default554 } from "../icons/flask-conical-off";
import { default as default555 } from "../icons/flask-conical";
import { default as default556 } from "../icons/flask-round";
import { default as default557 } from "../icons/flip-horizontal";
import { default as default558 } from "../icons/flip-horizontal-2";
import { default as default559 } from "../icons/flip-vertical-2";
import { default as default560 } from "../icons/flip-vertical";
import { default as default561 } from "../icons/flower-2";
import { default as default562 } from "../icons/flower";
import { default as default563 } from "../icons/fold-horizontal";
import { default as default564 } from "../icons/fold-vertical";
import { default as default565 } from "../icons/focus";
import { default as default566 } from "../icons/folder-archive";
import { default as default567 } from "../icons/folder-check";
import { default as default568 } from "../icons/folder-clock";
import { default as default569 } from "../icons/folder-closed";
import { default as default570 } from "../icons/folder-code";
import { default as default571 } from "../icons/folder-dot";
import { default as default572 } from "../icons/folder-down";
import { default as default573 } from "../icons/folder-git-2";
import { default as default574 } from "../icons/folder-git";
import { default as default575 } from "../icons/folder-heart";
import { default as default576 } from "../icons/folder-input";
import { default as default577 } from "../icons/folder-kanban";
import { default as default578 } from "../icons/folder-key";
import { default as default579 } from "../icons/folder-lock";
import { default as default580 } from "../icons/folder-minus";
import { default as default581 } from "../icons/folder-open-dot";
import { default as default582 } from "../icons/folder-open";
import { default as default583 } from "../icons/folder-output";
import { default as default584 } from "../icons/folder-plus";
import { default as default585 } from "../icons/folder-root";
import { default as default586 } from "../icons/folder-search-2";
import { default as default587 } from "../icons/folder-search";
import { default as default588 } from "../icons/folder-symlink";
import { default as default589 } from "../icons/folder-sync";
import { default as default590 } from "../icons/folder-tree";
import { default as default591 } from "../icons/folder-x";
import { default as default592 } from "../icons/folder-up";
import { default as default593 } from "../icons/folder";
import { default as default594 } from "../icons/folders";
import { default as default595 } from "../icons/forklift";
import { default as default596 } from "../icons/footprints";
import { default as default597 } from "../icons/forward";
import { default as default598 } from "../icons/framer";
import { default as default599 } from "../icons/frame";
import { default as default600 } from "../icons/fuel";
import { default as default601 } from "../icons/frown";
import { default as default602 } from "../icons/fullscreen";
import { default as default603 } from "../icons/funnel-plus";
import { default as default604 } from "../icons/gallery-horizontal-end";
import { default as default605 } from "../icons/gallery-horizontal";
import { default as default606 } from "../icons/gallery-vertical-end";
import { default as default607 } from "../icons/gallery-thumbnails";
import { default as default608 } from "../icons/gallery-vertical";
import { default as default609 } from "../icons/gamepad-2";
import { default as default610 } from "../icons/gamepad";
import { default as default611 } from "../icons/gauge";
import { default as default612 } from "../icons/gavel";
import { default as default613 } from "../icons/gem";
import { default as default614 } from "../icons/ghost";
import { default as default615 } from "../icons/gift";
import { default as default616 } from "../icons/git-branch";
import { default as default617 } from "../icons/git-branch-plus";
import { default as default618 } from "../icons/git-commit-vertical";
import { default as default619 } from "../icons/git-compare-arrows";
import { default as default620 } from "../icons/git-compare";
import { default as default621 } from "../icons/git-fork";
import { default as default622 } from "../icons/git-graph";
import { default as default623 } from "../icons/git-merge";
import { default as default624 } from "../icons/git-pull-request-arrow";
import { default as default625 } from "../icons/git-pull-request-closed";
import { default as default626 } from "../icons/git-pull-request-create";
import { default as default627 } from "../icons/git-pull-request-create-arrow";
import { default as default628 } from "../icons/git-pull-request-draft";
import { default as default629 } from "../icons/git-pull-request";
import { default as default630 } from "../icons/gitlab";
import { default as default631 } from "../icons/github";
import { default as default632 } from "../icons/glass-water";
import { default as default633 } from "../icons/glasses";
import { default as default634 } from "../icons/globe-lock";
import { default as default635 } from "../icons/globe";
import { default as default636 } from "../icons/goal";
import { default as default637 } from "../icons/gpu";
import { default as default638 } from "../icons/grab";
import { default as default639 } from "../icons/graduation-cap";
import { default as default640 } from "../icons/grape";
import { default as default641 } from "../icons/grid-3x2";
import { default as default642 } from "../icons/grip-horizontal";
import { default as default643 } from "../icons/grip-vertical";
import { default as default644 } from "../icons/grip";
import { default as default645 } from "../icons/group";
import { default as default646 } from "../icons/ham";
import { default as default647 } from "../icons/guitar";
import { default as default648 } from "../icons/hamburger";
import { default as default649 } from "../icons/hammer";
import { default as default650 } from "../icons/hand-coins";
import { default as default651 } from "../icons/hand-metal";
import { default as default652 } from "../icons/hand-heart";
import { default as default653 } from "../icons/hand-platter";
import { default as default654 } from "../icons/hand";
import { default as default655 } from "../icons/hard-drive-download";
import { default as default656 } from "../icons/handshake";
import { default as default657 } from "../icons/hard-drive-upload";
import { default as default658 } from "../icons/hard-drive";
import { default as default659 } from "../icons/hard-hat";
import { default as default660 } from "../icons/hash";
import { default as default661 } from "../icons/hdmi-port";
import { default as default662 } from "../icons/haze";
import { default as default663 } from "../icons/heading-1";
import { default as default664 } from "../icons/heading-2";
import { default as default665 } from "../icons/heading-3";
import { default as default666 } from "../icons/heading-5";
import { default as default667 } from "../icons/heading-4";
import { default as default668 } from "../icons/heading-6";
import { default as default669 } from "../icons/heading";
import { default as default670 } from "../icons/headphones";
import { default as default671 } from "../icons/headphone-off";
import { default as default672 } from "../icons/headset";
import { default as default673 } from "../icons/heart-crack";
import { default as default674 } from "../icons/heart-handshake";
import { default as default675 } from "../icons/heart-minus";
import { default as default676 } from "../icons/heart-off";
import { default as default677 } from "../icons/heart-pulse";
import { default as default678 } from "../icons/heart";
import { default as default679 } from "../icons/heart-plus";
import { default as default680 } from "../icons/heater";
import { default as default681 } from "../icons/hexagon";
import { default as default682 } from "../icons/highlighter";
import { default as default683 } from "../icons/history";
import { default as default684 } from "../icons/hop-off";
import { default as default685 } from "../icons/hop";
import { default as default686 } from "../icons/hospital";
import { default as default687 } from "../icons/hotel";
import { default as default688 } from "../icons/hourglass";
import { default as default689 } from "../icons/house-plug";
import { default as default690 } from "../icons/house-plus";
import { default as default691 } from "../icons/house-wifi";
import { default as default692 } from "../icons/id-card";
import { default as default693 } from "../icons/id-card-lanyard";
import { default as default694 } from "../icons/image-down";
import { default as default695 } from "../icons/image-off";
import { default as default696 } from "../icons/image-play";
import { default as default697 } from "../icons/image-minus";
import { default as default698 } from "../icons/image-plus";
import { default as default699 } from "../icons/image-up";
import { default as default700 } from "../icons/image";
import { default as default701 } from "../icons/image-upscale";
import { default as default702 } from "../icons/images";
import { default as default703 } from "../icons/inbox";
import { default as default704 } from "../icons/import";
import { default as default705 } from "../icons/indian-rupee";
import { default as default706 } from "../icons/infinity";
import { default as default707 } from "../icons/info";
import { default as default708 } from "../icons/instagram";
import { default as default709 } from "../icons/inspection-panel";
import { default as default710 } from "../icons/italic";
import { default as default711 } from "../icons/iteration-ccw";
import { default as default712 } from "../icons/iteration-cw";
import { default as default713 } from "../icons/japanese-yen";
import { default as default714 } from "../icons/joystick";
import { default as default715 } from "../icons/kanban";
import { default as default716 } from "../icons/key-round";
import { default as default717 } from "../icons/key-square";
import { default as default718 } from "../icons/key";
import { default as default719 } from "../icons/keyboard-music";
import { default as default720 } from "../icons/keyboard-off";
import { default as default721 } from "../icons/keyboard";
import { default as default722 } from "../icons/lamp-ceiling";
import { default as default723 } from "../icons/lamp-desk";
import { default as default724 } from "../icons/lamp-floor";
import { default as default725 } from "../icons/lamp-wall-down";
import { default as default726 } from "../icons/lamp-wall-up";
import { default as default727 } from "../icons/lamp";
import { default as default728 } from "../icons/land-plot";
import { default as default729 } from "../icons/landmark";
import { default as default730 } from "../icons/languages";
import { default as default731 } from "../icons/laptop-minimal-check";
import { default as default732 } from "../icons/laptop";
import { default as default733 } from "../icons/lasso-select";
import { default as default734 } from "../icons/lasso";
import { default as default735 } from "../icons/laugh";
import { default as default736 } from "../icons/layers-2";
import { default as default737 } from "../icons/layout-grid";
import { default as default738 } from "../icons/layout-dashboard";
import { default as default739 } from "../icons/layout-list";
import { default as default740 } from "../icons/layout-panel-top";
import { default as default741 } from "../icons/layout-panel-left";
import { default as default742 } from "../icons/leaf";
import { default as default743 } from "../icons/layout-template";
import { default as default744 } from "../icons/lectern";
import { default as default745 } from "../icons/leafy-green";
import { default as default746 } from "../icons/library-big";
import { default as default747 } from "../icons/letter-text";
import { default as default748 } from "../icons/library";
import { default as default749 } from "../icons/life-buoy";
import { default as default750 } from "../icons/lightbulb-off";
import { default as default751 } from "../icons/ligature";
import { default as default752 } from "../icons/lightbulb";
import { default as default753 } from "../icons/link-2-off";
import { default as default754 } from "../icons/link-2";
import { default as default755 } from "../icons/link";
import { default as default756 } from "../icons/linkedin";
import { default as default757 } from "../icons/list-check";
import { default as default758 } from "../icons/list-checks";
import { default as default759 } from "../icons/list-end";
import { default as default760 } from "../icons/list-collapse";
import { default as default761 } from "../icons/list-filter-plus";
import { default as default762 } from "../icons/list-filter";
import { default as default763 } from "../icons/list-minus";
import { default as default764 } from "../icons/list-music";
import { default as default765 } from "../icons/list-ordered";
import { default as default766 } from "../icons/list-plus";
import { default as default767 } from "../icons/list-restart";
import { default as default768 } from "../icons/list-start";
import { default as default769 } from "../icons/list-todo";
import { default as default770 } from "../icons/list-video";
import { default as default771 } from "../icons/list-tree";
import { default as default772 } from "../icons/list-x";
import { default as default773 } from "../icons/list";
import { default as default774 } from "../icons/loader-pinwheel";
import { default as default775 } from "../icons/loader";
import { default as default776 } from "../icons/locate-fixed";
import { default as default777 } from "../icons/locate-off";
import { default as default778 } from "../icons/locate";
import { default as default779 } from "../icons/location-edit";
import { default as default780 } from "../icons/lock-keyhole";
import { default as default781 } from "../icons/lock";
import { default as default782 } from "../icons/log-in";
import { default as default783 } from "../icons/log-out";
import { default as default784 } from "../icons/logs";
import { default as default785 } from "../icons/luggage";
import { default as default786 } from "../icons/lollipop";
import { default as default787 } from "../icons/magnet";
import { default as default788 } from "../icons/mail-minus";
import { default as default789 } from "../icons/mail-check";
import { default as default790 } from "../icons/mail-plus";
import { default as default791 } from "../icons/mail-open";
import { default as default792 } from "../icons/mail-question";
import { default as default793 } from "../icons/mail-search";
import { default as default794 } from "../icons/mail-warning";
import { default as default795 } from "../icons/mail";
import { default as default796 } from "../icons/mail-x";
import { default as default797 } from "../icons/mailbox";
import { default as default798 } from "../icons/mails";
import { default as default799 } from "../icons/map-pin-check-inside";
import { default as default800 } from "../icons/map-pin-check";
import { default as default801 } from "../icons/map-pin-house";
import { default as default802 } from "../icons/map-pin-minus";
import { default as default803 } from "../icons/map-pin-off";
import { default as default804 } from "../icons/map-pin-minus-inside";
import { default as default805 } from "../icons/map-pin-plus-inside";
import { default as default806 } from "../icons/map-pin-plus";
import { default as default807 } from "../icons/map-pin";
import { default as default808 } from "../icons/map-pin-x";
import { default as default809 } from "../icons/map-pin-x-inside";
import { default as default810 } from "../icons/map-pinned";
import { default as default811 } from "../icons/map-plus";
import { default as default812 } from "../icons/map";
import { default as default813 } from "../icons/mars";
import { default as default814 } from "../icons/martini";
import { default as default815 } from "../icons/mars-stroke";
import { default as default816 } from "../icons/maximize-2";
import { default as default817 } from "../icons/maximize";
import { default as default818 } from "../icons/medal";
import { default as default819 } from "../icons/megaphone-off";
import { default as default820 } from "../icons/megaphone";
import { default as default821 } from "../icons/meh";
import { default as default822 } from "../icons/menu";
import { default as default823 } from "../icons/memory-stick";
import { default as default824 } from "../icons/merge";
import { default as default825 } from "../icons/message-circle-heart";
import { default as default826 } from "../icons/message-circle-code";
import { default as default827 } from "../icons/message-circle-more";
import { default as default828 } from "../icons/message-circle-dashed";
import { default as default829 } from "../icons/message-circle-off";
import { default as default830 } from "../icons/message-circle-plus";
import { default as default831 } from "../icons/message-circle-question";
import { default as default832 } from "../icons/message-circle-reply";
import { default as default833 } from "../icons/message-circle-x";
import { default as default834 } from "../icons/message-circle-warning";
import { default as default835 } from "../icons/message-square-code";
import { default as default836 } from "../icons/message-circle";
import { default as default837 } from "../icons/message-square-dashed";
import { default as default838 } from "../icons/message-square-diff";
import { default as default839 } from "../icons/message-square-dot";
import { default as default840 } from "../icons/message-square-lock";
import { default as default841 } from "../icons/message-square-heart";
import { default as default842 } from "../icons/message-square-off";
import { default as default843 } from "../icons/message-square-more";
import { default as default844 } from "../icons/message-square-quote";
import { default as default845 } from "../icons/message-square-plus";
import { default as default846 } from "../icons/message-square-reply";
import { default as default847 } from "../icons/message-square-share";
import { default as default848 } from "../icons/message-square-text";
import { default as default849 } from "../icons/message-square-warning";
import { default as default850 } from "../icons/message-square-x";
import { default as default851 } from "../icons/message-square";
import { default as default852 } from "../icons/messages-square";
import { default as default853 } from "../icons/mic-off";
import { default as default854 } from "../icons/mic";
import { default as default855 } from "../icons/microchip";
import { default as default856 } from "../icons/microscope";
import { default as default857 } from "../icons/microwave";
import { default as default858 } from "../icons/milestone";
import { default as default859 } from "../icons/milk";
import { default as default860 } from "../icons/milk-off";
import { default as default861 } from "../icons/minimize-2";
import { default as default862 } from "../icons/minimize";
import { default as default863 } from "../icons/monitor-check";
import { default as default864 } from "../icons/monitor-dot";
import { default as default865 } from "../icons/minus";
import { default as default866 } from "../icons/monitor-cog";
import { default as default867 } from "../icons/monitor-down";
import { default as default868 } from "../icons/monitor-off";
import { default as default869 } from "../icons/monitor-pause";
import { default as default870 } from "../icons/monitor-play";
import { default as default871 } from "../icons/monitor-smartphone";
import { default as default872 } from "../icons/monitor-speaker";
import { default as default873 } from "../icons/monitor-stop";
import { default as default874 } from "../icons/monitor-up";
import { default as default875 } from "../icons/monitor-x";
import { default as default876 } from "../icons/monitor";
import { default as default877 } from "../icons/moon-star";
import { default as default878 } from "../icons/moon";
import { default as default879 } from "../icons/mountain-snow";
import { default as default880 } from "../icons/mountain";
import { default as default881 } from "../icons/mouse-off";
import { default as default882 } from "../icons/mouse-pointer-2";
import { default as default883 } from "../icons/mouse-pointer-ban";
import { default as default884 } from "../icons/mouse-pointer-click";
import { default as default885 } from "../icons/mouse-pointer";
import { default as default886 } from "../icons/mouse";
import { default as default887 } from "../icons/move-diagonal-2";
import { default as default888 } from "../icons/move-down-left";
import { default as default889 } from "../icons/move-diagonal";
import { default as default890 } from "../icons/move-down";
import { default as default891 } from "../icons/move-down-right";
import { default as default892 } from "../icons/move-left";
import { default as default893 } from "../icons/move-horizontal";
import { default as default894 } from "../icons/move-up-left";
import { default as default895 } from "../icons/move-right";
import { default as default896 } from "../icons/move-up";
import { default as default897 } from "../icons/move-up-right";
import { default as default898 } from "../icons/move";
import { default as default899 } from "../icons/music-2";
import { default as default900 } from "../icons/music-3";
import { default as default901 } from "../icons/move-vertical";
import { default as default902 } from "../icons/music-4";
import { default as default903 } from "../icons/navigation-2-off";
import { default as default904 } from "../icons/music";
import { default as default905 } from "../icons/navigation-2";
import { default as default906 } from "../icons/navigation";
import { default as default907 } from "../icons/navigation-off";
import { default as default908 } from "../icons/newspaper";
import { default as default909 } from "../icons/network";
import { default as default910 } from "../icons/nfc";
import { default as default911 } from "../icons/non-binary";
import { default as default912 } from "../icons/notebook-pen";
import { default as default913 } from "../icons/notebook-tabs";
import { default as default914 } from "../icons/notebook-text";
import { default as default915 } from "../icons/notebook";
import { default as default916 } from "../icons/notepad-text-dashed";
import { default as default917 } from "../icons/notepad-text";
import { default as default918 } from "../icons/nut-off";
import { default as default919 } from "../icons/octagon-minus";
import { default as default920 } from "../icons/nut";
import { default as default921 } from "../icons/octagon";
import { default as default922 } from "../icons/option";
import { default as default923 } from "../icons/omega";
import { default as default924 } from "../icons/origami";
import { default as default925 } from "../icons/orbit";
import { default as default926 } from "../icons/package-2";
import { default as default927 } from "../icons/package-check";
import { default as default928 } from "../icons/package-minus";
import { default as default929 } from "../icons/package-open";
import { default as default930 } from "../icons/package-plus";
import { default as default931 } from "../icons/package-x";
import { default as default932 } from "../icons/package";
import { default as default933 } from "../icons/package-search";
import { default as default934 } from "../icons/paint-bucket";
import { default as default935 } from "../icons/paintbrush";
import { default as default936 } from "../icons/paint-roller";
import { default as default937 } from "../icons/palette";
import { default as default938 } from "../icons/panda";
import { default as default939 } from "../icons/panel-bottom-close";
import { default as default940 } from "../icons/panel-bottom-open";
import { default as default941 } from "../icons/panel-bottom";
import { default as default942 } from "../icons/panel-right-close";
import { default as default943 } from "../icons/panel-right-open";
import { default as default944 } from "../icons/panel-right";
import { default as default945 } from "../icons/panel-top";
import { default as default946 } from "../icons/panel-top-close";
import { default as default947 } from "../icons/panel-top-open";
import { default as default948 } from "../icons/panels-left-bottom";
import { default as default949 } from "../icons/panels-right-bottom";
import { default as default950 } from "../icons/paperclip";
import { default as default951 } from "../icons/parentheses";
import { default as default952 } from "../icons/parking-meter";
import { default as default953 } from "../icons/party-popper";
import { default as default954 } from "../icons/pause";
import { default as default955 } from "../icons/pc-case";
import { default as default956 } from "../icons/pen-off";
import { default as default957 } from "../icons/paw-print";
import { default as default958 } from "../icons/pen-tool";
import { default as default959 } from "../icons/pencil-line";
import { default as default960 } from "../icons/pencil-off";
import { default as default961 } from "../icons/pencil-ruler";
import { default as default962 } from "../icons/pencil";
import { default as default963 } from "../icons/pentagon";
import { default as default964 } from "../icons/person-standing";
import { default as default965 } from "../icons/percent";
import { default as default966 } from "../icons/phone-call";
import { default as default967 } from "../icons/philippine-peso";
import { default as default968 } from "../icons/phone-forwarded";
import { default as default969 } from "../icons/phone-incoming";
import { default as default970 } from "../icons/phone-off";
import { default as default971 } from "../icons/phone";
import { default as default972 } from "../icons/phone-missed";
import { default as default973 } from "../icons/pi";
import { default as default974 } from "../icons/piano";
import { default as default975 } from "../icons/phone-outgoing";
import { default as default976 } from "../icons/pickaxe";
import { default as default977 } from "../icons/picture-in-picture-2";
import { default as default978 } from "../icons/piggy-bank";
import { default as default979 } from "../icons/pilcrow-left";
import { default as default980 } from "../icons/picture-in-picture";
import { default as default981 } from "../icons/pilcrow";
import { default as default982 } from "../icons/pilcrow-right";
import { default as default983 } from "../icons/pill-bottle";
import { default as default984 } from "../icons/pill";
import { default as default985 } from "../icons/pin-off";
import { default as default986 } from "../icons/pin";
import { default as default987 } from "../icons/pipette";
import { default as default988 } from "../icons/pizza";
import { default as default989 } from "../icons/plane-landing";
import { default as default990 } from "../icons/plane";
import { default as default991 } from "../icons/plane-takeoff";
import { default as default992 } from "../icons/plug";
import { default as default993 } from "../icons/play";
import { default as default994 } from "../icons/plus";
import { default as default995 } from "../icons/plug-2";
import { default as default996 } from "../icons/pocket";
import { default as default997 } from "../icons/pocket-knife";
import { default as default998 } from "../icons/podcast";
import { default as default999 } from "../icons/pointer-off";
import { default as default1000 } from "../icons/pointer";
import { default as default1001 } from "../icons/popcorn";
import { default as default1002 } from "../icons/pound-sterling";
import { default as default1003 } from "../icons/popsicle";
import { default as default1004 } from "../icons/power-off";
import { default as default1005 } from "../icons/power";
import { default as default1006 } from "../icons/presentation";
import { default as default1007 } from "../icons/printer";
import { default as default1008 } from "../icons/projector";
import { default as default1009 } from "../icons/printer-check";
import { default as default1010 } from "../icons/puzzle";
import { default as default1011 } from "../icons/proportions";
import { default as default1012 } from "../icons/pyramid";
import { default as default1013 } from "../icons/qr-code";
import { default as default1014 } from "../icons/quote";
import { default as default1015 } from "../icons/rabbit";
import { default as default1016 } from "../icons/radiation";
import { default as default1017 } from "../icons/radar";
import { default as default1018 } from "../icons/radio-tower";
import { default as default1019 } from "../icons/radical";
import { default as default1020 } from "../icons/radio-receiver";
import { default as default1021 } from "../icons/radio";
import { default as default1022 } from "../icons/radius";
import { default as default1023 } from "../icons/rail-symbol";
import { default as default1024 } from "../icons/rainbow";
import { default as default1025 } from "../icons/rat";
import { default as default1026 } from "../icons/ratio";
import { default as default1027 } from "../icons/receipt-cent";
import { default as default1028 } from "../icons/receipt-indian-rupee";
import { default as default1029 } from "../icons/receipt-euro";
import { default as default1030 } from "../icons/receipt-japanese-yen";
import { default as default1031 } from "../icons/receipt-pound-sterling";
import { default as default1032 } from "../icons/receipt-swiss-franc";
import { default as default1033 } from "../icons/receipt-russian-ruble";
import { default as default1034 } from "../icons/receipt";
import { default as default1035 } from "../icons/rectangle-goggles";
import { default as default1036 } from "../icons/receipt-text";
import { default as default1037 } from "../icons/rectangle-horizontal";
import { default as default1038 } from "../icons/rectangle-vertical";
import { default as default1039 } from "../icons/recycle";
import { default as default1040 } from "../icons/redo-dot";
import { default as default1041 } from "../icons/refresh-ccw-dot";
import { default as default1042 } from "../icons/redo-2";
import { default as default1043 } from "../icons/redo";
import { default as default1044 } from "../icons/refresh-ccw";
import { default as default1045 } from "../icons/refresh-cw-off";
import { default as default1046 } from "../icons/refresh-cw";
import { default as default1047 } from "../icons/refrigerator";
import { default as default1048 } from "../icons/regex";
import { default as default1049 } from "../icons/repeat-1";
import { default as default1050 } from "../icons/repeat-2";
import { default as default1051 } from "../icons/remove-formatting";
import { default as default1052 } from "../icons/repeat";
import { default as default1053 } from "../icons/replace-all";
import { default as default1054 } from "../icons/replace";
import { default as default1055 } from "../icons/reply-all";
import { default as default1056 } from "../icons/reply";
import { default as default1057 } from "../icons/rewind";
import { default as default1058 } from "../icons/ribbon";
import { default as default1059 } from "../icons/rocket";
import { default as default1060 } from "../icons/rocking-chair";
import { default as default1061 } from "../icons/rotate-ccw-key";
import { default as default1062 } from "../icons/roller-coaster";
import { default as default1063 } from "../icons/rotate-ccw-square";
import { default as default1064 } from "../icons/rotate-ccw";
import { default as default1065 } from "../icons/rotate-cw-square";
import { default as default1066 } from "../icons/rotate-cw";
import { default as default1067 } from "../icons/route-off";
import { default as default1068 } from "../icons/route";
import { default as default1069 } from "../icons/router";
import { default as default1070 } from "../icons/rows-4";
import { default as default1071 } from "../icons/rss";
import { default as default1072 } from "../icons/ruler-dimension-line";
import { default as default1073 } from "../icons/ruler";
import { default as default1074 } from "../icons/russian-ruble";
import { default as default1075 } from "../icons/sailboat";
import { default as default1076 } from "../icons/salad";
import { default as default1077 } from "../icons/satellite-dish";
import { default as default1078 } from "../icons/sandwich";
import { default as default1079 } from "../icons/satellite";
import { default as default1080 } from "../icons/saudi-riyal";
import { default as default1081 } from "../icons/save-off";
import { default as default1082 } from "../icons/save-all";
import { default as default1083 } from "../icons/save";
import { default as default1084 } from "../icons/scale";
import { default as default1085 } from "../icons/scaling";
import { default as default1086 } from "../icons/scan-barcode";
import { default as default1087 } from "../icons/scan-eye";
import { default as default1088 } from "../icons/scan-face";
import { default as default1089 } from "../icons/scan-heart";
import { default as default1090 } from "../icons/scan-qr-code";
import { default as default1091 } from "../icons/scan-line";
import { default as default1092 } from "../icons/scan-search";
import { default as default1093 } from "../icons/scan-text";
import { default as default1094 } from "../icons/scan";
import { default as default1095 } from "../icons/school";
import { default as default1096 } from "../icons/scissors-line-dashed";
import { default as default1097 } from "../icons/scissors";
import { default as default1098 } from "../icons/screen-share-off";
import { default as default1099 } from "../icons/screen-share";
import { default as default1100 } from "../icons/scroll-text";
import { default as default1101 } from "../icons/scroll";
import { default as default1102 } from "../icons/search-code";
import { default as default1103 } from "../icons/search-check";
import { default as default1104 } from "../icons/search-slash";
import { default as default1105 } from "../icons/search-x";
import { default as default1106 } from "../icons/search";
import { default as default1107 } from "../icons/section";
import { default as default1108 } from "../icons/send-to-back";
import { default as default1109 } from "../icons/send";
import { default as default1110 } from "../icons/separator-vertical";
import { default as default1111 } from "../icons/separator-horizontal";
import { default as default1112 } from "../icons/server-cog";
import { default as default1113 } from "../icons/server-crash";
import { default as default1114 } from "../icons/server-off";
import { default as default1115 } from "../icons/settings-2";
import { default as default1116 } from "../icons/server";
import { default as default1117 } from "../icons/settings";
import { default as default1118 } from "../icons/shapes";
import { default as default1119 } from "../icons/share-2";
import { default as default1120 } from "../icons/share";
import { default as default1121 } from "../icons/sheet";
import { default as default1122 } from "../icons/shell";
import { default as default1123 } from "../icons/shield-alert";
import { default as default1124 } from "../icons/shield-ban";
import { default as default1125 } from "../icons/shield-check";
import { default as default1126 } from "../icons/shield-ellipsis";
import { default as default1127 } from "../icons/shield-half";
import { default as default1128 } from "../icons/shield-off";
import { default as default1129 } from "../icons/shield-minus";
import { default as default1130 } from "../icons/shield-question";
import { default as default1131 } from "../icons/shield-plus";
import { default as default1132 } from "../icons/shield-user";
import { default as default1133 } from "../icons/shield";
import { default as default1134 } from "../icons/ship-wheel";
import { default as default1135 } from "../icons/shirt";
import { default as default1136 } from "../icons/ship";
import { default as default1137 } from "../icons/shopping-bag";
import { default as default1138 } from "../icons/shopping-basket";
import { default as default1139 } from "../icons/shopping-cart";
import { default as default1140 } from "../icons/shovel";
import { default as default1141 } from "../icons/shower-head";
import { default as default1142 } from "../icons/shredder";
import { default as default1143 } from "../icons/shrimp";
import { default as default1144 } from "../icons/shrink";
import { default as default1145 } from "../icons/shuffle";
import { default as default1146 } from "../icons/shrub";
import { default as default1147 } from "../icons/sigma";
import { default as default1148 } from "../icons/signal-high";
import { default as default1149 } from "../icons/signal-medium";
import { default as default1150 } from "../icons/signal-low";
import { default as default1151 } from "../icons/signal-zero";
import { default as default1152 } from "../icons/signal";
import { default as default1153 } from "../icons/signature";
import { default as default1154 } from "../icons/signpost-big";
import { default as default1155 } from "../icons/signpost";
import { default as default1156 } from "../icons/siren";
import { default as default1157 } from "../icons/skip-forward";
import { default as default1158 } from "../icons/skip-back";
import { default as default1159 } from "../icons/skull";
import { default as default1160 } from "../icons/slack";
import { default as default1161 } from "../icons/slash";
import { default as default1162 } from "../icons/slice";
import { default as default1163 } from "../icons/smartphone-charging";
import { default as default1164 } from "../icons/smartphone-nfc";
import { default as default1165 } from "../icons/sliders-horizontal";
import { default as default1166 } from "../icons/smartphone";
import { default as default1167 } from "../icons/smile-plus";
import { default as default1168 } from "../icons/smile";
import { default as default1169 } from "../icons/snail";
import { default as default1170 } from "../icons/snowflake";
import { default as default1171 } from "../icons/sofa";
import { default as default1172 } from "../icons/soap-dispenser-droplet";
import { default as default1173 } from "../icons/soup";
import { default as default1174 } from "../icons/space";
import { default as default1175 } from "../icons/spade";
import { default as default1176 } from "../icons/sparkle";
import { default as default1177 } from "../icons/speaker";
import { default as default1178 } from "../icons/speech";
import { default as default1179 } from "../icons/spell-check-2";
import { default as default1180 } from "../icons/spline-pointer";
import { default as default1181 } from "../icons/spell-check";
import { default as default1182 } from "../icons/spline";
import { default as default1183 } from "../icons/split";
import { default as default1184 } from "../icons/spray-can";
import { default as default1185 } from "../icons/sprout";
import { default as default1186 } from "../icons/square-dashed-bottom-code";
import { default as default1187 } from "../icons/square-dashed-bottom";
import { default as default1188 } from "../icons/square-dashed-top-solid";
import { default as default1189 } from "../icons/square-radical";
import { default as default1190 } from "../icons/square-round-corner";
import { default as default1191 } from "../icons/square-square";
import { default as default1192 } from "../icons/square-stack";
import { default as default1193 } from "../icons/squares-exclude";
import { default as default1194 } from "../icons/square";
import { default as default1195 } from "../icons/squares-subtract";
import { default as default1196 } from "../icons/squares-intersect";
import { default as default1197 } from "../icons/squares-unite";
import { default as default1198 } from "../icons/stamp";
import { default as default1199 } from "../icons/squirrel";
import { default as default1200 } from "../icons/star-half";
import { default as default1201 } from "../icons/squircle";
import { default as default1202 } from "../icons/star-off";
import { default as default1203 } from "../icons/star";
import { default as default1204 } from "../icons/step-forward";
import { default as default1205 } from "../icons/step-back";
import { default as default1206 } from "../icons/sticker";
import { default as default1207 } from "../icons/stethoscope";
import { default as default1208 } from "../icons/sticky-note";
import { default as default1209 } from "../icons/stretch-vertical";
import { default as default1210 } from "../icons/store";
import { default as default1211 } from "../icons/stretch-horizontal";
import { default as default1212 } from "../icons/strikethrough";
import { default as default1213 } from "../icons/sun-medium";
import { default as default1214 } from "../icons/subscript";
import { default as default1215 } from "../icons/sun-moon";
import { default as default1216 } from "../icons/sun-dim";
import { default as default1217 } from "../icons/sun-snow";
import { default as default1218 } from "../icons/sunrise";
import { default as default1219 } from "../icons/sunset";
import { default as default1220 } from "../icons/sun";
import { default as default1221 } from "../icons/superscript";
import { default as default1222 } from "../icons/swatch-book";
import { default as default1223 } from "../icons/switch-camera";
import { default as default1224 } from "../icons/swiss-franc";
import { default as default1225 } from "../icons/swords";
import { default as default1226 } from "../icons/sword";
import { default as default1227 } from "../icons/syringe";
import { default as default1228 } from "../icons/table-2";
import { default as default1229 } from "../icons/table-cells-split";
import { default as default1230 } from "../icons/table-cells-merge";
import { default as default1231 } from "../icons/table-properties";
import { default as default1232 } from "../icons/table-columns-split";
import { default as default1233 } from "../icons/table-of-contents";
import { default as default1234 } from "../icons/table-rows-split";
import { default as default1235 } from "../icons/tablet-smartphone";
import { default as default1236 } from "../icons/table";
import { default as default1237 } from "../icons/tablets";
import { default as default1238 } from "../icons/tablet";
import { default as default1239 } from "../icons/tags";
import { default as default1240 } from "../icons/tag";
import { default as default1241 } from "../icons/tally-2";
import { default as default1242 } from "../icons/tally-3";
import { default as default1243 } from "../icons/tally-1";
import { default as default1244 } from "../icons/tally-4";
import { default as default1245 } from "../icons/tally-5";
import { default as default1246 } from "../icons/tangent";
import { default as default1247 } from "../icons/target";
import { default as default1248 } from "../icons/telescope";
import { default as default1249 } from "../icons/tent";
import { default as default1250 } from "../icons/terminal";
import { default as default1251 } from "../icons/tent-tree";
import { default as default1252 } from "../icons/test-tube";
import { default as default1253 } from "../icons/test-tubes";
import { default as default1254 } from "../icons/text-cursor";
import { default as default1255 } from "../icons/text-search";
import { default as default1256 } from "../icons/text-cursor-input";
import { default as default1257 } from "../icons/text-quote";
import { default as default1258 } from "../icons/text";
import { default as default1259 } from "../icons/theater";
import { default as default1260 } from "../icons/thermometer-snowflake";
import { default as default1261 } from "../icons/thermometer-sun";
import { default as default1262 } from "../icons/thumbs-down";
import { default as default1263 } from "../icons/thermometer";
import { default as default1264 } from "../icons/thumbs-up";
import { default as default1265 } from "../icons/ticket-check";
import { default as default1266 } from "../icons/ticket-minus";
import { default as default1267 } from "../icons/ticket-percent";
import { default as default1268 } from "../icons/ticket-plus";
import { default as default1269 } from "../icons/ticket-slash";
import { default as default1270 } from "../icons/ticket-x";
import { default as default1271 } from "../icons/ticket";
import { default as default1272 } from "../icons/tickets-plane";
import { default as default1273 } from "../icons/tickets";
import { default as default1274 } from "../icons/timer-off";
import { default as default1275 } from "../icons/timer-reset";
import { default as default1276 } from "../icons/timer";
import { default as default1277 } from "../icons/toggle-left";
import { default as default1278 } from "../icons/toggle-right";
import { default as default1279 } from "../icons/toilet";
import { default as default1280 } from "../icons/tornado";
import { default as default1281 } from "../icons/torus";
import { default as default1282 } from "../icons/touchpad-off";
import { default as default1283 } from "../icons/tower-control";
import { default as default1284 } from "../icons/touchpad";
import { default as default1285 } from "../icons/toy-brick";
import { default as default1286 } from "../icons/tractor";
import { default as default1287 } from "../icons/traffic-cone";
import { default as default1288 } from "../icons/train-front-tunnel";
import { default as default1289 } from "../icons/train-front";
import { default as default1290 } from "../icons/train-track";
import { default as default1291 } from "../icons/transgender";
import { default as default1292 } from "../icons/trash-2";
import { default as default1293 } from "../icons/trash";
import { default as default1294 } from "../icons/tree-deciduous";
import { default as default1295 } from "../icons/tree-pine";
import { default as default1296 } from "../icons/trees";
import { default as default1297 } from "../icons/trello";
import { default as default1298 } from "../icons/trending-down";
import { default as default1299 } from "../icons/trending-up-down";
import { default as default1300 } from "../icons/trending-up";
import { default as default1301 } from "../icons/triangle-right";
import { default as default1302 } from "../icons/triangle-dashed";
import { default as default1303 } from "../icons/triangle";
import { default as default1304 } from "../icons/trophy";
import { default as default1305 } from "../icons/truck";
import { default as default1306 } from "../icons/truck-electric";
import { default as default1307 } from "../icons/turtle";
import { default as default1308 } from "../icons/tv-minimal-play";
import { default as default1309 } from "../icons/twitch";
import { default as default1310 } from "../icons/tv";
import { default as default1311 } from "../icons/twitter";
import { default as default1312 } from "../icons/umbrella-off";
import { default as default1313 } from "../icons/type-outline";
import { default as default1314 } from "../icons/type";
import { default as default1315 } from "../icons/umbrella";
import { default as default1316 } from "../icons/underline";
import { default as default1317 } from "../icons/undo-2";
import { default as default1318 } from "../icons/undo-dot";
import { default as default1319 } from "../icons/undo";
import { default as default1320 } from "../icons/unfold-horizontal";
import { default as default1321 } from "../icons/ungroup";
import { default as default1322 } from "../icons/unfold-vertical";
import { default as default1323 } from "../icons/unlink";
import { default as default1324 } from "../icons/unplug";
import { default as default1325 } from "../icons/unlink-2";
import { default as default1326 } from "../icons/upload";
import { default as default1327 } from "../icons/usb";
import { default as default1328 } from "../icons/user-cog";
import { default as default1329 } from "../icons/user-check";
import { default as default1330 } from "../icons/user-lock";
import { default as default1331 } from "../icons/user-minus";
import { default as default1332 } from "../icons/user-pen";
import { default as default1333 } from "../icons/user-plus";
import { default as default1334 } from "../icons/user-round-pen";
import { default as default1335 } from "../icons/user-round-search";
import { default as default1336 } from "../icons/user-search";
import { default as default1337 } from "../icons/user-x";
import { default as default1338 } from "../icons/user";
import { default as default1339 } from "../icons/users";
import { default as default1340 } from "../icons/utility-pole";
import { default as default1341 } from "../icons/variable";
import { default as default1342 } from "../icons/vault";
import { default as default1343 } from "../icons/vegan";
import { default as default1344 } from "../icons/venus-and-mars";
import { default as default1345 } from "../icons/venetian-mask";
import { default as default1346 } from "../icons/venus";
import { default as default1347 } from "../icons/vibrate-off";
import { default as default1348 } from "../icons/vibrate";
import { default as default1349 } from "../icons/video-off";
import { default as default1350 } from "../icons/video";
import { default as default1351 } from "../icons/videotape";
import { default as default1352 } from "../icons/view";
import { default as default1353 } from "../icons/voicemail";
import { default as default1354 } from "../icons/volleyball";
import { default as default1355 } from "../icons/volume-1";
import { default as default1356 } from "../icons/volume-2";
import { default as default1357 } from "../icons/volume-off";
import { default as default1358 } from "../icons/volume-x";
import { default as default1359 } from "../icons/volume";
import { default as default1360 } from "../icons/vote";
import { default as default1361 } from "../icons/wallet-cards";
import { default as default1362 } from "../icons/wallet";
import { default as default1363 } from "../icons/wallpaper";
import { default as default1364 } from "../icons/wand";
import { default as default1365 } from "../icons/warehouse";
import { default as default1366 } from "../icons/washing-machine";
import { default as default1367 } from "../icons/watch";
import { default as default1368 } from "../icons/waves-ladder";
import { default as default1369 } from "../icons/waves";
import { default as default1370 } from "../icons/webcam";
import { default as default1371 } from "../icons/waypoints";
import { default as default1372 } from "../icons/webhook";
import { default as default1373 } from "../icons/weight";
import { default as default1374 } from "../icons/wheat-off";
import { default as default1375 } from "../icons/webhook-off";
import { default as default1376 } from "../icons/wheat";
import { default as default1377 } from "../icons/wifi-high";
import { default as default1378 } from "../icons/whole-word";
import { default as default1379 } from "../icons/wifi-low";
import { default as default1380 } from "../icons/wifi-off";
import { default as default1381 } from "../icons/wifi-pen";
import { default as default1382 } from "../icons/wifi-zero";
import { default as default1383 } from "../icons/wind-arrow-down";
import { default as default1384 } from "../icons/wifi";
import { default as default1385 } from "../icons/wind";
import { default as default1386 } from "../icons/wine-off";
import { default as default1387 } from "../icons/workflow";
import { default as default1388 } from "../icons/wine";
import { default as default1389 } from "../icons/wrap-text";
import { default as default1390 } from "../icons/worm";
import { default as default1391 } from "../icons/wrench";
import { default as default1392 } from "../icons/x";
import { default as default1393 } from "../icons/zap-off";
import { default as default1394 } from "../icons/youtube";
import { default as default1395 } from "../icons/zoom-in";
import { default as default1396 } from "../icons/zap";
import { default as default1397 } from "../icons/zoom-out";
import { default as default1398 } from "../icons/arrow-down-0-1";
import { default as default1399 } from "../icons/arrow-down-1-0";
import { default as default1400 } from "../icons/arrow-up-0-1";
import { default as default1401 } from "../icons/arrow-up-1-0";
import { default as default1402 } from "../icons/alarm-clock-check";
import { default as default1403 } from "../icons/alarm-check";
import { default as default1404 } from "../icons/alarm-clock-minus";
import { default as default1405 } from "../icons/alarm-minus";
import { default as default1406 } from "../icons/alarm-clock-plus";
import { default as default1407 } from "../icons/alarm-plus";
import { default as default1408 } from "../icons/arrow-down-a-z";
import { default as default1409 } from "../icons/arrow-down-az";
import { default as default1410 } from "../icons/arrow-down-wide-narrow";
import { default as default1411 } from "../icons/sort-desc";
import { default as default1412 } from "../icons/arrow-down-z-a";
import { default as default1413 } from "../icons/arrow-down-za";
import { default as default1414 } from "../icons/arrow-up-a-z";
import { default as default1415 } from "../icons/arrow-up-az";
import { default as default1416 } from "../icons/arrow-up-narrow-wide";
import { default as default1417 } from "../icons/sort-asc";
import { default as default1418 } from "../icons/arrow-up-z-a";
import { default as default1419 } from "../icons/arrow-up-za";
import { default as default1420 } from "../icons/axis-3d";
import { default as default1421 } from "../icons/axis-3-d";
import { default as default1422 } from "../icons/badge-check";
import { default as default1423 } from "../icons/verified";
import { default as default1424 } from "../icons/between-horizontal-end";
import {
  default as default1425
} from "../icons/between-horizonal-end";
import { default as default1426 } from "../icons/book-dashed";
import { default as default1427 } from "../icons/book-template";
import { default as default1428 } from "../icons/between-horizontal-start";
import {
  default as default1429
} from "../icons/between-horizonal-start";
import { default as default1430 } from "../icons/braces";
import { default as default1431 } from "../icons/curly-braces";
import { default as default1432 } from "../icons/captions";
import { default as default1433 } from "../icons/subtitles";
import { default as default1434 } from "../icons/chart-area";
import {
  default as default1435
} from "../icons/area-chart";
import { default as default1436 } from "../icons/chart-bar-big";
import {
  default as default1437
} from "../icons/bar-chart-horizontal-big";
import { default as default1438 } from "../icons/chart-bar";
import {
  default as default1439
} from "../icons/bar-chart-horizontal";
import { default as default1440 } from "../icons/chart-candlestick";
import {
  default as default1441
} from "../icons/candlestick-chart";
import { default as default1442 } from "../icons/chart-column-big";
import {
  default as default1443
} from "../icons/bar-chart-big";
import { default as default1444 } from "../icons/chart-column-increasing";
import {
  default as default1445
} from "../icons/bar-chart-4";
import { default as default1446 } from "../icons/chart-column";
import {
  default as default1447
} from "../icons/bar-chart-3";
import { default as default1448 } from "../icons/chart-line";
import {
  default as default1449
} from "../icons/line-chart";
import { default as default1450 } from "../icons/chart-no-axes-column-increasing";
import {
  default as default1451
} from "../icons/bar-chart";
import { default as default1452 } from "../icons/chart-no-axes-column";
import {
  default as default1453
} from "../icons/bar-chart-2";
import { default as default1454 } from "../icons/chart-no-axes-gantt";
import {
  default as default1455
} from "../icons/gantt-chart";
import { default as default1456 } from "../icons/chart-pie";
import {
  default as default1457
} from "../icons/pie-chart";
import { default as default1458 } from "../icons/chart-scatter";
import {
  default as default1459
} from "../icons/scatter-chart";
import { default as default1460 } from "../icons/circle-alert";
import { default as default1461 } from "../icons/alert-circle";
import { default as default1462 } from "../icons/circle-arrow-down";
import { default as default1463 } from "../icons/arrow-down-circle";
import { default as default1464 } from "../icons/circle-arrow-left";
import { default as default1465 } from "../icons/arrow-left-circle";
import { default as default1466 } from "../icons/circle-arrow-out-down-left";
import { default as default1467 } from "../icons/arrow-down-left-from-circle";
import { default as default1468 } from "../icons/circle-arrow-out-down-right";
import { default as default1469 } from "../icons/arrow-down-right-from-circle";
import { default as default1470 } from "../icons/circle-arrow-out-up-right";
import { default as default1471 } from "../icons/arrow-up-right-from-circle";
import { default as default1472 } from "../icons/circle-arrow-out-up-left";
import { default as default1473 } from "../icons/arrow-up-left-from-circle";
import { default as default1474 } from "../icons/circle-arrow-right";
import { default as default1475 } from "../icons/arrow-right-circle";
import { default as default1476 } from "../icons/circle-arrow-up";
import { default as default1477 } from "../icons/arrow-up-circle";
import { default as default1478 } from "../icons/circle-check-big";
import { default as default1479 } from "../icons/check-circle";
import { default as default1480 } from "../icons/circle-check";
import { default as default1481 } from "../icons/check-circle-2";
import { default as default1482 } from "../icons/circle-chevron-down";
import { default as default1483 } from "../icons/chevron-down-circle";
import { default as default1484 } from "../icons/circle-chevron-left";
import { default as default1485 } from "../icons/chevron-left-circle";
import { default as default1486 } from "../icons/circle-chevron-right";
import { default as default1487 } from "../icons/chevron-right-circle";
import { default as default1488 } from "../icons/circle-divide";
import { default as default1489 } from "../icons/divide-circle";
import { default as default1490 } from "../icons/circle-chevron-up";
import { default as default1491 } from "../icons/chevron-up-circle";
import { default as default1492 } from "../icons/circle-gauge";
import { default as default1493 } from "../icons/gauge-circle";
import { default as default1494 } from "../icons/circle-help";
import { default as default1495 } from "../icons/help-circle";
import { default as default1496 } from "../icons/circle-minus";
import { default as default1497 } from "../icons/minus-circle";
import { default as default1498 } from "../icons/circle-parking-off";
import { default as default1499 } from "../icons/parking-circle-off";
import { default as default1500 } from "../icons/circle-parking";
import { default as default1501 } from "../icons/parking-circle";
import { default as default1502 } from "../icons/circle-percent";
import { default as default1503 } from "../icons/percent-circle";
import { default as default1504 } from "../icons/circle-pause";
import { default as default1505 } from "../icons/pause-circle";
import { default as default1506 } from "../icons/circle-plus";
import { default as default1507 } from "../icons/plus-circle";
import { default as default1508 } from "../icons/circle-play";
import { default as default1509 } from "../icons/play-circle";
import { default as default1510 } from "../icons/circle-power";
import { default as default1511 } from "../icons/power-circle";
import { default as default1512 } from "../icons/circle-slash-2";
import { default as default1513 } from "../icons/circle-slashed";
import { default as default1514 } from "../icons/circle-user-round";
import {
  default as default1515
} from "../icons/user-circle-2";
import { default as default1516 } from "../icons/circle-stop";
import { default as default1517 } from "../icons/stop-circle";
import { default as default1518 } from "../icons/circle-user";
import { default as default1519 } from "../icons/user-circle";
import { default as default1520 } from "../icons/circle-x";
import { default as default1521 } from "../icons/x-circle";
import { default as default1522 } from "../icons/clipboard-pen-line";
import { default as default1523 } from "../icons/clipboard-signature";
import { default as default1524 } from "../icons/clipboard-pen";
import { default as default1525 } from "../icons/clipboard-edit";
import { default as default1526 } from "../icons/cloud-download";
import { default as default1527 } from "../icons/download-cloud";
import { default as default1528 } from "../icons/cloud-upload";
import { default as default1529 } from "../icons/upload-cloud";
import { default as default1530 } from "../icons/columns-2";
import { default as default1531 } from "../icons/columns";
import { default as default1532 } from "../icons/code-xml";
import {
  default as default1533
} from "../icons/code-2";
import { default as default1534 } from "../icons/columns-3-cog";
import { default as default1535 } from "../icons/columns-settings";
import { default as default1536 } from "../icons/table-config";
import { default as default1537 } from "../icons/columns-3";
import { default as default1538 } from "../icons/panels-left-right";
import { default as default1539 } from "../icons/contact-round";
import {
  default as default1540
} from "../icons/contact-2";
import { default as default1541 } from "../icons/diamond-percent";
import { default as default1542 } from "../icons/percent-diamond";
import { default as default1543 } from "../icons/earth";
import {
  default as default1544
} from "../icons/globe-2";
import { default as default1545 } from "../icons/ellipsis-vertical";
import { default as default1546 } from "../icons/more-vertical";
import { default as default1547 } from "../icons/ellipsis";
import { default as default1548 } from "../icons/more-horizontal";
import { default as default1549 } from "../icons/file-axis-3d";
import { default as default1550 } from "../icons/file-axis-3-d";
import { default as default1551 } from "../icons/file-chart-column-increasing";
import {
  default as default1552
} from "../icons/file-bar-chart";
import { default as default1553 } from "../icons/file-chart-line";
import {
  default as default1554
} from "../icons/file-line-chart";
import { default as default1555 } from "../icons/file-chart-column";
import {
  default as default1556
} from "../icons/file-bar-chart-2";
import { default as default1557 } from "../icons/file-chart-pie";
import {
  default as default1558
} from "../icons/file-pie-chart";
import { default as default1559 } from "../icons/file-cog";
import {
  default as default1560
} from "../icons/file-cog-2";
import { default as default1561 } from "../icons/file-pen";
import { default as default1562 } from "../icons/file-edit";
import { default as default1563 } from "../icons/file-pen-line";
import { default as default1564 } from "../icons/file-signature";
import { default as default1565 } from "../icons/folder-cog";
import {
  default as default1566
} from "../icons/folder-cog-2";
import { default as default1567 } from "../icons/folder-pen";
import { default as default1568 } from "../icons/folder-edit";
import { default as default1569 } from "../icons/funnel-x";
import {
  default as default1570
} from "../icons/filter-x";
import { default as default1571 } from "../icons/funnel";
import {
  default as default1572
} from "../icons/filter";
import { default as default1573 } from "../icons/git-commit-horizontal";
import { default as default1574 } from "../icons/git-commit";
import { default as default1575 } from "../icons/grid-2x2-check";
import { default as default1576 } from "../icons/grid-2-x-2-check";
import { default as default1577 } from "../icons/grid-2x2-plus";
import { default as default1578 } from "../icons/grid-2-x-2-plus";
import { default as default1579 } from "../icons/grid-2x2-x";
import { default as default1580 } from "../icons/grid-2-x-2-x";
import { default as default1581 } from "../icons/grid-2x2";
import { default as default1582 } from "../icons/grid-2-x-2";
import { default as default1583 } from "../icons/hand-helping";
import { default as default1584 } from "../icons/helping-hand";
import { default as default1585 } from "../icons/ice-cream-bowl";
import {
  default as default1586
} from "../icons/ice-cream-2";
import { default as default1587 } from "../icons/grid-3x3";
import { default as default1588 } from "../icons/grid";
import { default as default1589 } from "../icons/grid-3-x-3";
import { default as default1590 } from "../icons/house";
import { default as default1591 } from "../icons/home";
import { default as default1592 } from "../icons/ice-cream-cone";
import { default as default1593 } from "../icons/ice-cream";
import { default as default1594 } from "../icons/indent-decrease";
import { default as default1595 } from "../icons/outdent";
import { default as default1596 } from "../icons/indent-increase";
import { default as default1597 } from "../icons/indent";
import { default as default1598 } from "../icons/laptop-minimal";
import {
  default as default1599
} from "../icons/laptop-2";
import { default as default1600 } from "../icons/layers";
import { default as default1601 } from "../icons/layers-3";
import { default as default1602 } from "../icons/loader-circle";
import { default as default1603 } from "../icons/loader-2";
import { default as default1604 } from "../icons/lock-keyhole-open";
import { default as default1605 } from "../icons/unlock-keyhole";
import { default as default1606 } from "../icons/lock-open";
import { default as default1607 } from "../icons/unlock";
import { default as default1608 } from "../icons/mic-vocal";
import {
  default as default1609
} from "../icons/mic-2";
import { default as default1610 } from "../icons/move-3d";
import { default as default1611 } from "../icons/move-3-d";
import { default as default1612 } from "../icons/octagon-alert";
import { default as default1613 } from "../icons/alert-octagon";
import { default as default1614 } from "../icons/octagon-pause";
import { default as default1615 } from "../icons/pause-octagon";
import { default as default1616 } from "../icons/octagon-x";
import { default as default1617 } from "../icons/x-octagon";
import { default as default1618 } from "../icons/paintbrush-vertical";
import { default as default1619 } from "../icons/paintbrush-2";
import { default as default1620 } from "../icons/panel-bottom-dashed";
import { default as default1621 } from "../icons/panel-bottom-inactive";
import { default as default1622 } from "../icons/panel-left-dashed";
import { default as default1623 } from "../icons/panel-left-inactive";
import { default as default1624 } from "../icons/panel-left-close";
import { default as default1625 } from "../icons/sidebar-close";
import { default as default1626 } from "../icons/panel-left-open";
import { default as default1627 } from "../icons/sidebar-open";
import { default as default1628 } from "../icons/panel-left";
import { default as default1629 } from "../icons/sidebar";
import { default as default1630 } from "../icons/panel-right-dashed";
import { default as default1631 } from "../icons/panel-right-inactive";
import { default as default1632 } from "../icons/panel-top-dashed";
import { default as default1633 } from "../icons/panel-top-inactive";
import { default as default1634 } from "../icons/panels-top-left";
import { default as default1635 } from "../icons/layout";
import { default as default1636 } from "../icons/pen-line";
import {
  default as default1637
} from "../icons/edit-3";
import { default as default1638 } from "../icons/pen";
import {
  default as default1639
} from "../icons/edit-2";
import { default as default1640 } from "../icons/plug-zap";
import { default as default1641 } from "../icons/plug-zap-2";
import { default as default1642 } from "../icons/rectangle-ellipsis";
import { default as default1643 } from "../icons/form-input";
import { default as default1644 } from "../icons/rotate-3d";
import { default as default1645 } from "../icons/rotate-3-d";
import { default as default1646 } from "../icons/rows-2";
import { default as default1647 } from "../icons/rows";
import { default as default1648 } from "../icons/rows-3";
import { default as default1649 } from "../icons/panels-top-bottom";
import { default as default1650 } from "../icons/scale-3d";
import { default as default1651 } from "../icons/scale-3-d";
import { default as default1652 } from "../icons/send-horizontal";
import {
  default as default1653
} from "../icons/send-horizonal";
import { default as default1654 } from "../icons/shield-x";
import { default as default1655 } from "../icons/shield-close";
import { default as default1656 } from "../icons/sliders-vertical";
import { default as default1657 } from "../icons/sliders";
import { default as default1658 } from "../icons/sparkles";
import { default as default1659 } from "../icons/stars";
import { default as default1660 } from "../icons/square-activity";
import { default as default1661 } from "../icons/activity-square";
import { default as default1662 } from "../icons/square-arrow-down-left";
import { default as default1663 } from "../icons/arrow-down-left-square";
import { default as default1664 } from "../icons/square-arrow-down-right";
import { default as default1665 } from "../icons/arrow-down-right-square";
import { default as default1666 } from "../icons/square-arrow-left";
import { default as default1667 } from "../icons/arrow-left-square";
import { default as default1668 } from "../icons/square-arrow-down";
import { default as default1669 } from "../icons/arrow-down-square";
import { default as default1670 } from "../icons/square-arrow-out-down-left";
import { default as default1671 } from "../icons/arrow-down-left-from-square";
import { default as default1672 } from "../icons/square-arrow-out-down-right";
import { default as default1673 } from "../icons/arrow-down-right-from-square";
import { default as default1674 } from "../icons/square-arrow-out-up-left";
import { default as default1675 } from "../icons/arrow-up-left-from-square";
import { default as default1676 } from "../icons/square-arrow-out-up-right";
import { default as default1677 } from "../icons/arrow-up-right-from-square";
import { default as default1678 } from "../icons/square-arrow-right";
import { default as default1679 } from "../icons/arrow-right-square";
import { default as default1680 } from "../icons/square-arrow-up-right";
import { default as default1681 } from "../icons/arrow-up-right-square";
import { default as default1682 } from "../icons/square-arrow-up-left";
import { default as default1683 } from "../icons/arrow-up-left-square";
import { default as default1684 } from "../icons/square-arrow-up";
import { default as default1685 } from "../icons/arrow-up-square";
import { default as default1686 } from "../icons/square-asterisk";
import { default as default1687 } from "../icons/asterisk-square";
import { default as default1688 } from "../icons/square-bottom-dashed-scissors";
import { default as default1689 } from "../icons/scissors-square-dashed-bottom";
import { default as default1690 } from "../icons/square-chart-gantt";
import { default as default1691 } from "../icons/gantt-chart-square";
import { default as default1692 } from "../icons/square-gantt-chart";
import { default as default1693 } from "../icons/square-chevron-down";
import { default as default1694 } from "../icons/chevron-down-square";
import { default as default1695 } from "../icons/square-check";
import { default as default1696 } from "../icons/check-square-2";
import { default as default1697 } from "../icons/square-chevron-left";
import { default as default1698 } from "../icons/chevron-left-square";
import { default as default1699 } from "../icons/square-check-big";
import { default as default1700 } from "../icons/check-square";
import { default as default1701 } from "../icons/square-chevron-right";
import { default as default1702 } from "../icons/chevron-right-square";
import { default as default1703 } from "../icons/square-chevron-up";
import { default as default1704 } from "../icons/chevron-up-square";
import { default as default1705 } from "../icons/square-code";
import { default as default1706 } from "../icons/code-square";
import { default as default1707 } from "../icons/square-dashed-kanban";
import { default as default1708 } from "../icons/kanban-square-dashed";
import { default as default1709 } from "../icons/square-dashed-mouse-pointer";
import { default as default1710 } from "../icons/mouse-pointer-square-dashed";
import { default as default1711 } from "../icons/square-dashed";
import { default as default1712 } from "../icons/box-select";
import { default as default1713 } from "../icons/square-divide";
import { default as default1714 } from "../icons/divide-square";
import { default as default1715 } from "../icons/square-dot";
import { default as default1716 } from "../icons/dot-square";
import { default as default1717 } from "../icons/square-function";
import { default as default1718 } from "../icons/function-square";
import { default as default1719 } from "../icons/square-equal";
import { default as default1720 } from "../icons/equal-square";
import { default as default1721 } from "../icons/square-kanban";
import { default as default1722 } from "../icons/kanban-square";
import { default as default1723 } from "../icons/square-m";
import { default as default1724 } from "../icons/m-square";
import { default as default1725 } from "../icons/square-library";
import { default as default1726 } from "../icons/library-square";
import { default as default1727 } from "../icons/square-mouse-pointer";
import { default as default1728 } from "../icons/inspect";
import { default as default1729 } from "../icons/square-minus";
import { default as default1730 } from "../icons/minus-square";
import { default as default1731 } from "../icons/square-parking-off";
import { default as default1732 } from "../icons/parking-square-off";
import { default as default1733 } from "../icons/square-menu";
import { default as default1734 } from "../icons/menu-square";
import { default as default1735 } from "../icons/square-parking";
import { default as default1736 } from "../icons/parking-square";
import { default as default1737 } from "../icons/square-pen";
import { default as default1738 } from "../icons/pen-box";
import { default as default1739 } from "../icons/pen-square";
import { default as default1740 } from "../icons/edit";
import { default as default1741 } from "../icons/square-percent";
import { default as default1742 } from "../icons/percent-square";
import { default as default1743 } from "../icons/square-pi";
import { default as default1744 } from "../icons/pi-square";
import { default as default1745 } from "../icons/square-play";
import { default as default1746 } from "../icons/play-square";
import { default as default1747 } from "../icons/square-pilcrow";
import { default as default1748 } from "../icons/pilcrow-square";
import { default as default1749 } from "../icons/square-power";
import { default as default1750 } from "../icons/power-square";
import { default as default1751 } from "../icons/square-plus";
import { default as default1752 } from "../icons/plus-square";
import { default as default1753 } from "../icons/square-scissors";
import { default as default1754 } from "../icons/scissors-square";
import { default as default1755 } from "../icons/square-slash";
import { default as default1756 } from "../icons/slash-square";
import { default as default1757 } from "../icons/square-sigma";
import { default as default1758 } from "../icons/sigma-square";
import { default as default1759 } from "../icons/square-split-horizontal";
import { default as default1760 } from "../icons/split-square-horizontal";
import { default as default1761 } from "../icons/square-split-vertical";
import { default as default1762 } from "../icons/split-square-vertical";
import { default as default1763 } from "../icons/square-user-round";
import {
  default as default1764
} from "../icons/user-square-2";
import { default as default1765 } from "../icons/square-terminal";
import { default as default1766 } from "../icons/terminal-square";
import { default as default1767 } from "../icons/square-user";
import { default as default1768 } from "../icons/user-square";
import { default as default1769 } from "../icons/square-x";
import { default as default1770 } from "../icons/x-square";
import { default as default1771 } from "../icons/test-tube-diagonal";
import { default as default1772 } from "../icons/test-tube-2";
import { default as default1773 } from "../icons/text-select";
import { default as default1774 } from "../icons/text-selection";
import { default as default1775 } from "../icons/tram-front";
import { default as default1776 } from "../icons/train";
import { default as default1777 } from "../icons/tree-palm";
import { default as default1778 } from "../icons/palmtree";
import { default as default1779 } from "../icons/triangle-alert";
import { default as default1780 } from "../icons/alert-triangle";
import { default as default1781 } from "../icons/tv-minimal";
import {
  default as default1782
} from "../icons/tv-2";
import { default as default1783 } from "../icons/university";
import {
  default as default1784
} from "../icons/school-2";
import { default as default1785 } from "../icons/user-round-check";
import {
  default as default1786
} from "../icons/user-check-2";
import { default as default1787 } from "../icons/user-round-cog";
import {
  default as default1788
} from "../icons/user-cog-2";
import { default as default1789 } from "../icons/user-round-minus";
import {
  default as default1790
} from "../icons/user-minus-2";
import { default as default1791 } from "../icons/user-round-plus";
import {
  default as default1792
} from "../icons/user-plus-2";
import { default as default1793 } from "../icons/user-round-x";
import {
  default as default1794
} from "../icons/user-x-2";
import { default as default1795 } from "../icons/user-round";
import {
  default as default1796
} from "../icons/user-2";
import { default as default1797 } from "../icons/users-round";
import {
  default as default1798
} from "../icons/users-2";
import { default as default1799 } from "../icons/utensils-crossed";
import { default as default1800 } from "../icons/fork-knife-crossed";
import { default as default1801 } from "../icons/utensils";
import { default as default1802 } from "../icons/fork-knife";
import { default as default1803 } from "../icons/wallet-minimal";
import {
  default as default1804
} from "../icons/wallet-2";
import { default as default1805 } from "../icons/wand-sparkles";
import {
  default as default1806
} from "../icons/wand-2";
export {
  default2 as AArrowDownIcon,
  default3 as AArrowUpIcon,
  default4 as ALargeSmallIcon,
  default5 as AccessibilityIcon,
  default6 as ActivityIcon,
  default1661 as ActivitySquareIcon,
  default7 as AirVentIcon,
  default8 as AirplayIcon,
  default1403 as AlarmCheckIcon,
  default1402 as AlarmClockCheckIcon,
  default10 as AlarmClockIcon,
  default1404 as AlarmClockMinusIcon,
  default9 as AlarmClockOffIcon,
  default1406 as AlarmClockPlusIcon,
  default1405 as AlarmMinusIcon,
  default1407 as AlarmPlusIcon,
  default11 as AlarmSmokeIcon,
  default12 as AlbumIcon,
  default1461 as AlertCircleIcon,
  default1613 as AlertOctagonIcon,
  default1780 as AlertTriangleIcon,
  default13 as AlignCenterHorizontalIcon,
  default15 as AlignCenterIcon,
  default14 as AlignCenterVerticalIcon,
  default16 as AlignEndHorizontalIcon,
  default17 as AlignEndVerticalIcon,
  default18 as AlignHorizontalDistributeCenterIcon,
  default19 as AlignHorizontalDistributeEndIcon,
  default21 as AlignHorizontalDistributeStartIcon,
  default20 as AlignHorizontalJustifyCenterIcon,
  default22 as AlignHorizontalJustifyEndIcon,
  default23 as AlignHorizontalJustifyStartIcon,
  default24 as AlignHorizontalSpaceAroundIcon,
  default25 as AlignHorizontalSpaceBetweenIcon,
  default26 as AlignJustifyIcon,
  default27 as AlignLeftIcon,
  default28 as AlignRightIcon,
  default29 as AlignStartHorizontalIcon,
  default30 as AlignStartVerticalIcon,
  default31 as AlignVerticalDistributeCenterIcon,
  default32 as AlignVerticalDistributeEndIcon,
  default33 as AlignVerticalDistributeStartIcon,
  default34 as AlignVerticalJustifyCenterIcon,
  default37 as AlignVerticalJustifyEndIcon,
  default35 as AlignVerticalJustifyStartIcon,
  default36 as AlignVerticalSpaceAroundIcon,
  default38 as AlignVerticalSpaceBetweenIcon,
  default39 as AmbulanceIcon,
  default40 as AmpersandIcon,
  default41 as AmpersandsIcon,
  default42 as AmphoraIcon,
  default43 as AnchorIcon,
  default44 as AngryIcon,
  default46 as AnnoyedIcon,
  default45 as AntennaIcon,
  default47 as AnvilIcon,
  default48 as ApertureIcon,
  default52 as AppWindowIcon,
  default49 as AppWindowMacIcon,
  default50 as AppleIcon,
  default54 as ArchiveIcon,
  default51 as ArchiveRestoreIcon,
  default53 as ArchiveXIcon,
  default1435 as AreaChartIcon,
  default56 as ArmchairIcon,
  default55 as ArrowBigDownDashIcon,
  default58 as ArrowBigDownIcon,
  default57 as ArrowBigLeftDashIcon,
  default60 as ArrowBigLeftIcon,
  default59 as ArrowBigRightDashIcon,
  default61 as ArrowBigRightIcon,
  default62 as ArrowBigUpDashIcon,
  default63 as ArrowBigUpIcon,
  default1398 as ArrowDown01Icon,
  default1399 as ArrowDown10Icon,
  default1408 as ArrowDownAZIcon,
  default1409 as ArrowDownAzIcon,
  default1463 as ArrowDownCircleIcon,
  default64 as ArrowDownFromLineIcon,
  default71 as ArrowDownIcon,
  default1467 as ArrowDownLeftFromCircleIcon,
  default1671 as ArrowDownLeftFromSquareIcon,
  default65 as ArrowDownLeftIcon,
  default1663 as ArrowDownLeftSquareIcon,
  default66 as ArrowDownNarrowWideIcon,
  default1469 as ArrowDownRightFromCircleIcon,
  default1673 as ArrowDownRightFromSquareIcon,
  default67 as ArrowDownRightIcon,
  default1665 as ArrowDownRightSquareIcon,
  default1669 as ArrowDownSquareIcon,
  default69 as ArrowDownToDotIcon,
  default70 as ArrowDownToLineIcon,
  default68 as ArrowDownUpIcon,
  default1410 as ArrowDownWideNarrowIcon,
  default1412 as ArrowDownZAIcon,
  default1413 as ArrowDownZaIcon,
  default1465 as ArrowLeftCircleIcon,
  default72 as ArrowLeftFromLineIcon,
  default75 as ArrowLeftIcon,
  default73 as ArrowLeftRightIcon,
  default1667 as ArrowLeftSquareIcon,
  default74 as ArrowLeftToLineIcon,
  default1475 as ArrowRightCircleIcon,
  default76 as ArrowRightFromLineIcon,
  default79 as ArrowRightIcon,
  default78 as ArrowRightLeftIcon,
  default1679 as ArrowRightSquareIcon,
  default77 as ArrowRightToLineIcon,
  default1400 as ArrowUp01Icon,
  default1401 as ArrowUp10Icon,
  default1414 as ArrowUpAZIcon,
  default1415 as ArrowUpAzIcon,
  default1477 as ArrowUpCircleIcon,
  default80 as ArrowUpDownIcon,
  default81 as ArrowUpFromDotIcon,
  default82 as ArrowUpFromLineIcon,
  default87 as ArrowUpIcon,
  default1473 as ArrowUpLeftFromCircleIcon,
  default1675 as ArrowUpLeftFromSquareIcon,
  default83 as ArrowUpLeftIcon,
  default1683 as ArrowUpLeftSquareIcon,
  default1416 as ArrowUpNarrowWideIcon,
  default1471 as ArrowUpRightFromCircleIcon,
  default1677 as ArrowUpRightFromSquareIcon,
  default84 as ArrowUpRightIcon,
  default1681 as ArrowUpRightSquareIcon,
  default1685 as ArrowUpSquareIcon,
  default85 as ArrowUpToLineIcon,
  default86 as ArrowUpWideNarrowIcon,
  default1418 as ArrowUpZAIcon,
  default1419 as ArrowUpZaIcon,
  default88 as ArrowsUpFromLineIcon,
  default89 as AsteriskIcon,
  default1687 as AsteriskSquareIcon,
  default91 as AtSignIcon,
  default90 as AtomIcon,
  default93 as AudioLinesIcon,
  default92 as AudioWaveformIcon,
  default94 as AwardIcon,
  default95 as AxeIcon,
  default1421 as Axis3DIcon,
  default1420 as Axis3dIcon,
  default96 as BabyIcon,
  default97 as BackpackIcon,
  default98 as BadgeAlertIcon,
  default100 as BadgeCentIcon,
  default1422 as BadgeCheckIcon,
  default99 as BadgeDollarSignIcon,
  default101 as BadgeEuroIcon,
  default104 as BadgeHelpIcon,
  default113 as BadgeIcon,
  default102 as BadgeIndianRupeeIcon,
  default103 as BadgeInfoIcon,
  default105 as BadgeJapaneseYenIcon,
  default106 as BadgeMinusIcon,
  default108 as BadgePercentIcon,
  default107 as BadgePlusIcon,
  default109 as BadgePoundSterlingIcon,
  default110 as BadgeRussianRubleIcon,
  default111 as BadgeSwissFrancIcon,
  default112 as BadgeXIcon,
  default114 as BaggageClaimIcon,
  default115 as BanIcon,
  default116 as BananaIcon,
  default117 as BandageIcon,
  default118 as BanknoteArrowDownIcon,
  default119 as BanknoteArrowUpIcon,
  default121 as BanknoteIcon,
  default120 as BanknoteXIcon,
  default1453 as BarChart2Icon,
  default1447 as BarChart3Icon,
  default1445 as BarChart4Icon,
  default1443 as BarChartBigIcon,
  default1437 as BarChartHorizontalBigIcon,
  default1439 as BarChartHorizontalIcon,
  default1451 as BarChartIcon,
  default122 as BarcodeIcon,
  default123 as BaselineIcon,
  default124 as BathIcon,
  default125 as BatteryChargingIcon,
  default126 as BatteryFullIcon,
  default130 as BatteryIcon,
  default127 as BatteryLowIcon,
  default128 as BatteryMediumIcon,
  default131 as BatteryPlusIcon,
  default129 as BatteryWarningIcon,
  default132 as BeakerIcon,
  default134 as BeanIcon,
  default133 as BeanOffIcon,
  default135 as BedDoubleIcon,
  default136 as BedIcon,
  default137 as BedSingleIcon,
  default138 as BeefIcon,
  default140 as BeerIcon,
  default139 as BeerOffIcon,
  default141 as BellDotIcon,
  default142 as BellElectricIcon,
  default147 as BellIcon,
  default144 as BellMinusIcon,
  default143 as BellOffIcon,
  default145 as BellPlusIcon,
  default146 as BellRingIcon,
  default1425 as BetweenHorizonalEndIcon,
  default1429 as BetweenHorizonalStartIcon,
  default1424 as BetweenHorizontalEndIcon,
  default1428 as BetweenHorizontalStartIcon,
  default148 as BetweenVerticalEndIcon,
  default149 as BetweenVerticalStartIcon,
  default150 as BicepsFlexedIcon,
  default151 as BikeIcon,
  default152 as BinaryIcon,
  default153 as BinocularsIcon,
  default156 as BiohazardIcon,
  default154 as BirdIcon,
  default157 as BitcoinIcon,
  default155 as BlendIcon,
  default158 as BlindsIcon,
  default160 as BlocksIcon,
  default159 as BluetoothConnectedIcon,
  default163 as BluetoothIcon,
  default161 as BluetoothOffIcon,
  default162 as BluetoothSearchingIcon,
  default166 as BoldIcon,
  default165 as BoltIcon,
  default164 as BombIcon,
  default167 as BoneIcon,
  default168 as BookAIcon,
  default169 as BookAlertIcon,
  default170 as BookAudioIcon,
  default171 as BookCheckIcon,
  default172 as BookCopyIcon,
  default1426 as BookDashedIcon,
  default173 as BookDownIcon,
  default174 as BookHeadphonesIcon,
  default175 as BookHeartIcon,
  default191 as BookIcon,
  default176 as BookImageIcon,
  default177 as BookKeyIcon,
  default178 as BookLockIcon,
  default179 as BookMarkedIcon,
  default180 as BookMinusIcon,
  default181 as BookOpenCheckIcon,
  default183 as BookOpenIcon,
  default182 as BookOpenTextIcon,
  default184 as BookPlusIcon,
  default1427 as BookTemplateIcon,
  default185 as BookTextIcon,
  default186 as BookTypeIcon,
  default187 as BookUp2Icon,
  default188 as BookUpIcon,
  default189 as BookUserIcon,
  default190 as BookXIcon,
  default192 as BookmarkCheckIcon,
  default196 as BookmarkIcon,
  default193 as BookmarkMinusIcon,
  default195 as BookmarkPlusIcon,
  default194 as BookmarkXIcon,
  default197 as BoomBoxIcon,
  default200 as BotIcon,
  default198 as BotMessageSquareIcon,
  default199 as BotOffIcon,
  default202 as BowArrowIcon,
  default201 as BoxIcon,
  default1712 as BoxSelectIcon,
  default203 as BoxesIcon,
  default1430 as BracesIcon,
  default205 as BracketsIcon,
  default204 as BrainCircuitIcon,
  default206 as BrainCogIcon,
  default207 as BrainIcon,
  default208 as BrickWallFireIcon,
  default209 as BrickWallIcon,
  default210 as BriefcaseBusinessIcon,
  default212 as BriefcaseConveyorBeltIcon,
  default213 as BriefcaseIcon,
  default211 as BriefcaseMedicalIcon,
  default214 as BringToFrontIcon,
  default215 as BrushCleaningIcon,
  default216 as BrushIcon,
  default218 as BubblesIcon,
  default219 as BugIcon,
  default217 as BugOffIcon,
  default220 as BugPlayIcon,
  default221 as Building2Icon,
  default222 as BuildingIcon,
  default224 as BusFrontIcon,
  default223 as BusIcon,
  default225 as CableCarIcon,
  default226 as CableIcon,
  default228 as CakeIcon,
  default227 as CakeSliceIcon,
  default229 as CalculatorIcon,
  default231 as Calendar1Icon,
  default230 as CalendarArrowDownIcon,
  default233 as CalendarArrowUpIcon,
  default232 as CalendarCheck2Icon,
  default234 as CalendarCheckIcon,
  default235 as CalendarClockIcon,
  default236 as CalendarCogIcon,
  default237 as CalendarDaysIcon,
  default238 as CalendarFoldIcon,
  default239 as CalendarHeartIcon,
  default250 as CalendarIcon,
  default240 as CalendarMinus2Icon,
  default241 as CalendarMinusIcon,
  default242 as CalendarOffIcon,
  default243 as CalendarPlus2Icon,
  default244 as CalendarPlusIcon,
  default245 as CalendarRangeIcon,
  default246 as CalendarSearchIcon,
  default247 as CalendarSyncIcon,
  default248 as CalendarX2Icon,
  default249 as CalendarXIcon,
  default252 as CameraIcon,
  default251 as CameraOffIcon,
  default1441 as CandlestickChartIcon,
  default253 as CandyCaneIcon,
  default255 as CandyIcon,
  default254 as CandyOffIcon,
  default256 as CannabisIcon,
  default1432 as CaptionsIcon,
  default257 as CaptionsOffIcon,
  default258 as CarFrontIcon,
  default259 as CarIcon,
  default260 as CarTaxiFrontIcon,
  default261 as CaravanIcon,
  default262 as CardSimIcon,
  default263 as CarrotIcon,
  default264 as CaseLowerIcon,
  default265 as CaseSensitiveIcon,
  default266 as CaseUpperIcon,
  default267 as CassetteTapeIcon,
  default268 as CastIcon,
  default270 as CastleIcon,
  default269 as CatIcon,
  default271 as CctvIcon,
  default1434 as ChartAreaIcon,
  default1436 as ChartBarBigIcon,
  default272 as ChartBarDecreasingIcon,
  default1438 as ChartBarIcon,
  default273 as ChartBarIncreasingIcon,
  default274 as ChartBarStackedIcon,
  default1440 as ChartCandlestickIcon,
  default1442 as ChartColumnBigIcon,
  default275 as ChartColumnDecreasingIcon,
  default1446 as ChartColumnIcon,
  default1444 as ChartColumnIncreasingIcon,
  default276 as ChartColumnStackedIcon,
  default277 as ChartGanttIcon,
  default1448 as ChartLineIcon,
  default278 as ChartNetworkIcon,
  default279 as ChartNoAxesColumnDecreasingIcon,
  default1452 as ChartNoAxesColumnIcon,
  default1450 as ChartNoAxesColumnIncreasingIcon,
  default280 as ChartNoAxesCombinedIcon,
  default1454 as ChartNoAxesGanttIcon,
  default1456 as ChartPieIcon,
  default1458 as ChartScatterIcon,
  default281 as ChartSplineIcon,
  default282 as CheckCheckIcon,
  default1481 as CheckCircle2Icon,
  default1479 as CheckCircleIcon,
  default284 as CheckIcon,
  default283 as CheckLineIcon,
  default1696 as CheckSquare2Icon,
  default1700 as CheckSquareIcon,
  default285 as ChefHatIcon,
  default287 as CherryIcon,
  default1483 as ChevronDownCircleIcon,
  default286 as ChevronDownIcon,
  default1694 as ChevronDownSquareIcon,
  default288 as ChevronFirstIcon,
  default289 as ChevronLastIcon,
  default1485 as ChevronLeftCircleIcon,
  default291 as ChevronLeftIcon,
  default1698 as ChevronLeftSquareIcon,
  default1487 as ChevronRightCircleIcon,
  default290 as ChevronRightIcon,
  default1702 as ChevronRightSquareIcon,
  default1491 as ChevronUpCircleIcon,
  default292 as ChevronUpIcon,
  default1704 as ChevronUpSquareIcon,
  default294 as ChevronsDownIcon,
  default293 as ChevronsDownUpIcon,
  default297 as ChevronsLeftIcon,
  default295 as ChevronsLeftRightEllipsisIcon,
  default296 as ChevronsLeftRightIcon,
  default299 as ChevronsRightIcon,
  default298 as ChevronsRightLeftIcon,
  default300 as ChevronsUpDownIcon,
  default302 as ChevronsUpIcon,
  default301 as ChromeIcon,
  default303 as ChurchIcon,
  default305 as CigaretteIcon,
  default304 as CigaretteOffIcon,
  default1460 as CircleAlertIcon,
  default1462 as CircleArrowDownIcon,
  default1464 as CircleArrowLeftIcon,
  default1466 as CircleArrowOutDownLeftIcon,
  default1468 as CircleArrowOutDownRightIcon,
  default1472 as CircleArrowOutUpLeftIcon,
  default1470 as CircleArrowOutUpRightIcon,
  default1474 as CircleArrowRightIcon,
  default1476 as CircleArrowUpIcon,
  default1478 as CircleCheckBigIcon,
  default1480 as CircleCheckIcon,
  default1482 as CircleChevronDownIcon,
  default1484 as CircleChevronLeftIcon,
  default1486 as CircleChevronRightIcon,
  default1490 as CircleChevronUpIcon,
  default306 as CircleDashedIcon,
  default1488 as CircleDivideIcon,
  default308 as CircleDollarSignIcon,
  default307 as CircleDotDashedIcon,
  default309 as CircleDotIcon,
  default310 as CircleEllipsisIcon,
  default311 as CircleEqualIcon,
  default312 as CircleFadingArrowUpIcon,
  default313 as CircleFadingPlusIcon,
  default1492 as CircleGaugeIcon,
  default1494 as CircleHelpIcon,
  default318 as CircleIcon,
  default1496 as CircleMinusIcon,
  default314 as CircleOffIcon,
  default1500 as CircleParkingIcon,
  default1498 as CircleParkingOffIcon,
  default1504 as CirclePauseIcon,
  default1502 as CirclePercentIcon,
  default1508 as CirclePlayIcon,
  default1506 as CirclePlusIcon,
  default315 as CirclePoundSterlingIcon,
  default1510 as CirclePowerIcon,
  default1512 as CircleSlash2Icon,
  default316 as CircleSlashIcon,
  default1513 as CircleSlashedIcon,
  default317 as CircleSmallIcon,
  default1516 as CircleStopIcon,
  default1518 as CircleUserIcon,
  default1514 as CircleUserRoundIcon,
  default1520 as CircleXIcon,
  default319 as CircuitBoardIcon,
  default320 as CitrusIcon,
  default321 as ClapperboardIcon,
  default322 as ClipboardCheckIcon,
  default323 as ClipboardCopyIcon,
  default1525 as ClipboardEditIcon,
  default330 as ClipboardIcon,
  default324 as ClipboardListIcon,
  default325 as ClipboardMinusIcon,
  default326 as ClipboardPasteIcon,
  default1524 as ClipboardPenIcon,
  default1522 as ClipboardPenLineIcon,
  default327 as ClipboardPlusIcon,
  default1523 as ClipboardSignatureIcon,
  default329 as ClipboardTypeIcon,
  default328 as ClipboardXIcon,
  default331 as Clock10Icon,
  default332 as Clock11Icon,
  default334 as Clock12Icon,
  default333 as Clock1Icon,
  default335 as Clock2Icon,
  default336 as Clock3Icon,
  default337 as Clock4Icon,
  default338 as Clock5Icon,
  default339 as Clock6Icon,
  default340 as Clock7Icon,
  default341 as Clock8Icon,
  default342 as Clock9Icon,
  default343 as ClockAlertIcon,
  default344 as ClockArrowDownIcon,
  default345 as ClockArrowUpIcon,
  default347 as ClockFadingIcon,
  default348 as ClockIcon,
  default346 as ClockPlusIcon,
  default351 as CloudAlertIcon,
  default349 as CloudCheckIcon,
  default350 as CloudCogIcon,
  default1526 as CloudDownloadIcon,
  default352 as CloudDrizzleIcon,
  default353 as CloudFogIcon,
  default354 as CloudHailIcon,
  default364 as CloudIcon,
  default355 as CloudLightningIcon,
  default357 as CloudMoonIcon,
  default356 as CloudMoonRainIcon,
  default358 as CloudOffIcon,
  default363 as CloudRainIcon,
  default359 as CloudRainWindIcon,
  default360 as CloudSnowIcon,
  default362 as CloudSunIcon,
  default361 as CloudSunRainIcon,
  default1528 as CloudUploadIcon,
  default365 as CloudyIcon,
  default368 as CloverIcon,
  default366 as ClubIcon,
  default1533 as Code2Icon,
  default367 as CodeIcon,
  default1706 as CodeSquareIcon,
  default1532 as CodeXmlIcon,
  default369 as CodepenIcon,
  default370 as CodesandboxIcon,
  default371 as CoffeeIcon,
  default372 as CogIcon,
  default373 as CoinsIcon,
  default1530 as Columns2Icon,
  default1534 as Columns3CogIcon,
  default1537 as Columns3Icon,
  default374 as Columns4Icon,
  default1531 as ColumnsIcon,
  default1535 as ColumnsSettingsIcon,
  default376 as CombineIcon,
  default375 as CommandIcon,
  default377 as CompassIcon,
  default379 as ComponentIcon,
  default378 as ComputerIcon,
  default380 as ConciergeBellIcon,
  default382 as ConeIcon,
  default381 as ConstructionIcon,
  default1540 as Contact2Icon,
  default383 as ContactIcon,
  default1539 as ContactRoundIcon,
  default385 as ContainerIcon,
  default384 as ContrastIcon,
  default386 as CookieIcon,
  default387 as CookingPotIcon,
  default388 as CopyCheckIcon,
  default393 as CopyIcon,
  default389 as CopyMinusIcon,
  default390 as CopyPlusIcon,
  default391 as CopySlashIcon,
  default392 as CopyXIcon,
  default394 as CopyleftIcon,
  default395 as CopyrightIcon,
  default396 as CornerDownLeftIcon,
  default397 as CornerDownRightIcon,
  default399 as CornerLeftDownIcon,
  default398 as CornerLeftUpIcon,
  default400 as CornerRightDownIcon,
  default401 as CornerRightUpIcon,
  default404 as CornerUpLeftIcon,
  default402 as CornerUpRightIcon,
  default403 as CpuIcon,
  default405 as CreativeCommonsIcon,
  default407 as CreditCardIcon,
  default406 as CroissantIcon,
  default408 as CropIcon,
  default409 as CrossIcon,
  default410 as CrosshairIcon,
  default411 as CrownIcon,
  default412 as CuboidIcon,
  default413 as CupSodaIcon,
  default1431 as CurlyBracesIcon,
  default414 as CurrencyIcon,
  default415 as CylinderIcon,
  default416 as DamIcon,
  default417 as DatabaseBackupIcon,
  default419 as DatabaseIcon,
  default418 as DatabaseZapIcon,
  default420 as DecimalsArrowLeftIcon,
  default422 as DecimalsArrowRightIcon,
  default421 as DeleteIcon,
  default424 as DessertIcon,
  default423 as DiameterIcon,
  default427 as DiamondIcon,
  default425 as DiamondMinusIcon,
  default1541 as DiamondPercentIcon,
  default426 as DiamondPlusIcon,
  default428 as Dice1Icon,
  default429 as Dice2Icon,
  default430 as Dice3Icon,
  default431 as Dice4Icon,
  default432 as Dice5Icon,
  default433 as Dice6Icon,
  default434 as DicesIcon,
  default436 as DiffIcon,
  default435 as Disc2Icon,
  default438 as Disc3Icon,
  default437 as DiscAlbumIcon,
  default439 as DiscIcon,
  default1489 as DivideCircleIcon,
  default440 as DivideIcon,
  default1714 as DivideSquareIcon,
  default441 as DnaIcon,
  default442 as DnaOffIcon,
  default443 as DockIcon,
  default444 as DogIcon,
  default445 as DollarSignIcon,
  default447 as DonutIcon,
  default448 as DoorClosedIcon,
  default446 as DoorClosedLockedIcon,
  default449 as DoorOpenIcon,
  default450 as DotIcon,
  default1716 as DotSquareIcon,
  default1527 as DownloadCloudIcon,
  default451 as DownloadIcon,
  default452 as DraftingCompassIcon,
  default453 as DramaIcon,
  default454 as DribbbleIcon,
  default455 as DrillIcon,
  default457 as DropletIcon,
  default456 as DropletOffIcon,
  default458 as DropletsIcon,
  default459 as DrumIcon,
  default460 as DrumstickIcon,
  default461 as DumbbellIcon,
  default463 as EarIcon,
  default462 as EarOffIcon,
  default1543 as EarthIcon,
  default464 as EarthLockIcon,
  default465 as EclipseIcon,
  default1639 as Edit2Icon,
  default1637 as Edit3Icon,
  default1740 as EditIcon,
  default466 as EggFriedIcon,
  default468 as EggIcon,
  default467 as EggOffIcon,
  default1547 as EllipsisIcon,
  default1545 as EllipsisVerticalIcon,
  default469 as EqualApproximatelyIcon,
  default470 as EqualIcon,
  default471 as EqualNotIcon,
  default1720 as EqualSquareIcon,
  default472 as EraserIcon,
  default473 as EthernetPortIcon,
  default474 as EuroIcon,
  default475 as ExpandIcon,
  default476 as ExternalLinkIcon,
  default477 as EyeClosedIcon,
  default479 as EyeIcon,
  default478 as EyeOffIcon,
  default480 as FacebookIcon,
  default481 as FactoryIcon,
  default482 as FanIcon,
  default483 as FastForwardIcon,
  default484 as FeatherIcon,
  default485 as FenceIcon,
  default486 as FerrisWheelIcon,
  default488 as FigmaIcon,
  default487 as FileArchiveIcon,
  default489 as FileAudio2Icon,
  default490 as FileAudioIcon,
  default1550 as FileAxis3DIcon,
  default1549 as FileAxis3dIcon,
  default492 as FileBadge2Icon,
  default491 as FileBadgeIcon,
  default1556 as FileBarChart2Icon,
  default1552 as FileBarChartIcon,
  default493 as FileBoxIcon,
  default1555 as FileChartColumnIcon,
  default1551 as FileChartColumnIncreasingIcon,
  default1553 as FileChartLineIcon,
  default1557 as FileChartPieIcon,
  default494 as FileCheck2Icon,
  default495 as FileCheckIcon,
  default496 as FileClockIcon,
  default497 as FileCode2Icon,
  default498 as FileCodeIcon,
  default1560 as FileCog2Icon,
  default1559 as FileCogIcon,
  default499 as FileDiffIcon,
  default500 as FileDigitIcon,
  default501 as FileDownIcon,
  default1562 as FileEditIcon,
  default502 as FileHeartIcon,
  default538 as FileIcon,
  default503 as FileImageIcon,
  default504 as FileInputIcon,
  default505 as FileJson2Icon,
  default506 as FileJsonIcon,
  default507 as FileKey2Icon,
  default508 as FileKeyIcon,
  default1554 as FileLineChartIcon,
  default509 as FileLock2Icon,
  default510 as FileLockIcon,
  default511 as FileMinus2Icon,
  default512 as FileMinusIcon,
  default513 as FileMusicIcon,
  default514 as FileOutputIcon,
  default1561 as FilePenIcon,
  default1563 as FilePenLineIcon,
  default1558 as FilePieChartIcon,
  default515 as FilePlus2Icon,
  default516 as FilePlusIcon,
  default517 as FileQuestionIcon,
  default518 as FileScanIcon,
  default519 as FileSearch2Icon,
  default520 as FileSearchIcon,
  default1564 as FileSignatureIcon,
  default522 as FileSlidersIcon,
  default521 as FileSpreadsheetIcon,
  default523 as FileStackIcon,
  default524 as FileSymlinkIcon,
  default526 as FileTerminalIcon,
  default525 as FileTextIcon,
  default527 as FileType2Icon,
  default528 as FileTypeIcon,
  default529 as FileUpIcon,
  default530 as FileUserIcon,
  default531 as FileVideo2Icon,
  default532 as FileVideoIcon,
  default533 as FileVolume2Icon,
  default534 as FileVolumeIcon,
  default535 as FileWarningIcon,
  default536 as FileX2Icon,
  default537 as FileXIcon,
  default539 as FilesIcon,
  default540 as FilmIcon,
  default1572 as FilterIcon,
  default1570 as FilterXIcon,
  default541 as FingerprintIcon,
  default542 as FireExtinguisherIcon,
  default545 as FishIcon,
  default543 as FishOffIcon,
  default544 as FishSymbolIcon,
  default549 as FlagIcon,
  default546 as FlagOffIcon,
  default547 as FlagTriangleLeftIcon,
  default548 as FlagTriangleRightIcon,
  default551 as FlameIcon,
  default550 as FlameKindlingIcon,
  default553 as FlashlightIcon,
  default552 as FlashlightOffIcon,
  default555 as FlaskConicalIcon,
  default554 as FlaskConicalOffIcon,
  default556 as FlaskRoundIcon,
  default558 as FlipHorizontal2Icon,
  default557 as FlipHorizontalIcon,
  default559 as FlipVertical2Icon,
  default560 as FlipVerticalIcon,
  default561 as Flower2Icon,
  default562 as FlowerIcon,
  default565 as FocusIcon,
  default563 as FoldHorizontalIcon,
  default564 as FoldVerticalIcon,
  default566 as FolderArchiveIcon,
  default567 as FolderCheckIcon,
  default568 as FolderClockIcon,
  default569 as FolderClosedIcon,
  default570 as FolderCodeIcon,
  default1566 as FolderCog2Icon,
  default1565 as FolderCogIcon,
  default571 as FolderDotIcon,
  default572 as FolderDownIcon,
  default1568 as FolderEditIcon,
  default573 as FolderGit2Icon,
  default574 as FolderGitIcon,
  default575 as FolderHeartIcon,
  default593 as FolderIcon,
  default576 as FolderInputIcon,
  default577 as FolderKanbanIcon,
  default578 as FolderKeyIcon,
  default579 as FolderLockIcon,
  default580 as FolderMinusIcon,
  default581 as FolderOpenDotIcon,
  default582 as FolderOpenIcon,
  default583 as FolderOutputIcon,
  default1567 as FolderPenIcon,
  default584 as FolderPlusIcon,
  default585 as FolderRootIcon,
  default586 as FolderSearch2Icon,
  default587 as FolderSearchIcon,
  default588 as FolderSymlinkIcon,
  default589 as FolderSyncIcon,
  default590 as FolderTreeIcon,
  default592 as FolderUpIcon,
  default591 as FolderXIcon,
  default594 as FoldersIcon,
  default596 as FootprintsIcon,
  default1800 as ForkKnifeCrossedIcon,
  default1802 as ForkKnifeIcon,
  default595 as ForkliftIcon,
  default1643 as FormInputIcon,
  default597 as ForwardIcon,
  default599 as FrameIcon,
  default598 as FramerIcon,
  default601 as FrownIcon,
  default600 as FuelIcon,
  default602 as FullscreenIcon,
  default1718 as FunctionSquareIcon,
  default1571 as FunnelIcon,
  default603 as FunnelPlusIcon,
  default1569 as FunnelXIcon,
  default604 as GalleryHorizontalEndIcon,
  default605 as GalleryHorizontalIcon,
  default607 as GalleryThumbnailsIcon,
  default606 as GalleryVerticalEndIcon,
  default608 as GalleryVerticalIcon,
  default609 as Gamepad2Icon,
  default610 as GamepadIcon,
  default1455 as GanttChartIcon,
  default1691 as GanttChartSquareIcon,
  default1493 as GaugeCircleIcon,
  default611 as GaugeIcon,
  default612 as GavelIcon,
  default613 as GemIcon,
  default614 as GhostIcon,
  default615 as GiftIcon,
  default616 as GitBranchIcon,
  default617 as GitBranchPlusIcon,
  default1573 as GitCommitHorizontalIcon,
  default1574 as GitCommitIcon,
  default618 as GitCommitVerticalIcon,
  default619 as GitCompareArrowsIcon,
  default620 as GitCompareIcon,
  default621 as GitForkIcon,
  default622 as GitGraphIcon,
  default623 as GitMergeIcon,
  default624 as GitPullRequestArrowIcon,
  default625 as GitPullRequestClosedIcon,
  default627 as GitPullRequestCreateArrowIcon,
  default626 as GitPullRequestCreateIcon,
  default628 as GitPullRequestDraftIcon,
  default629 as GitPullRequestIcon,
  default631 as GithubIcon,
  default630 as GitlabIcon,
  default632 as GlassWaterIcon,
  default633 as GlassesIcon,
  default1544 as Globe2Icon,
  default635 as GlobeIcon,
  default634 as GlobeLockIcon,
  default636 as GoalIcon,
  default637 as GpuIcon,
  default638 as GrabIcon,
  default639 as GraduationCapIcon,
  default640 as GrapeIcon,
  default1576 as Grid2X2CheckIcon,
  default1582 as Grid2X2Icon,
  default1578 as Grid2X2PlusIcon,
  default1580 as Grid2X2XIcon,
  default1575 as Grid2x2CheckIcon,
  default1581 as Grid2x2Icon,
  default1577 as Grid2x2PlusIcon,
  default1579 as Grid2x2XIcon,
  default1589 as Grid3X3Icon,
  default641 as Grid3x2Icon,
  default1587 as Grid3x3Icon,
  default1588 as GridIcon,
  default642 as GripHorizontalIcon,
  default644 as GripIcon,
  default643 as GripVerticalIcon,
  default645 as GroupIcon,
  default647 as GuitarIcon,
  default646 as HamIcon,
  default648 as HamburgerIcon,
  default649 as HammerIcon,
  default650 as HandCoinsIcon,
  default652 as HandHeartIcon,
  default1583 as HandHelpingIcon,
  default654 as HandIcon,
  default651 as HandMetalIcon,
  default653 as HandPlatterIcon,
  default656 as HandshakeIcon,
  default655 as HardDriveDownloadIcon,
  default658 as HardDriveIcon,
  default657 as HardDriveUploadIcon,
  default659 as HardHatIcon,
  default660 as HashIcon,
  default662 as HazeIcon,
  default661 as HdmiPortIcon,
  default663 as Heading1Icon,
  default664 as Heading2Icon,
  default665 as Heading3Icon,
  default667 as Heading4Icon,
  default666 as Heading5Icon,
  default668 as Heading6Icon,
  default669 as HeadingIcon,
  default671 as HeadphoneOffIcon,
  default670 as HeadphonesIcon,
  default672 as HeadsetIcon,
  default673 as HeartCrackIcon,
  default674 as HeartHandshakeIcon,
  default678 as HeartIcon,
  default675 as HeartMinusIcon,
  default676 as HeartOffIcon,
  default679 as HeartPlusIcon,
  default677 as HeartPulseIcon,
  default680 as HeaterIcon,
  default1495 as HelpCircleIcon,
  default1584 as HelpingHandIcon,
  default681 as HexagonIcon,
  default682 as HighlighterIcon,
  default683 as HistoryIcon,
  default1591 as HomeIcon,
  default685 as HopIcon,
  default684 as HopOffIcon,
  default686 as HospitalIcon,
  default687 as HotelIcon,
  default688 as HourglassIcon,
  default1590 as HouseIcon,
  default689 as HousePlugIcon,
  default690 as HousePlusIcon,
  default691 as HouseWifiIcon,
  default1586 as IceCream2Icon,
  default1585 as IceCreamBowlIcon,
  default1592 as IceCreamConeIcon,
  default1593 as IceCreamIcon,
  default692 as IdCardIcon,
  default693 as IdCardLanyardIcon,
  default694 as ImageDownIcon,
  default700 as ImageIcon,
  default697 as ImageMinusIcon,
  default695 as ImageOffIcon,
  default696 as ImagePlayIcon,
  default698 as ImagePlusIcon,
  default699 as ImageUpIcon,
  default701 as ImageUpscaleIcon,
  default702 as ImagesIcon,
  default704 as ImportIcon,
  default703 as InboxIcon,
  default1594 as IndentDecreaseIcon,
  default1597 as IndentIcon,
  default1596 as IndentIncreaseIcon,
  default705 as IndianRupeeIcon,
  default706 as InfinityIcon,
  default707 as InfoIcon,
  default1728 as InspectIcon,
  default709 as InspectionPanelIcon,
  default708 as InstagramIcon,
  default710 as ItalicIcon,
  default711 as IterationCcwIcon,
  default712 as IterationCwIcon,
  default713 as JapaneseYenIcon,
  default714 as JoystickIcon,
  default715 as KanbanIcon,
  default1708 as KanbanSquareDashedIcon,
  default1722 as KanbanSquareIcon,
  default718 as KeyIcon,
  default716 as KeyRoundIcon,
  default717 as KeySquareIcon,
  default721 as KeyboardIcon,
  default719 as KeyboardMusicIcon,
  default720 as KeyboardOffIcon,
  default722 as LampCeilingIcon,
  default723 as LampDeskIcon,
  default724 as LampFloorIcon,
  default727 as LampIcon,
  default725 as LampWallDownIcon,
  default726 as LampWallUpIcon,
  default728 as LandPlotIcon,
  default729 as LandmarkIcon,
  default730 as LanguagesIcon,
  default1599 as Laptop2Icon,
  default732 as LaptopIcon,
  default731 as LaptopMinimalCheckIcon,
  default1598 as LaptopMinimalIcon,
  default734 as LassoIcon,
  default733 as LassoSelectIcon,
  default735 as LaughIcon,
  default736 as Layers2Icon,
  default1601 as Layers3Icon,
  default1600 as LayersIcon,
  default738 as LayoutDashboardIcon,
  default737 as LayoutGridIcon,
  default1635 as LayoutIcon,
  default739 as LayoutListIcon,
  default741 as LayoutPanelLeftIcon,
  default740 as LayoutPanelTopIcon,
  default743 as LayoutTemplateIcon,
  default742 as LeafIcon,
  default745 as LeafyGreenIcon,
  default744 as LecternIcon,
  default747 as LetterTextIcon,
  default746 as LibraryBigIcon,
  default748 as LibraryIcon,
  default1726 as LibrarySquareIcon,
  default749 as LifeBuoyIcon,
  default751 as LigatureIcon,
  default752 as LightbulbIcon,
  default750 as LightbulbOffIcon,
  default1449 as LineChartIcon,
  default754 as Link2Icon,
  default753 as Link2OffIcon,
  default755 as LinkIcon,
  default756 as LinkedinIcon,
  default757 as ListCheckIcon,
  default758 as ListChecksIcon,
  default760 as ListCollapseIcon,
  default759 as ListEndIcon,
  default762 as ListFilterIcon,
  default761 as ListFilterPlusIcon,
  default773 as ListIcon,
  default763 as ListMinusIcon,
  default764 as ListMusicIcon,
  default765 as ListOrderedIcon,
  default766 as ListPlusIcon,
  default767 as ListRestartIcon,
  default768 as ListStartIcon,
  default769 as ListTodoIcon,
  default771 as ListTreeIcon,
  default770 as ListVideoIcon,
  default772 as ListXIcon,
  default1603 as Loader2Icon,
  default1602 as LoaderCircleIcon,
  default775 as LoaderIcon,
  default774 as LoaderPinwheelIcon,
  default776 as LocateFixedIcon,
  default778 as LocateIcon,
  default777 as LocateOffIcon,
  default779 as LocationEditIcon,
  default781 as LockIcon,
  default780 as LockKeyholeIcon,
  default1604 as LockKeyholeOpenIcon,
  default1606 as LockOpenIcon,
  default782 as LogInIcon,
  default783 as LogOutIcon,
  default784 as LogsIcon,
  default786 as LollipopIcon,
  default785 as LuggageIcon,
  default1724 as MSquareIcon,
  default787 as MagnetIcon,
  default789 as MailCheckIcon,
  default795 as MailIcon,
  default788 as MailMinusIcon,
  default791 as MailOpenIcon,
  default790 as MailPlusIcon,
  default792 as MailQuestionIcon,
  default793 as MailSearchIcon,
  default794 as MailWarningIcon,
  default796 as MailXIcon,
  default797 as MailboxIcon,
  default798 as MailsIcon,
  default812 as MapIcon,
  default800 as MapPinCheckIcon,
  default799 as MapPinCheckInsideIcon,
  default801 as MapPinHouseIcon,
  default807 as MapPinIcon,
  default802 as MapPinMinusIcon,
  default804 as MapPinMinusInsideIcon,
  default803 as MapPinOffIcon,
  default806 as MapPinPlusIcon,
  default805 as MapPinPlusInsideIcon,
  default808 as MapPinXIcon,
  default809 as MapPinXInsideIcon,
  default810 as MapPinnedIcon,
  default811 as MapPlusIcon,
  default813 as MarsIcon,
  default815 as MarsStrokeIcon,
  default814 as MartiniIcon,
  default816 as Maximize2Icon,
  default817 as MaximizeIcon,
  default818 as MedalIcon,
  default820 as MegaphoneIcon,
  default819 as MegaphoneOffIcon,
  default821 as MehIcon,
  default823 as MemoryStickIcon,
  default822 as MenuIcon,
  default1734 as MenuSquareIcon,
  default824 as MergeIcon,
  default826 as MessageCircleCodeIcon,
  default828 as MessageCircleDashedIcon,
  default825 as MessageCircleHeartIcon,
  default836 as MessageCircleIcon,
  default827 as MessageCircleMoreIcon,
  default829 as MessageCircleOffIcon,
  default830 as MessageCirclePlusIcon,
  default831 as MessageCircleQuestionIcon,
  default832 as MessageCircleReplyIcon,
  default834 as MessageCircleWarningIcon,
  default833 as MessageCircleXIcon,
  default835 as MessageSquareCodeIcon,
  default837 as MessageSquareDashedIcon,
  default838 as MessageSquareDiffIcon,
  default839 as MessageSquareDotIcon,
  default841 as MessageSquareHeartIcon,
  default851 as MessageSquareIcon,
  default840 as MessageSquareLockIcon,
  default843 as MessageSquareMoreIcon,
  default842 as MessageSquareOffIcon,
  default845 as MessageSquarePlusIcon,
  default844 as MessageSquareQuoteIcon,
  default846 as MessageSquareReplyIcon,
  default847 as MessageSquareShareIcon,
  default848 as MessageSquareTextIcon,
  default849 as MessageSquareWarningIcon,
  default850 as MessageSquareXIcon,
  default852 as MessagesSquareIcon,
  default1609 as Mic2Icon,
  default854 as MicIcon,
  default853 as MicOffIcon,
  default1608 as MicVocalIcon,
  default855 as MicrochipIcon,
  default856 as MicroscopeIcon,
  default857 as MicrowaveIcon,
  default858 as MilestoneIcon,
  default859 as MilkIcon,
  default860 as MilkOffIcon,
  default861 as Minimize2Icon,
  default862 as MinimizeIcon,
  default1497 as MinusCircleIcon,
  default865 as MinusIcon,
  default1730 as MinusSquareIcon,
  default863 as MonitorCheckIcon,
  default866 as MonitorCogIcon,
  default864 as MonitorDotIcon,
  default867 as MonitorDownIcon,
  default876 as MonitorIcon,
  default868 as MonitorOffIcon,
  default869 as MonitorPauseIcon,
  default870 as MonitorPlayIcon,
  default871 as MonitorSmartphoneIcon,
  default872 as MonitorSpeakerIcon,
  default873 as MonitorStopIcon,
  default874 as MonitorUpIcon,
  default875 as MonitorXIcon,
  default878 as MoonIcon,
  default877 as MoonStarIcon,
  default1548 as MoreHorizontalIcon,
  default1546 as MoreVerticalIcon,
  default880 as MountainIcon,
  default879 as MountainSnowIcon,
  default886 as MouseIcon,
  default881 as MouseOffIcon,
  default882 as MousePointer2Icon,
  default883 as MousePointerBanIcon,
  default884 as MousePointerClickIcon,
  default885 as MousePointerIcon,
  default1710 as MousePointerSquareDashedIcon,
  default1611 as Move3DIcon,
  default1610 as Move3dIcon,
  default887 as MoveDiagonal2Icon,
  default889 as MoveDiagonalIcon,
  default890 as MoveDownIcon,
  default888 as MoveDownLeftIcon,
  default891 as MoveDownRightIcon,
  default893 as MoveHorizontalIcon,
  default898 as MoveIcon,
  default892 as MoveLeftIcon,
  default895 as MoveRightIcon,
  default896 as MoveUpIcon,
  default894 as MoveUpLeftIcon,
  default897 as MoveUpRightIcon,
  default901 as MoveVerticalIcon,
  default899 as Music2Icon,
  default900 as Music3Icon,
  default902 as Music4Icon,
  default904 as MusicIcon,
  default905 as Navigation2Icon,
  default903 as Navigation2OffIcon,
  default906 as NavigationIcon,
  default907 as NavigationOffIcon,
  default909 as NetworkIcon,
  default908 as NewspaperIcon,
  default910 as NfcIcon,
  default911 as NonBinaryIcon,
  default915 as NotebookIcon,
  default912 as NotebookPenIcon,
  default913 as NotebookTabsIcon,
  default914 as NotebookTextIcon,
  default916 as NotepadTextDashedIcon,
  default917 as NotepadTextIcon,
  default920 as NutIcon,
  default918 as NutOffIcon,
  default1612 as OctagonAlertIcon,
  default921 as OctagonIcon,
  default919 as OctagonMinusIcon,
  default1614 as OctagonPauseIcon,
  default1616 as OctagonXIcon,
  default923 as OmegaIcon,
  default922 as OptionIcon,
  default925 as OrbitIcon,
  default924 as OrigamiIcon,
  default1595 as OutdentIcon,
  default926 as Package2Icon,
  default927 as PackageCheckIcon,
  default932 as PackageIcon,
  default928 as PackageMinusIcon,
  default929 as PackageOpenIcon,
  default930 as PackagePlusIcon,
  default933 as PackageSearchIcon,
  default931 as PackageXIcon,
  default934 as PaintBucketIcon,
  default936 as PaintRollerIcon,
  default1619 as Paintbrush2Icon,
  default935 as PaintbrushIcon,
  default1618 as PaintbrushVerticalIcon,
  default937 as PaletteIcon,
  default1778 as PalmtreeIcon,
  default938 as PandaIcon,
  default939 as PanelBottomCloseIcon,
  default1620 as PanelBottomDashedIcon,
  default941 as PanelBottomIcon,
  default1621 as PanelBottomInactiveIcon,
  default940 as PanelBottomOpenIcon,
  default1624 as PanelLeftCloseIcon,
  default1622 as PanelLeftDashedIcon,
  default1628 as PanelLeftIcon,
  default1623 as PanelLeftInactiveIcon,
  default1626 as PanelLeftOpenIcon,
  default942 as PanelRightCloseIcon,
  default1630 as PanelRightDashedIcon,
  default944 as PanelRightIcon,
  default1631 as PanelRightInactiveIcon,
  default943 as PanelRightOpenIcon,
  default946 as PanelTopCloseIcon,
  default1632 as PanelTopDashedIcon,
  default945 as PanelTopIcon,
  default1633 as PanelTopInactiveIcon,
  default947 as PanelTopOpenIcon,
  default948 as PanelsLeftBottomIcon,
  default1538 as PanelsLeftRightIcon,
  default949 as PanelsRightBottomIcon,
  default1649 as PanelsTopBottomIcon,
  default1634 as PanelsTopLeftIcon,
  default950 as PaperclipIcon,
  default951 as ParenthesesIcon,
  default1501 as ParkingCircleIcon,
  default1499 as ParkingCircleOffIcon,
  default952 as ParkingMeterIcon,
  default1736 as ParkingSquareIcon,
  default1732 as ParkingSquareOffIcon,
  default953 as PartyPopperIcon,
  default1505 as PauseCircleIcon,
  default954 as PauseIcon,
  default1615 as PauseOctagonIcon,
  default957 as PawPrintIcon,
  default955 as PcCaseIcon,
  default1738 as PenBoxIcon,
  default1638 as PenIcon,
  default1636 as PenLineIcon,
  default956 as PenOffIcon,
  default1739 as PenSquareIcon,
  default958 as PenToolIcon,
  default962 as PencilIcon,
  default959 as PencilLineIcon,
  default960 as PencilOffIcon,
  default961 as PencilRulerIcon,
  default963 as PentagonIcon,
  default1503 as PercentCircleIcon,
  default1542 as PercentDiamondIcon,
  default965 as PercentIcon,
  default1742 as PercentSquareIcon,
  default964 as PersonStandingIcon,
  default967 as PhilippinePesoIcon,
  default966 as PhoneCallIcon,
  default968 as PhoneForwardedIcon,
  default971 as PhoneIcon,
  default969 as PhoneIncomingIcon,
  default972 as PhoneMissedIcon,
  default970 as PhoneOffIcon,
  default975 as PhoneOutgoingIcon,
  default973 as PiIcon,
  default1744 as PiSquareIcon,
  default974 as PianoIcon,
  default976 as PickaxeIcon,
  default977 as PictureInPicture2Icon,
  default980 as PictureInPictureIcon,
  default1457 as PieChartIcon,
  default978 as PiggyBankIcon,
  default981 as PilcrowIcon,
  default979 as PilcrowLeftIcon,
  default982 as PilcrowRightIcon,
  default1748 as PilcrowSquareIcon,
  default983 as PillBottleIcon,
  default984 as PillIcon,
  default986 as PinIcon,
  default985 as PinOffIcon,
  default987 as PipetteIcon,
  default988 as PizzaIcon,
  default990 as PlaneIcon,
  default989 as PlaneLandingIcon,
  default991 as PlaneTakeoffIcon,
  default1509 as PlayCircleIcon,
  default993 as PlayIcon,
  default1746 as PlaySquareIcon,
  default995 as Plug2Icon,
  default992 as PlugIcon,
  default1641 as PlugZap2Icon,
  default1640 as PlugZapIcon,
  default1507 as PlusCircleIcon,
  default994 as PlusIcon,
  default1752 as PlusSquareIcon,
  default996 as PocketIcon,
  default997 as PocketKnifeIcon,
  default998 as PodcastIcon,
  default1000 as PointerIcon,
  default999 as PointerOffIcon,
  default1001 as PopcornIcon,
  default1003 as PopsicleIcon,
  default1002 as PoundSterlingIcon,
  default1511 as PowerCircleIcon,
  default1005 as PowerIcon,
  default1004 as PowerOffIcon,
  default1750 as PowerSquareIcon,
  default1006 as PresentationIcon,
  default1009 as PrinterCheckIcon,
  default1007 as PrinterIcon,
  default1008 as ProjectorIcon,
  default1011 as ProportionsIcon,
  default1010 as PuzzleIcon,
  default1012 as PyramidIcon,
  default1013 as QrCodeIcon,
  default1014 as QuoteIcon,
  default1015 as RabbitIcon,
  default1017 as RadarIcon,
  default1016 as RadiationIcon,
  default1019 as RadicalIcon,
  default1021 as RadioIcon,
  default1020 as RadioReceiverIcon,
  default1018 as RadioTowerIcon,
  default1022 as RadiusIcon,
  default1023 as RailSymbolIcon,
  default1024 as RainbowIcon,
  default1025 as RatIcon,
  default1026 as RatioIcon,
  default1027 as ReceiptCentIcon,
  default1029 as ReceiptEuroIcon,
  default1034 as ReceiptIcon,
  default1028 as ReceiptIndianRupeeIcon,
  default1030 as ReceiptJapaneseYenIcon,
  default1031 as ReceiptPoundSterlingIcon,
  default1033 as ReceiptRussianRubleIcon,
  default1032 as ReceiptSwissFrancIcon,
  default1036 as ReceiptTextIcon,
  default1642 as RectangleEllipsisIcon,
  default1035 as RectangleGogglesIcon,
  default1037 as RectangleHorizontalIcon,
  default1038 as RectangleVerticalIcon,
  default1039 as RecycleIcon,
  default1042 as Redo2Icon,
  default1040 as RedoDotIcon,
  default1043 as RedoIcon,
  default1041 as RefreshCcwDotIcon,
  default1044 as RefreshCcwIcon,
  default1046 as RefreshCwIcon,
  default1045 as RefreshCwOffIcon,
  default1047 as RefrigeratorIcon,
  default1048 as RegexIcon,
  default1051 as RemoveFormattingIcon,
  default1049 as Repeat1Icon,
  default1050 as Repeat2Icon,
  default1052 as RepeatIcon,
  default1053 as ReplaceAllIcon,
  default1054 as ReplaceIcon,
  default1055 as ReplyAllIcon,
  default1056 as ReplyIcon,
  default1057 as RewindIcon,
  default1058 as RibbonIcon,
  default1059 as RocketIcon,
  default1060 as RockingChairIcon,
  default1062 as RollerCoasterIcon,
  default1645 as Rotate3DIcon,
  default1644 as Rotate3dIcon,
  default1064 as RotateCcwIcon,
  default1061 as RotateCcwKeyIcon,
  default1063 as RotateCcwSquareIcon,
  default1066 as RotateCwIcon,
  default1065 as RotateCwSquareIcon,
  default1068 as RouteIcon,
  default1067 as RouteOffIcon,
  default1069 as RouterIcon,
  default1646 as Rows2Icon,
  default1648 as Rows3Icon,
  default1070 as Rows4Icon,
  default1647 as RowsIcon,
  default1071 as RssIcon,
  default1072 as RulerDimensionLineIcon,
  default1073 as RulerIcon,
  default1074 as RussianRubleIcon,
  default1075 as SailboatIcon,
  default1076 as SaladIcon,
  default1078 as SandwichIcon,
  default1077 as SatelliteDishIcon,
  default1079 as SatelliteIcon,
  default1080 as SaudiRiyalIcon,
  default1082 as SaveAllIcon,
  default1083 as SaveIcon,
  default1081 as SaveOffIcon,
  default1651 as Scale3DIcon,
  default1650 as Scale3dIcon,
  default1084 as ScaleIcon,
  default1085 as ScalingIcon,
  default1086 as ScanBarcodeIcon,
  default1087 as ScanEyeIcon,
  default1088 as ScanFaceIcon,
  default1089 as ScanHeartIcon,
  default1094 as ScanIcon,
  default1091 as ScanLineIcon,
  default1090 as ScanQrCodeIcon,
  default1092 as ScanSearchIcon,
  default1093 as ScanTextIcon,
  default1459 as ScatterChartIcon,
  default1784 as School2Icon,
  default1095 as SchoolIcon,
  default1097 as ScissorsIcon,
  default1096 as ScissorsLineDashedIcon,
  default1689 as ScissorsSquareDashedBottomIcon,
  default1754 as ScissorsSquareIcon,
  default1099 as ScreenShareIcon,
  default1098 as ScreenShareOffIcon,
  default1101 as ScrollIcon,
  default1100 as ScrollTextIcon,
  default1103 as SearchCheckIcon,
  default1102 as SearchCodeIcon,
  default1106 as SearchIcon,
  default1104 as SearchSlashIcon,
  default1105 as SearchXIcon,
  default1107 as SectionIcon,
  default1653 as SendHorizonalIcon,
  default1652 as SendHorizontalIcon,
  default1109 as SendIcon,
  default1108 as SendToBackIcon,
  default1111 as SeparatorHorizontalIcon,
  default1110 as SeparatorVerticalIcon,
  default1112 as ServerCogIcon,
  default1113 as ServerCrashIcon,
  default1116 as ServerIcon,
  default1114 as ServerOffIcon,
  default1115 as Settings2Icon,
  default1117 as SettingsIcon,
  default1118 as ShapesIcon,
  default1119 as Share2Icon,
  default1120 as ShareIcon,
  default1121 as SheetIcon,
  default1122 as ShellIcon,
  default1123 as ShieldAlertIcon,
  default1124 as ShieldBanIcon,
  default1125 as ShieldCheckIcon,
  default1655 as ShieldCloseIcon,
  default1126 as ShieldEllipsisIcon,
  default1127 as ShieldHalfIcon,
  default1133 as ShieldIcon,
  default1129 as ShieldMinusIcon,
  default1128 as ShieldOffIcon,
  default1131 as ShieldPlusIcon,
  default1130 as ShieldQuestionIcon,
  default1132 as ShieldUserIcon,
  default1654 as ShieldXIcon,
  default1136 as ShipIcon,
  default1134 as ShipWheelIcon,
  default1135 as ShirtIcon,
  default1137 as ShoppingBagIcon,
  default1138 as ShoppingBasketIcon,
  default1139 as ShoppingCartIcon,
  default1140 as ShovelIcon,
  default1141 as ShowerHeadIcon,
  default1142 as ShredderIcon,
  default1143 as ShrimpIcon,
  default1144 as ShrinkIcon,
  default1146 as ShrubIcon,
  default1145 as ShuffleIcon,
  default1625 as SidebarCloseIcon,
  default1629 as SidebarIcon,
  default1627 as SidebarOpenIcon,
  default1147 as SigmaIcon,
  default1758 as SigmaSquareIcon,
  default1148 as SignalHighIcon,
  default1152 as SignalIcon,
  default1150 as SignalLowIcon,
  default1149 as SignalMediumIcon,
  default1151 as SignalZeroIcon,
  default1153 as SignatureIcon,
  default1154 as SignpostBigIcon,
  default1155 as SignpostIcon,
  default1156 as SirenIcon,
  default1158 as SkipBackIcon,
  default1157 as SkipForwardIcon,
  default1159 as SkullIcon,
  default1160 as SlackIcon,
  default1161 as SlashIcon,
  default1756 as SlashSquareIcon,
  default1162 as SliceIcon,
  default1165 as SlidersHorizontalIcon,
  default1657 as SlidersIcon,
  default1656 as SlidersVerticalIcon,
  default1163 as SmartphoneChargingIcon,
  default1166 as SmartphoneIcon,
  default1164 as SmartphoneNfcIcon,
  default1168 as SmileIcon,
  default1167 as SmilePlusIcon,
  default1169 as SnailIcon,
  default1170 as SnowflakeIcon,
  default1172 as SoapDispenserDropletIcon,
  default1171 as SofaIcon,
  default1417 as SortAscIcon,
  default1411 as SortDescIcon,
  default1173 as SoupIcon,
  default1174 as SpaceIcon,
  default1175 as SpadeIcon,
  default1176 as SparkleIcon,
  default1658 as SparklesIcon,
  default1177 as SpeakerIcon,
  default1178 as SpeechIcon,
  default1179 as SpellCheck2Icon,
  default1181 as SpellCheckIcon,
  default1182 as SplineIcon,
  default1180 as SplinePointerIcon,
  default1183 as SplitIcon,
  default1760 as SplitSquareHorizontalIcon,
  default1762 as SplitSquareVerticalIcon,
  default1184 as SprayCanIcon,
  default1185 as SproutIcon,
  default1660 as SquareActivityIcon,
  default1668 as SquareArrowDownIcon,
  default1662 as SquareArrowDownLeftIcon,
  default1664 as SquareArrowDownRightIcon,
  default1666 as SquareArrowLeftIcon,
  default1670 as SquareArrowOutDownLeftIcon,
  default1672 as SquareArrowOutDownRightIcon,
  default1674 as SquareArrowOutUpLeftIcon,
  default1676 as SquareArrowOutUpRightIcon,
  default1678 as SquareArrowRightIcon,
  default1684 as SquareArrowUpIcon,
  default1682 as SquareArrowUpLeftIcon,
  default1680 as SquareArrowUpRightIcon,
  default1686 as SquareAsteriskIcon,
  default1688 as SquareBottomDashedScissorsIcon,
  default1690 as SquareChartGanttIcon,
  default1699 as SquareCheckBigIcon,
  default1695 as SquareCheckIcon,
  default1693 as SquareChevronDownIcon,
  default1697 as SquareChevronLeftIcon,
  default1701 as SquareChevronRightIcon,
  default1703 as SquareChevronUpIcon,
  default1705 as SquareCodeIcon,
  default1186 as SquareDashedBottomCodeIcon,
  default1187 as SquareDashedBottomIcon,
  default1711 as SquareDashedIcon,
  default1707 as SquareDashedKanbanIcon,
  default1709 as SquareDashedMousePointerIcon,
  default1188 as SquareDashedTopSolidIcon,
  default1713 as SquareDivideIcon,
  default1715 as SquareDotIcon,
  default1719 as SquareEqualIcon,
  default1717 as SquareFunctionIcon,
  default1692 as SquareGanttChartIcon,
  default1194 as SquareIcon,
  default1721 as SquareKanbanIcon,
  default1725 as SquareLibraryIcon,
  default1723 as SquareMIcon,
  default1733 as SquareMenuIcon,
  default1729 as SquareMinusIcon,
  default1727 as SquareMousePointerIcon,
  default1735 as SquareParkingIcon,
  default1731 as SquareParkingOffIcon,
  default1737 as SquarePenIcon,
  default1741 as SquarePercentIcon,
  default1743 as SquarePiIcon,
  default1747 as SquarePilcrowIcon,
  default1745 as SquarePlayIcon,
  default1751 as SquarePlusIcon,
  default1749 as SquarePowerIcon,
  default1189 as SquareRadicalIcon,
  default1190 as SquareRoundCornerIcon,
  default1753 as SquareScissorsIcon,
  default1757 as SquareSigmaIcon,
  default1755 as SquareSlashIcon,
  default1759 as SquareSplitHorizontalIcon,
  default1761 as SquareSplitVerticalIcon,
  default1191 as SquareSquareIcon,
  default1192 as SquareStackIcon,
  default1765 as SquareTerminalIcon,
  default1767 as SquareUserIcon,
  default1763 as SquareUserRoundIcon,
  default1769 as SquareXIcon,
  default1193 as SquaresExcludeIcon,
  default1196 as SquaresIntersectIcon,
  default1195 as SquaresSubtractIcon,
  default1197 as SquaresUniteIcon,
  default1201 as SquircleIcon,
  default1199 as SquirrelIcon,
  default1198 as StampIcon,
  default1200 as StarHalfIcon,
  default1203 as StarIcon,
  default1202 as StarOffIcon,
  default1659 as StarsIcon,
  default1205 as StepBackIcon,
  default1204 as StepForwardIcon,
  default1207 as StethoscopeIcon,
  default1206 as StickerIcon,
  default1208 as StickyNoteIcon,
  default1517 as StopCircleIcon,
  default1210 as StoreIcon,
  default1211 as StretchHorizontalIcon,
  default1209 as StretchVerticalIcon,
  default1212 as StrikethroughIcon,
  default1214 as SubscriptIcon,
  default1433 as SubtitlesIcon,
  default1216 as SunDimIcon,
  default1220 as SunIcon,
  default1213 as SunMediumIcon,
  default1215 as SunMoonIcon,
  default1217 as SunSnowIcon,
  default1218 as SunriseIcon,
  default1219 as SunsetIcon,
  default1221 as SuperscriptIcon,
  default1222 as SwatchBookIcon,
  default1224 as SwissFrancIcon,
  default1223 as SwitchCameraIcon,
  default1226 as SwordIcon,
  default1225 as SwordsIcon,
  default1227 as SyringeIcon,
  default1228 as Table2Icon,
  default1230 as TableCellsMergeIcon,
  default1229 as TableCellsSplitIcon,
  default1232 as TableColumnsSplitIcon,
  default1536 as TableConfigIcon,
  default1236 as TableIcon,
  default1233 as TableOfContentsIcon,
  default1231 as TablePropertiesIcon,
  default1234 as TableRowsSplitIcon,
  default1238 as TabletIcon,
  default1235 as TabletSmartphoneIcon,
  default1237 as TabletsIcon,
  default1240 as TagIcon,
  default1239 as TagsIcon,
  default1243 as Tally1Icon,
  default1241 as Tally2Icon,
  default1242 as Tally3Icon,
  default1244 as Tally4Icon,
  default1245 as Tally5Icon,
  default1246 as TangentIcon,
  default1247 as TargetIcon,
  default1248 as TelescopeIcon,
  default1249 as TentIcon,
  default1251 as TentTreeIcon,
  default1250 as TerminalIcon,
  default1766 as TerminalSquareIcon,
  default1772 as TestTube2Icon,
  default1771 as TestTubeDiagonalIcon,
  default1252 as TestTubeIcon,
  default1253 as TestTubesIcon,
  default1254 as TextCursorIcon,
  default1256 as TextCursorInputIcon,
  default1258 as TextIcon,
  default1257 as TextQuoteIcon,
  default1255 as TextSearchIcon,
  default1773 as TextSelectIcon,
  default1774 as TextSelectionIcon,
  default1259 as TheaterIcon,
  default1263 as ThermometerIcon,
  default1260 as ThermometerSnowflakeIcon,
  default1261 as ThermometerSunIcon,
  default1262 as ThumbsDownIcon,
  default1264 as ThumbsUpIcon,
  default1265 as TicketCheckIcon,
  default1271 as TicketIcon,
  default1266 as TicketMinusIcon,
  default1267 as TicketPercentIcon,
  default1268 as TicketPlusIcon,
  default1269 as TicketSlashIcon,
  default1270 as TicketXIcon,
  default1273 as TicketsIcon,
  default1272 as TicketsPlaneIcon,
  default1276 as TimerIcon,
  default1274 as TimerOffIcon,
  default1275 as TimerResetIcon,
  default1277 as ToggleLeftIcon,
  default1278 as ToggleRightIcon,
  default1279 as ToiletIcon,
  default1280 as TornadoIcon,
  default1281 as TorusIcon,
  default1284 as TouchpadIcon,
  default1282 as TouchpadOffIcon,
  default1283 as TowerControlIcon,
  default1285 as ToyBrickIcon,
  default1286 as TractorIcon,
  default1287 as TrafficConeIcon,
  default1289 as TrainFrontIcon,
  default1288 as TrainFrontTunnelIcon,
  default1776 as TrainIcon,
  default1290 as TrainTrackIcon,
  default1775 as TramFrontIcon,
  default1291 as TransgenderIcon,
  default1292 as Trash2Icon,
  default1293 as TrashIcon,
  default1294 as TreeDeciduousIcon,
  default1777 as TreePalmIcon,
  default1295 as TreePineIcon,
  default1296 as TreesIcon,
  default1297 as TrelloIcon,
  default1298 as TrendingDownIcon,
  default1299 as TrendingUpDownIcon,
  default1300 as TrendingUpIcon,
  default1779 as TriangleAlertIcon,
  default1302 as TriangleDashedIcon,
  default1303 as TriangleIcon,
  default1301 as TriangleRightIcon,
  default1304 as TrophyIcon,
  default1306 as TruckElectricIcon,
  default1305 as TruckIcon,
  default1307 as TurtleIcon,
  default1782 as Tv2Icon,
  default1310 as TvIcon,
  default1781 as TvMinimalIcon,
  default1308 as TvMinimalPlayIcon,
  default1309 as TwitchIcon,
  default1311 as TwitterIcon,
  default1314 as TypeIcon,
  default1313 as TypeOutlineIcon,
  default1315 as UmbrellaIcon,
  default1312 as UmbrellaOffIcon,
  default1316 as UnderlineIcon,
  default1317 as Undo2Icon,
  default1318 as UndoDotIcon,
  default1319 as UndoIcon,
  default1320 as UnfoldHorizontalIcon,
  default1322 as UnfoldVerticalIcon,
  default1321 as UngroupIcon,
  default1783 as UniversityIcon,
  default1325 as Unlink2Icon,
  default1323 as UnlinkIcon,
  default1607 as UnlockIcon,
  default1605 as UnlockKeyholeIcon,
  default1324 as UnplugIcon,
  default1529 as UploadCloudIcon,
  default1326 as UploadIcon,
  default1327 as UsbIcon,
  default1796 as User2Icon,
  default1786 as UserCheck2Icon,
  default1329 as UserCheckIcon,
  default1515 as UserCircle2Icon,
  default1519 as UserCircleIcon,
  default1788 as UserCog2Icon,
  default1328 as UserCogIcon,
  default1338 as UserIcon,
  default1330 as UserLockIcon,
  default1790 as UserMinus2Icon,
  default1331 as UserMinusIcon,
  default1332 as UserPenIcon,
  default1792 as UserPlus2Icon,
  default1333 as UserPlusIcon,
  default1785 as UserRoundCheckIcon,
  default1787 as UserRoundCogIcon,
  default1795 as UserRoundIcon,
  default1789 as UserRoundMinusIcon,
  default1334 as UserRoundPenIcon,
  default1791 as UserRoundPlusIcon,
  default1335 as UserRoundSearchIcon,
  default1793 as UserRoundXIcon,
  default1336 as UserSearchIcon,
  default1764 as UserSquare2Icon,
  default1768 as UserSquareIcon,
  default1794 as UserX2Icon,
  default1337 as UserXIcon,
  default1798 as Users2Icon,
  default1339 as UsersIcon,
  default1797 as UsersRoundIcon,
  default1799 as UtensilsCrossedIcon,
  default1801 as UtensilsIcon,
  default1340 as UtilityPoleIcon,
  default1341 as VariableIcon,
  default1342 as VaultIcon,
  default1343 as VeganIcon,
  default1345 as VenetianMaskIcon,
  default1344 as VenusAndMarsIcon,
  default1346 as VenusIcon,
  default1423 as VerifiedIcon,
  default1348 as VibrateIcon,
  default1347 as VibrateOffIcon,
  default1350 as VideoIcon,
  default1349 as VideoOffIcon,
  default1351 as VideotapeIcon,
  default1352 as ViewIcon,
  default1353 as VoicemailIcon,
  default1354 as VolleyballIcon,
  default1355 as Volume1Icon,
  default1356 as Volume2Icon,
  default1359 as VolumeIcon,
  default1357 as VolumeOffIcon,
  default1358 as VolumeXIcon,
  default1360 as VoteIcon,
  default1804 as Wallet2Icon,
  default1361 as WalletCardsIcon,
  default1362 as WalletIcon,
  default1803 as WalletMinimalIcon,
  default1363 as WallpaperIcon,
  default1806 as Wand2Icon,
  default1364 as WandIcon,
  default1805 as WandSparklesIcon,
  default1365 as WarehouseIcon,
  default1366 as WashingMachineIcon,
  default1367 as WatchIcon,
  default1369 as WavesIcon,
  default1368 as WavesLadderIcon,
  default1371 as WaypointsIcon,
  default1370 as WebcamIcon,
  default1372 as WebhookIcon,
  default1375 as WebhookOffIcon,
  default1373 as WeightIcon,
  default1376 as WheatIcon,
  default1374 as WheatOffIcon,
  default1378 as WholeWordIcon,
  default1377 as WifiHighIcon,
  default1384 as WifiIcon,
  default1379 as WifiLowIcon,
  default1380 as WifiOffIcon,
  default1381 as WifiPenIcon,
  default1382 as WifiZeroIcon,
  default1383 as WindArrowDownIcon,
  default1385 as WindIcon,
  default1388 as WineIcon,
  default1386 as WineOffIcon,
  default1387 as WorkflowIcon,
  default1390 as WormIcon,
  default1389 as WrapTextIcon,
  default1391 as WrenchIcon,
  default1521 as XCircleIcon,
  default1392 as XIcon,
  default1617 as XOctagonIcon,
  default1770 as XSquareIcon,
  default1394 as YoutubeIcon,
  default1396 as ZapIcon,
  default1393 as ZapOffIcon,
  default1395 as ZoomInIcon,
  default1397 as ZoomOutIcon
};
