// .ignored/@ark-ui/solid/dist/chunk/DPP7XCXL.js
import { createSplitProps } from './ZMHI4GDJ.js';
import { ark } from './EPLBB4QN.js';
import { useEnvironmentContext } from './YO2MCGXO.js';
import { useLocaleContext } from './OKZ64GSY.js';
import { createContext } from './TROPIN4C.js';
import { runIfFn } from './DT73WLR4.js';
import { __export } from './ESLJRKWD.js';
import { createComponent } from 'solid-js/web';
import { mergeProps, useMachine, normalizeProps } from '@zag-js/solid';
import * as toggleGroup from '@zag-js/toggle-group';
export { anatomy } from '@zag-js/toggle-group';
import { createUniqueId, createMemo } from 'solid-js';

// src/components/toggle-group/use-toggle-group-context.ts
var [ToggleGroupProvider, useToggleGroupContext] = createContext({
  hookName: "useToggleGroupContext",
  providerName: "<ToggleGroupProvider />"
});

// src/components/toggle-group/toggle-group-context.tsx
var ToggleGroupContext = (props) => props.children(useToggleGroupContext());
var ToggleGroupItem = (props) => {
  const [toggleProps, restProps] = createSplitProps()(props, ["value", "disabled"]);
  const api = useToggleGroupContext();
  const mergedProps = mergeProps(() => api().getItemProps(toggleProps), restProps);
  return createComponent(ark.button, mergedProps);
};
var useToggleGroup = (props) => {
  const id = createUniqueId();
  const locale = useLocaleContext();
  const environment = useEnvironmentContext();
  const machineProps = createMemo(() => ({
    id,
    dir: locale().dir,
    getRootNode: environment().getRootNode,
    ...runIfFn(props)
  }));
  const service = useMachine(toggleGroup.machine, machineProps);
  return createMemo(() => toggleGroup.connect(service, normalizeProps));
};

// src/components/toggle-group/toggle-group-root.tsx
var ToggleGroupRoot = (props) => {
  const [useToggleGroupProps, restProps] = createSplitProps()(props, ["defaultValue", "deselectable", "disabled", "id", "ids", "loopFocus", "multiple", "onValueChange", "orientation", "rovingFocus", "value"]);
  const api = useToggleGroup(useToggleGroupProps);
  const mergedProps = mergeProps(() => api().getRootProps(), restProps);
  return createComponent(ToggleGroupProvider, {
    value: api,
    get children() {
      return createComponent(ark.div, mergedProps);
    }
  });
};
var ToggleGroupRootProvider = (props) => {
  const [{
    value: toggleGroup2
  }, localProps] = createSplitProps()(props, ["value"]);
  const mergedProps = mergeProps(() => toggleGroup2().getRootProps(), localProps);
  return createComponent(ToggleGroupProvider, {
    value: toggleGroup2,
    get children() {
      return createComponent(ark.div, mergedProps);
    }
  });
};

// src/components/toggle-group/toggle-group.ts
var toggle_group_exports = {};
__export(toggle_group_exports, {
  Context: () => ToggleGroupContext,
  Item: () => ToggleGroupItem,
  Root: () => ToggleGroupRoot,
  RootProvider: () => ToggleGroupRootProvider
});

export { ToggleGroupContext, ToggleGroupItem, ToggleGroupRoot, ToggleGroupRootProvider, toggle_group_exports, useToggleGroup, useToggleGroupContext };
